---
layout: post
title:  "Clone 31 aptamer"
date:   2024-05-06 00:00:00
author: Ke Chen, Yangyi Ren
categories: Aptamer
---
<!--p style="font-size: 24px"><strong>Keywords: </strong> <br/></p>
<font ><strong>small molecule, Synthetic, dyes, PubChem CID 25863</strong></font >
<br /-->

<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
<!--body>
  <p style="font-size: 16px">Click the buttons to navigate to different sections:</p>
  <div class="button-container">
    <a class="button" style="color:#520049" href="#description">Description</a>
    <a class="button" style="color:#520049" href="#SELEX">SELEX</a>
    <a class="button" style="color:#520049" href="#structure">Structure</a>
    <a class="button" style="color:#520049" href="#ligand-recognition">Ligand recognition
    <a class="button" style="color:#520049" href="#references">References</a>
   </a>
  </div>
</body-->
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
    .header_box {
     display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #c9c5c5;
      border-radius: 5px;
      width:225px;
	    height:50px;
  }
  .header_boxx1 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:115px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx2 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:140px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx3 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:85px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx4 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:120px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx5 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:215px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx6 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:135px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .blowheader_box{
    display: block;
      padding: 10px;
      font-size:18px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 5px;
      width:170px;
	    height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<br>
<br>


<p class="header_boxx1" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8841594/" target="_blank" style="color:#520049">1995</a></h3>
  </div>
  <div class="body">
    <p>Starting from a RNA pool that has a 120 base random sequence core, aptamers that bind specifically to the Tat protein were selected by repeating 11 rounds of selection and amplification<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10886365/" target="_blank" style="color:#520049">2000</a></h3>
  </div>
  <div class="body">
    <p>Rika Yamamoto et al. used an in vitro selection method and isolated a novel aptamer RNATat, a 37-mer RNA oligomer, that binds efficiently to the Tat protein of HIV-1<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/11132628/" target="_blank" style="color:#520049">2000</a></h3>
  </div>
  <div class="body">
    <p>Junji Kawakami et al. used Zn2+ for in vitro selecting to isolate novel RNA molecules that bind to the HIV-1 Tat protein, known as zinc-dependent aptamers<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12737819/" target="_blank" style="color:#520049">2003</a></h3>
  </div>
  <div class="body">
    <p>Akimasa Matsugami et al. determined the structure of an aptamer complexed with two argininamide molecules, two adjacent U:A:U base triples were formed<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/28620664/" target="_blank" style="color:#520049">2017</a></h3>
  </div>
  <div class="body">
    <p>Yudai Yamaoki et al. proposed an RNA aptamer that acquires binding capacity against HIV-1 Tat protein via G-quadruplex formation in response to potassium ions<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/31707021/" target="_blank" style="color:#520049">2020</a></h3>
  </div>
  <div class="body">
    <p>Mustafa Oguzhan Caglayan et al. reported the HIV-Tat protein detection performance of antiTat aptamers both for the spectroscopic ellipsometry (SE) and for the surface plasmon resonance enhanced total internal reflection ellipsometry (SPReTIRE) for the first time<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/37240414/" target="_blank" style="color:#520049">2023</a></h3>
  </div>
  <div class="body">
    <p>Using in vitro nuclear magnetic resonance (NMR), Omar Eladl et al. found that two U-A∗U base triples are formed in aptamer upon binding of Tat<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
  </div>
 </div>
</div>





<font><p class="header_boxx2" id="description">Description</p></font>
<font>In 2000, Junji Kawakami and colleagues isolated RNA aptamers that bind specifically to the HIV-1 Tat protein. It was worth mentioning that they used Zn2+ to participate in in vitro selecting and obtained a series of zinc-dependent aptamers<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.<br></font>
<br>
<br>


<p class="header_boxx3" id="SELEX">SELEX</p>
<p>In 2000, Junji Kawakam and colleagues prepared an RNA library having a 30 nucleotide random region and applied to an in vitro RNA aptamer selection with Zn2+. After 12 rounds of selecting, many unique sequences were revealed from a library selected with Zn2+ and the RNA with most abundant sequence (clone 31) bound to Tat tightly only when Zn2+ existed<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br>
<br>


<p class="header_boxx4" id="Structure">Structure</p>
<font>Clone 31 was the aptamer sequence mainly studied in the article, which had a high affinity with HIV-1 Tat protein. The 2D structure of the figures is based on the article by ribodraw tool to draw. These are two different prediction results in the article<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.<br></font>
<font><p>5'-GGGAGAAUUCCGACCAGAAGCUUUGGUUAUCAUGUUUAUGCGUACGGGCGCCCAUAUGUGCGUCUACAUGGAUCCUCA-3'</p></font>
<!--b>2D representation</b-->
        
<!--font><p>Szostak et al. obtained 17 different sequences through SELEX, and designed a 40nt sequence after comparing the information of 17 sequences. The aptamer of the 40nt is a sequence of analytic structures, and the following diagram shows its secondary structure.
Through SELEX, 17 different sequences were obtained, and after comparing 17 sequence information, a 40nt sequence was designed
 <sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p></font-->
 <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/clone 31_aptamer_2D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/clone 31_aptamer_2D2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
 <br>
                     
<!--b>3D visualisation</b><br>               
<font >ATP aptamer complex was generated from PDB ID:  at 3.05 Å . Additional available structures that have been solved and detailed information are accessible on <i>Structures</i> page <sup>[<a href="#ref3" style="color:#520049" >3</a>]</sup>.</font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/AMP-3D.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/AMP-3D-bp.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table-->
  <br>

<font ><p class="header_boxx5" id="ligand-information">Ligand information</p></font>  
  

<p class="blowheader_box">SELEX ligand</p>
<font>The retroviral Tat protein binds to the TAR RNA. This activates transcriptional initiation and elongation from the LTR promoter. Binding is mediated by an arginine rich region.-----from Pfam</font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">HIV-1 Tat protein</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P04608/entry" target="_blank" style="color:#520049"><b>P04608</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00539/" target="_blank" style="color:#520049"><b>PF00539</b></a></td>
        <td name="td3">9.8 kDa</td>
        <td name="td4">MEPVDPRLEPWKHPGSQPKTACTNCYCKKCCFHCQVCFITKALGISYGRKKRRQRRRAHQNSQTHQASLSKQPTSQPRGDPTGPKE</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/6CYT" target="_blank" style="color:#520049"><b>6CYT</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/155871" target="_blank" style="color:#520049"><b>155871</b></a></td>
      </tr>
	  </tbody>
  </table>
<font>Some isolated sequences bind to the affinity of the protein.</font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">clone 31</td>
        <td name="td1">5'-GGGAGAAUUCCGACCAGAAGCUUUGGUUAUCAUGUUUAUGCGUACGGGCGCCCAUAUGUGCGUCUACAUGGAUCCUCA-3'</td>
        <td name="td2">HIV-1 Tat protein</td>
        <td name="td3">0.31 uM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/clone 31_aptamer_SELEX_ligand.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>
<br>



<p class="blowheader_box">Similar compound</p>                    
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
       <tr>
      <td name="td0">3MI9-C</td>
      <td name="td1">9.7</td>
      <td name="td2">0.0</td>
      <td name="td3">Cell division protein kinase 9</td>
    </tr>
     <tr>
      <td name="td0">3MIA-C</td>
      <td name="td1">8.9</td>
      <td name="td2">0.2</td>
      <td name="td3">Cell division protein kinase 9</td>
    </tr>
     <tr>
      <td name="td0">4OR5-C</td>
      <td name="td1">8.6</td>
      <td name="td2">0.4</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">4OGR-D</td>
      <td name="td1">8.6</td>
      <td name="td2">0.5</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">4OR5-H</td>
      <td name="td1">8.6</td>
      <td name="td2">0.3</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">4OGR-H</td>
      <td name="td1">8.6</td>
      <td name="td2">0.8</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">4OGR-M</td>
      <td name="td1">8.5</td>
      <td name="td2">0.5</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">6CYT-D</td>
      <td name="td1">8.4</td>
      <td name="td2">0.5</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">5L1Z-D</td>
      <td name="td1">8.1</td>
      <td name="td2">0.9</td>
      <td name="td3">Cyclin-dependent kinase 9</td>
    </tr>
     <tr>
      <td name="td0">6GS9</td>
      <td name="td1">0.303</td>
      <td name="td2">1.36</td>
      <td name="td3">NMR structure of aurein 2.5 in SDS micelles</td>
    </tr>
    </tbody>
  </table>
<p><br /></p>

                 
<p class="header_boxx6" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] In vitro selection of RNA aptamers that can bind specifically to Tat protein of HIV-1.</strong></font><br />
Yamamoto, R., Toyoda, S., Viljanen, P., Machida, K., Nishikawa, S., Murakami, K., Taira, K., & Kumar, P. K.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8841594/" target="_blank" style="color:#520049">Nucleic acids symposium series, (34), 145–146. (1995)</a>
<br/>
            
<a id="ref2"></a><font><strong>[2] A novel RNA motif that binds efficiently and specifically to the Ttat protein of HIV and inhibits the trans-activation by Tat of transcription in vitro and in vivo.</strong></font><br />
Yamamoto, R., Katahira, M., Nishikawa, S., Baba, T., Taira, K., & Kumar, P. K.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10886365/" target="_blank" style="color:#520049">Genes to cells : devoted to molecular & cellular mechanisms, 5(5), 371–388. (2000)</a>
<br/>
            
<a id="ref3"></a><font><strong>[3]  In vitro selection of aptamers that act with Zn2+.</strong></font><br />
Kawakami, J., Imanaka, H., Yokota, Y., & Sugimoto, N.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/11132628/" target="_blank" style="color:#520049">Journal of inorganic biochemistry, 82(1-4), 197–206. (2000)</a>
<br/>
            
<a id="ref4"></a><font><strong>[4] Structural basis of the highly efficient trapping of the HIV Tat protein by an RNA aptamer.</strong></font><br />
Matsugami, A., Kobayashi, S., Ouhashi, K., Uesugi, S., Yamamoto, R., Taira, K., Nishikawa, S., Kumar, P. K., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12737819/" target="_blank" style="color:#520049">Structure (London, England : 1993), 11(5), 533–545. (2003)</a>
<br/>
            
<a id="ref5"></a><font><strong>[5] Development of an RNA aptamer that acquires binding capacity against HIV-1 Tat protein via G-quadruplex formation in response to potassium ions.</strong></font><br />
Yamaoki, Y., Nagata, T., Mashima, T., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/28620664/" target="_blank" style="color:#520049">Chemical communications (Cambridge, England), 53(52), 7056–7059. (2017)</a>
<br/>
            
<a id="ref6"></a><font><strong>[6] Spectrophotometric ellipsometry based Tat-protein RNA-aptasensor for HIV-1 diagnosis.</strong></font><br />
Caglayan, M. O., & Üstündağ, Z.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/31707021/" target="_blank" style="color:#520049">Spectrochimica acta. Part A, Molecular and biomolecular spectroscopy, 227, 117748. (2020)</a>
<br/>
            
<a id="ref7"></a><font><strong>[7] Complex Formation of an RNA Aptamer with a Part of HIV-1 Tat through Induction of Base Triples in Living Human Cells Proven by In-Cell NMR.</strong></font><br />
Eladl, O., Yamaoki, Y., Kondo, K., Nagata, T., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/37240414/" target="_blank" style="color:#520049">International journal of molecular sciences, 24(10), 9069. (2023)</a>
<br/>