---
layout: post
title:  "Colicin E3 aptamer"
date:   2024-05-27 00:00:00
author: Qiaozhen Liu, Zhijie Tan
categories: Aptamer
---
<!--p style="font-size: 24px"><strong>Keywords: </strong> <br/></p>
<font ><strong>small molecule, Synthetic, dyes, PubChem CID 25863</strong></font >
<br /-->

<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
    .header_box {
    display: block;
    font-size: 24px;
    background-color: #ffffff;
    text-decoration: none;
    border-radius: 1px;
    width: 500px;
    border-width: 1px 1px 2px 1px;
    border-color: #ffffff #ffffff #ffffff #ffffff;
}
.blowheader_box{
    display: block;
      padding: 6px;
      font-size:20px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 1px;
      width:170px;
	    height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<br>
<br>



<font><p class="header_box" id="description">Description</p></font>
<font>In 2004, Ichiro Hirao and colleagues used in vitro selection to isolate RNA aptamers that bind to the C-terminal ribonuclease domain of colicin E3. They specifically bound to the protein and prevented the 16S rRNA cleavage by the C-terminal ribonuclease domain<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<br>
<br>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2004, Ichiro Hirao et al. isolated specific sequences which bind colicin E3 CRD by 12 cycles of iterative selection<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br>
<br>


<p class="header_box" id="Structure">Structure</p>
<font>The aptamers have sequences similar to the region of residues 1484-1506, which includes the E. coli 16S rRNA cleavage site. The 2D structure of the figure is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<font><p>5'-UCCCUGGCCCAAGAUCCUAAUAAAGUUUUUUCGGACCGGAGCGAAACCACUAUCCUCUUAAGCAAUCUGU-3'</p></font>
<img src="/images/2D/E3_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>
<br>


<font ><p class="header_box" id="ligand-information">Ligand information</p></font>  

<p class="blowheader_box">SELEX ligand</p>
<font>Colicins are classified according to the cell-surface receptor they bind to, colicin E3 binding to the BtuB receptor involved in vitamin B12 uptake. The lethal action of colicin E3 arises from its ability to inactivate the ribosome by site-specific RNase cleavage of the 16S ribosomal RNA, which is carried out by the catalytic, or ribonuclease domain. Colicin E3 is comprised of three domains, each domain being involved in a different stage of infection: receptor binding, translocation and cytotoxicity. Colicin E3 is a Y-shaped molecule with the receptor-binding middle domain forming the stalk, the N-terminal translocation domain forming the two globular heads, and the C-terminal catalytic domain forming the two globular arms. To neutralise the toxic effects of colicin E3, the host cell produces an immunity protein, which binds to the C-terminal end of the ribonuclease domain and effectively suppresses its activity.-----from Pfam</font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Colicin E3</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P00646/entry" target="_blank" style="color:#520049"><b>P00646</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR009105/" target="_blank" style="color:#520049"><b>IPR009105</b></a></td>
        <td name="td3">67.73 kDa</td>
        <td name="td4">MSGGDGRGHNTGAHSTSGNINGGPTGLGVGGGASDGSGWSSENNPWGGGSGSGIHWGGGSGHGNGGGNGNSGGGSGTGGNLSAVAAPVAFGFPALSTPGAGGLAVSISAGALSAAIADIMAALKGPFKFGLWGVALYGVLPSQIAKDDPNMMSKIVTSLPADDITESPVSSLPLDKATVNVNVRVVDDVKDERQNISVVSGVPMSVPVVDAKPTERPGVFTASIPGAPVLNISVNNSTPAVQTLSPGVTNNTDKDVRPAFGTQGGNTRDAVIRFPKDSGHNAVYVSVSDVLSPDQVKQRQDEENRRQQEWDATHPVEAAERNYERARAELNQANEDVARNQERQAKAVQVYNSRKSELDAANKTLADAIAEIKQFNRFAHDPMAGGHRMWQMAGLKAQRAQTDVNNKQAAFDAAAKEKSDADAALSSAMESRKKKEDKKRSAENNLNDEKNKPRKGFKDYGHDYHPAPKTENIKGLGDLKPGIPKTPKQNGGGKRKRWTGDKGRKIYEWDSQHGELEGYRASDGQHLGSFDPKTGNQLKGPDPKRNIKKYLGLKLDLTWFDKSTEDFKGEEYSKDFGDDGSVMESLGVPFKDNVNNGCFDVIAEWVPLLQPYFNHQIDISDNEYFVSFDYRDGDW</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1JCH" target="_blank" style="color:#520049"><b>1JCH</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/nuccore/J01574.1" target="_blank" style="color:#520049"><b>J01574.1</b></a></td>
      </tr>
	  </tbody>
  </table>

<font>Affinity of selected the aptamer for colicin E3 CRD was determined by nitrocellulose filter-binding assays.</font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">F2-1</td>
        <td name="td1">5'-UCCCUGGCCCAAGAUCCUAAUAAAGUUUUUUCGGACCGGAGCGAAACCACUAUCCUCUUAAGCAAUCUGU-3'</td>
        <td name="td2">Colicin E3 CRD</td>
        <td name="td3">14±2 nM</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/E3_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>



<p class="blowheader_box">Similar compound</p>                    
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">2B5U-A</td>
      <td name="td1">33.6</td>
      <td name="td2">1.0</td>
      <td name="td3">Colicin E3</td>
    </tr>
            
     <tr>
      <td name="td0">7NST-D</td>
      <td name="td1">23.8</td>
      <td name="td2">1.6</td>
      <td name="td3">Outer membrane protein f</td>
    </tr>
            
     <tr>
      <td name="td0">5EW5-A</td>
      <td name="td1">22.4</td>
      <td name="td2">2.5</td>
      <td name="td3">Colicin-E9</td>
    </tr>
            
     <tr>
      <td name="td0">1RH1-A</td>
      <td name="td1">19.7</td>
      <td name="td2">9.8</td>
      <td name="td3">Colicin B</td>
    </tr>
            
     <tr>
      <td name="td0">8AUC-B</td>
      <td name="td1">11.8</td>
      <td name="td2">17.3</td>
      <td name="td3">Cell wall-associated hydrolases （invasion-associa</td>
    </tr>
            
     <tr>
      <td name="td0">8W20-A</td>
      <td name="td1">11.7</td>
      <td name="td2">46.9</td>
      <td name="td3">Secreted protein</td>
    </tr>
            
     <tr>
      <td name="td0">7NNA-A</td>
      <td name="td1">11.5</td>
      <td name="td2">9.7</td>
      <td name="td3">Klebicin c activity</td>
    </tr>
            
     <tr>
      <td name="td0">3K29-A</td>
      <td name="td1">11.4</td>
      <td name="td2">8.7</td>
      <td name="td3">Putative uncharacterrized protein</td>
    </tr>
            
     <tr>
      <td name="td0">1CII-A</td>
      <td name="td1">11.3</td>
      <td name="td2">21.3</td>
      <td name="td3">Colicin IA</td>
    </tr>
            
     <tr>
      <td name="td0">4MH6-A</td>
      <td name="td1">11.1</td>
      <td name="td2">4.4</td>
      <td name="td3">Putative type iii secretion protein ysco</td>
    </tr>
    </tbody>
  </table>
<br>
<br>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] In vitro selection of RNA aptamers that bind to colicin E3 and structurally resemble the decoding site of 16S ribosomal RNA.</strong></font><br />
Hirao, I., Harada, Y., Nojima, T., Osawa, Y., Masaki, H., & Yokoyama, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/35970037/" target="_blank" style="color:#520049">Biochemistry, 43(11), 3214–3221. (2004)</a>
<br/>