---
layout: post
title:  "FC RNA aptamer"
date:   2024-05-03 00:00:00
author: Zhizhong Lu, Ying Ao
categories: Aptamer
---
<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
<!--body>
  <p style="font-size: 16px">Click the buttons to navigate to different sections:</p>
  <div class="button-container">
    <a class="button" style="color:#520049" href="#timeline">Timeline</a>
    <a class="button" style="color:#520049" href="#description">Description</a>
    <a class="button" style="color:#520049" href="#SELEX">SELEX</a>
    <a class="button" style="color:#520049" href="#structure">Structure</a>
    <a class="button" style="color:#520049" href="#ligand-recognition">Ligand recognition
    <a class="button" style="color:#520049" href="#references">References</a>
   </a>
  </div>
</body-->
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
  .header_box {
     display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #c9c5c5;
      border-radius: 5px;
      width:225px;
	    height:50px;
  }
  .header_boxx1 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:115px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx2 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:140px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx3 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:85px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx4 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:120px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx5 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:215px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx6 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:135px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .blowheader_box{
    display: block;
      padding: 10px;
      font-size:18px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 5px;
      width:170px;
	    height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<p><br /></p>


<p class="header_boxx1" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9346949/" target="_blank" style="color:#520049">1997</a></h3>
    </div>
    <div class="body">
      <p>Isolated  RNA molecules that selectively bind to yeast Pol II<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/16341226/" target="_blank" style="color:#520049">2006</a></h3>
    </div>
    <div class="body">
      <p>Report the crystal structure of FC RNA (aptamer) bound to Pol II<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/29570714/" target="_blank" style="color:#520049">2018</a></h3>
    </div>
    <div class="body">
      <p>RNA sequences with high affinity to RNA Pol by performing genomic SELEX in Escherichia coli genome<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/32663063/" target="_blank" style="color:#520049">2020</a></h3>
    </div>
    <div class="body">
      <p>Identified a variety of Pol II-binding aptamers, the nascent RNA binds Pol II to silence its own expression<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
</div>


         
<font ><p class="header_boxx2" id="description">Description</p></font>
<font >In 1997, Thomas, M. et al. isolated RNA aptamers targeted yeast RNA polymerase II (Pol II), one selected RNA ligand was a specific inhibitor of Saccharomyces cerevisiae Pol II. In 2006, Kettenberger, H. et al. determined the crystal structure of FC RNA aptamer bound to Pol II. The noncoding RNA B2 and the RNA aptamer FC bind RNA polymerase (Pol) II and inhibit messenger RNA transcription initiation<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.<br></font>
<p><br /></p> <br>

<p class="header_boxx3" id="SELEX">SELEX</p>
<p>RNA aptamers that bind to S. cerevisiae Pol II were selected by the Selex strategy. The starting RNA library consisted of a randomized region of 40 nucleotides flanked by defined sequences at the 5′- and 3′- ends.  An excess of RNAs (40 pmol) was incubated with a highly purified preparation of Pol II (5 pmol). The RNA·protein complexes were retained on a nitrocellulose filter by filtration, whereas free RNAs passed through the membrane. After extensive washing of the membrane, the protein·RNA complexes were dissociated by phenol extraction, and the RNA was amplified. This selection/amplification procedure was repeated nine times and resulted in the new RNA pool<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br> <br>    



<p class="header_boxx4" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<font><p>Here we used ribodraw to complete the figure, through the 3D structure information<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p></font>
<font><p>5'-ACAGCACUGAUUGCGGUCGAGGUAGCUUGAUGG-3'</p></font>
<img src="/images/2D/FC_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;">   
</div>
<br>


<p class="blowheader_box">3D visualisation</p>             
<font >FC RNA was cocrystallized with the complete 12-subunit Pol II, and the X-ray structure was determined at 3.8-Å resolution. The molecular architecture reveals that the FC RNA molecule is strategically positioned within the polymerase's active center cleft, which is situated superior to the bridge helix. The RNA establishes contact with both the clamp and fork domains, which are located on opposing sides of the cleft, aligning with the cross-linking evidence that FC RNA interacts with the two major subunits of Pol II. The PDB ID of this structure is 2B63<sup>[<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>. Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</font>
<div ><font ><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p></font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/FC_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A', 
  start_residue_number:2, 
  end_residue_number:1455, 
  color:{r:245,g:222,b:179}
},
{
  struct_asym_id: 'B', 
  start_residue_number:20, 
  end_residue_number:1224, 
  color:{r:245,g:222,b:179}
},
{
  struct_asym_id: 'C', 
  start_residue_number:3, 
  end_residue_number:268, 
  color:{r:245,g:222,b:179}
},
{
  struct_asym_id: 'D', 
  start_residue_number:4, 
  end_residue_number:221, 
  color:{r:245,g:222,b:179}
},
{
  struct_asym_id: 'E', 
  start_residue_number:2, 
  end_residue_number:215, 
  color:{r:245,g:222,b:179}
},
{
  struct_asym_id: 'F', 
  start_residue_number:72, 
  end_residue_number:155, 
  color:{r:245,g:222,b:179}
},
{
  struct_asym_id: 'G', 
  start_residue_number:1, 
  end_residue_number:171, 
  color:{r:245,g:222,b:179}
},
{
  struct_asym_id: 'H', 
  start_residue_number:2, 
  end_residue_number:146, 
  color:{r:245,g:222,b:179}
},
{
  struct_asym_id: 'I', 
  start_residue_number:2, 
  end_residue_number:120, 
  color:{r:245,g:222,b:179}
},
{
  struct_asym_id: 'J', 
  start_residue_number:1, 
  end_residue_number:65, 
  color:{r:245,g:222,b:179}
},
{
  struct_asym_id: 'K', 
  start_residue_number:1, 
  end_residue_number:115, 
  color:{r:245,g:222,b:179}
},
{
  struct_asym_id: 'L', 
  start_residue_number:25, 
  end_residue_number:70, 
  color:{r:245,g:222,b:179}
},
{
  struct_asym_id: 'R', 
  start_residue_number:2, 
  end_residue_number:19, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'R', 
  start_residue_number:6, 
  end_residue_number:6, 
  color:{r:0,g:128,b:0}
},
{
  struct_asym_id: 'R', 
  start_residue_number:10, 
  end_residue_number:10, 
  color:{r:0,g:128,b:0}
},
{
  struct_asym_id: 'R', 
  start_residue_number:20, 
  end_residue_number:32, 
  color:{r:0,g:128,b:0}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/FC-2B63-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html></td>
</tr></table><br>
</div>


<p class="blowheader_box">Binding pocket</p>             
<font >Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 2B63 at 3.8 Å resolution. RNA polymerase (Pol) II (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with RNA polymerase (Pol) II. The hydrogen bond interaction refers to pymol prediction and articles.</font>
<br><br>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/FC_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/FC_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br><br>





<font ><p class="header_boxx5" id="ligand-recognition">Ligand information</p></font>  
<p class="blowheader_box">SELEX ligand</p>
<font>Affinity determined by in vitro binding assays. For filter-binding competition assays, 5′ 32P-labeled RNA was incubated with 100 nM protein under the same conditions, adding increasing amounts (1–1,000 nM) of unlabeled RNA competitor and then incubating at 30 °C for 30 min<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</font >
<div style="display: flex; justify-content: center;">
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">FC RNA aptamer</sup></td>
        <td name="td1">5'-ACAGCACUGAUUGCGGUCGAGGUAGCUUGAUGG-3'</td>
        <td name="td3">Pol II</td>
        <td name="td4">33±2 nM</td>
      </tr>
      <tr>
        <td name="td0">Mutant 1</td>
        <td name="td1">5'-ACAGGACUGAUUGCGGUCGAGGUAGCUUGAUGG-3'</td>
        <td name="td3">Pol II</td>
        <td name="td4">43±1 nM</td>
      </tr>
      <tr>
        <td name="td0">Mutant 2</td>
        <td name="td1">5'-ACAGCACUAAUUGCGGUCGAGGUAGCUUGAUGG-3'</td>
        <td name="td3">Pol II</td>
        <td name="td4">45±2 nM</td>
      </tr>
      <tr>
        <td name="td0">Mutant 3</td>
        <td name="td1">5'-ACAGCACUGGUUGCGGUCGAGGUAGCUUGAUGG-3'</td>
        <td name="td3">Pol II</td>
        <td name="td4">62±7 nM</td>
      </tr>
      <tr>
        <td name="td0">Mutant 4</td>
        <td name="td1">5'-ACAGCACUGAUUCCGGUCGAGGUAGCUUGAUGG-3'</td>
        <td name="td3">Pol II</td>
        <td name="td4">35±4 nM</td>
      </tr>
      <tr>
        <td name="td0">Mutant 5</td>
        <td name="td1">5'-ACAGGACUGAUUCCGGUCGAGGUAGCUUGAUGG-3'</td>
        <td name="td3">Pol II</td>
        <td name="td4">45±6 nM</td>
      </tr>
      <tr>
        <td name="td0">Mutant 6</td>
        <td name="td1">5'-ACAGCCACUGAUUGGCGGUCGAGGUAGCUUGAUGG-3'</td>
        <td name="td3">Pol II</td>
        <td name="td4">37±3 nM</td>
      </tr>
      <tr>
        <td name="td0">Mutant 7</td>
        <td name="td1">5'-ACAGCGCACUGAUUGCGCGGUCGAGGUAGCUUGAUGG-3'</td>
        <td name="td3">Pol II</td>
        <td name="td4">53±4 nM</td>
      </tr>
      <tr>
        <td name="td0">Mutant 8</td>
        <td name="td1">5'-ACAGCACUGAUUGCGGUCGAUGGUAGCGUUGAUGG-3'</td>
        <td name="td3">Pol II</td>
        <td name="td4">25±3 nM</td>
      </tr>
      <tr>
        <td name="td0">Mutant 9</td>
        <td name="td1">5'-AGCUUGAUGAGCACUGAUUGCCGUCGAGGU-3'</td>
        <td name="td3">Pol II</td>
        <td name="td4">No binding</td>
      </tr>
	  </tbody>
  </table>
  </div>
<br>

<p class="blowheader_box">Structure ligand</p>
<font><p>DNA-directed RNA polymerases (also known as DNA-dependent RNA polymerases) are responsible for the polymerisation of ribonucleotides into a sequence complementary to the template DNA. In eukaryotes, there are three different forms of DNA-directed RNA polymerases transcribing different sets of genes. Eukaryotic nuclei contain three distinct types of RNA polymerases that differ in the RNA they synthesise: RNA polymerase I: located in the nucleoli, synthesises precursors of most ribosomal RNAs. RNA polymerase II: occurs in the nucleoplasm, synthesises mRNA precursors. RNA polymerase III: also occurs in the nucleoplasm, synthesises the precursors of 5S ribosomal RNA, the tRNAs, and a variety of other small nuclear and cytosolic RNAs.</p></font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">MW</th>
        <th onclick="sortTable(3)">Amino acids sequences</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P04050/" target="_blank" style="color:#520049"><b>P04050</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR045867/" target="_blank" style="color:#520049"><b>IPR045867</b></a></td>
        <td name="td2">191.64 kDa</td>
        <td name="td3">MVGQQYSSAPLRTVKEVQFGLFSPEEVRAISVAKIRFPETMDETQTRAKIGGLNDPRLGSIDRNLKCQTCQEGMNECPGHFGHIDLAKPVFHVGFIAKIKKVCECVCMHCGKLLLDEHNELMRQALAIKDSKKRFAAIWTLCKTKMVCETDVPSEDDPTQLVSRGGCGNTQPTIRKDGLKLVGSWKKDRATGDADEPELRVLSTEEILNIFKHISVKDFTSLGFNEVFSRPEWMILTCLPVPPPPVRPSISFNESQRGEDDLTFKLADILKANISLETLEHNGAPHHAIEEAESLLQFHVATYMDNDIAGQPQALQKSGRPVKSIRARLKGKEGRIRGNLMGKRVDFSARTVISGDPNLELDQVGVPKSIAKTLTYPEVVTPYNIDRLTQLVRNGPNEHPGAKYVIRDSGDRIDLRYSKRAGDIQLQYGWKVERHIMDNDPVLFNRQPSLHKMSMMAHRVKVIPYSTFRLNLSVTSPYNADFDGDEMNLHVPQSEETRAELSQLCAVPLQIVSPQSNKPCMGIVQDTLCGIRKLTLRDTFIELDQVLNMLYWVPDWDGVIPTPAIIKPKPLWSGKQILSVAIPNGIHLQRFDEGTTLLSPKDNGMLIIDGQIIFGVVEKKTVGSSNGGLIHVVTREKGPQVCAKLFGNIQKVVNFWLLHNGFSTGIGDTIADGPTMREITETIAEAKKKVLDVTKEAQANLLTAKHGMTLRESFEDNVVRFLNEARDKAGRLAEVNLKDLNNVKQMVMAGSKGSFINIAQMSACVGQQSVEGKRIAFGFVDRTLPHFSKDDYSPESKGFVENSYLRGLTPQEFFFHAMGGREGLIDTAVKTAETGYIQRRLVKALEDIMVHYDNTTRNSLGNVIQFIYGEDGMDAAHIEKQSLDTIGGSDAAFEKRYRVDLLNTDHTLDPSLLESGSEILGDLKLQVLLDEEYKQLVKDRKFLREVFVDGEANWPLPVNIRRIIQNAQQTFHIDHTKPSDLTIKDIVLGVKDLQENLLVLRGKNEIIQNAQRDAVTLFCCLLRSRLATRRVLQEYRLTKQAFDWVLSNIEAQFLRSVVHPGEMVGVLAAQSIGEPATQMTLNTFHFAGVASKKVTSGVPRLKEILNVAKNMKTPSLTVYLEPGHAADQEQAKLIRSAIEHTTLKSVTIASEIYYDPDPRSTVIPEDEEIIQLHFSLLDEEAEQSFDQQSPWLLRLELDRAAMNDKDLTMGQVGERIKQTFKNDLFVIWSEDNDEKLIIRCRVVRPKSLDAETEAEEDHMLKKIENTMLENITLRGVENIERVVMMKYDRKVPSPTGEYVKEPEWVLETDGVNLSEVMTVPGIDPTRIYTNSFIDIMEVLGIEAGRAALYKEVYNVIASDGSYVNYRHMALLVDVMTTQGGLTSVTRHGFNRSNTGALMRCSFEETVEILFEAGASAELDDCRGVSENVILGQMAPIGTGAFDVMIDEESLVKYMPEQKITEIEDGQDGGVTPYSNESGLVNADLDVKDELMFSPLVDSGSNDAMAGGFTAYGGADYGEATSPFGAYGEAPTSPGFGVSSPGFSPTSPTYSPTSPAYSPTSPSYSPTSPSYSPTSPSYSPTSPSYSPTSPSYSPTSPSYSPTSPSYSPTSPSYSPTSPSYSPTSPSYSPTSPSYSPTSPSYSPTSPSYSPTSPSYSPTSPAYSPTSPSYSPTSPSYSPTSPSYSPTSPSYSPTSPNYSPTSPSYSPTSPGYSPGSPAYSPKQDEQKHNENENSR (RNA polymerase II largest subunit)</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1I50" target="_blank" style="color:#520049"><b>1I50</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&id=559292" target="_blank" style="color:#520049"><b>559292</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/FC_stru_ligand.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;">
  </div>


<p class="blowheader_box">Similar compound</p>
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">2JA7-A</td>
        <td name="td2">47.0</td>
        <td name="td3">0.8</td>
        <td name="td4">DNA-directed rna polymerase II largest subunit</td>
      </tr>
      <tr>
        <td name="td1">2JA7-M</td>
        <td name="td2">46</td>
        <td name="td3">0.8</td>
        <td name="td4">DNA-directed rna polymerase II largest subunit</td>
      </tr>
      <tr>
        <td name="td1">5FMF-A</td>
        <td name="td2">37.9</td>
        <td name="td3">1.0</td>
        <td name="td4">Dna repair helicase rad25, ssl2</td>
      </tr>
      <tr>
        <td name="td1">3HOY-A</td>
        <td name="td2">45.9</td>
        <td name="td3">0.9</td>
        <td name="td4">DNA-directed rna polymerase II subunit rpb1</td>
      </tr>
      <tr>
        <td name="td1">3K1F-A</td>
        <td name="td2">45.9</td>
        <td name="td3">1.2</td>
        <td name="td4">DNA-directed rna polymerase II subunit rpb1</td>
      </tr>
      <tr>
        <td name="td1">3S1M-A</td>
        <td name="td2">45.5</td>
        <td name="td3">1.2</td>
        <td name="td4">DNA-directed rna polymerase II subunit rpb1</td>
      </tr>
      <tr>
        <td name="td1">1WCM-A</td>
        <td name="td2">40.0</td>
        <td name="td3">0.6</td>
        <td name="td4">DNA-directed rna polymerase II largest subunit</td>
      </tr>
      <tr>
        <td name="td1">3J0K-A</td>
        <td name="td2">45.1</td>
        <td name="td3">1.4</td>
        <td name="td4">DNA-directed rna polymerase II largest subunit</td>
      </tr>
      <tr>
        <td name="td1">7O4J-A</td>
        <td name="td2">45.1</td>
        <td name="td3">1.1</td>
        <td name="td4">General transcription and DNA repair factor II</td>
      </tr>
      <tr>
        <td name="td1">3HOV-A</td>
        <td name="td2">44.7</td>
        <td name="td3">0.8</td>
        <td name="td4">DNA-directed rna polymerase II subunit rpb1</td>
      </tr>
	  </tbody>
  </table>

<br><br>

                 
<p class="header_boxx6" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Selective targeting and inhibition of yeast RNA polymerase II by RNA aptamers.</strong></font><br />
Thomas, M., Chédin, S., Carles, C., Riva, M., Famulok, M., & Sentenac, A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9346949/" target="_blank" style="color:#520049">The Journal of biological chemistry, 272(44), 27980–27986. (1997)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Structure of an RNA polymerase II-RNA inhibitor complex elucidates transcription regulation by noncoding RNAs.</strong></font><br />
Kettenberger, H., Eisenführ, A., Brueckner, F., Theis, M., Famulok, M., & Cramer, P.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16341226/" target="_blank" style="color:#520049">Nature structural & molecular biology, 13(1), 44–48. (2006)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Nascent RNA signaling to yeast RNA Pol II during transcription elongation.</strong></font><br />
Klopf, E., Moes, M., Amman, F., Zimmermann, B., von Pelchrzim, F., Wagner, C., & Schroeder, R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/29570714/" target="_blank" style="color:#520049">PloS one, 13(3), e0194438. (2018)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] RNA polymerase II-binding aptamers in human ACRO1 satellites disrupt transcription in cis.</strong></font><br />
Boots, J. L., von Pelchrzim, F., Weiss, A., Zimmermann, B., Friesacher, T., Radtke, M., Żywicki, M., Chen, D., Matylla-Kulińska, K., Zagrovic, B., & Schroeder, R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/32663063/" target="_blank" style="color:#520049">Transcription, 11(5), 217–229. (2020)</a>
<br />