---
layout: post
title:  "GlnRs aptamer"
date:   2024-04-16 00:00:00
author: Jiali Wang, Zhizhong Lu
categories: Aptamer
---
<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
<!--body>
  <p style="font-size: 16px">Click the buttons to navigate to different sections:</p>
  <div class="button-container">
    <a class="button" style="color:#520049" href="#timeline">Timeline</a>
    <a class="button" style="color:#520049" href="#description">Description</a>
    <a class="button" style="color:#520049" href="#SELEX">SELEX</a>
    <a class="button" style="color:#520049" href="#structure">Structure</a>
    <a class="button" style="color:#520049" href="#ligand-recognition">Ligand recognition
    <a class="button" style="color:#520049" href="#references">References</a>
   </a>
  </div>
</body-->
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
  .header_box {
     display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #c9c5c5;
      border-radius: 5px;
      width:225px;
	    height:50px;
  }
  .header_boxx1 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:115px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx2 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:140px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx3 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:85px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx4 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:120px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx5 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:215px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx6 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:135px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .blowheader_box{
    display: block;
      padding: 10px;
      font-size:18px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 5px;
      width:170px;
	    height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<p><br /></p>


<p class="header_boxx1" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/2459391/" target="_blank" style="color:#520049">1988</a></h3>
    </div>
    <div class="body">
      <p>Natural RNA sequences that bind to GlnRs<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/2479982/" target="_blank" style="color:#520049">1989</a></h3>
    </div>
    <div class="body">
      <p>Structure of E. coli glutaminyl-tRNA synthetase complexed with tRNA(Gln) and ATP at 2.8 A resolution<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://www.proquest.com/openview/6857e35043f04e11f53714d80cdc4dcf/1?pq-origsite=gscholar&cbl=18750&diss=y" target="_blank" style="color:#520049">1996</a></h3>
    </div>
    <div class="body">
      <p>The aptamer is initially selected through the SELEX<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10881199/" target="_blank" style="color:#520049">2000</a></h3>
    </div>
    <div class="body">
      <p>A tRNA mutant with a 30 fold increased affinity for binding to GlnRS compared to the wild-type<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/14681579/" target="_blank" style="color:#520049">2004</a></h3>
    </div>
    <div class="body">
      <p>Aptamer redesigned tRNA is nonfunctional and degraded in cells<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
</div>


         
<font ><p class="header_boxx2" id="description">Description</p></font>
<font >In 1992, Stubenrauch, M. used standard in vitro selection techniques to randomly select tRNAGln molecular pools in D, T, and variable loops based on the binding to GlnRS. Afterwards, in 2000, Bullock T L et al. analyzed the complex structure of a tight binding transfer RNA aptamer with GlnRs by X-ray crystallographic methods<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.<br></font>
<p><br /></p> <br>

<p class="header_boxx3" id="SELEX">SELEX</p>
<p>In 2000, Bullock TL, Sherlin LD, Perona JJ. used Stubenrauch's tRNAGln library. The library comprised randomization of nucleotides at positions 15–21, 47–48 and 53–61, and RNAs capable of binding GlnRS were isolated by filtration through nitrocellulose. They obtained 46 possessed a four-nucleotide replacement of the wild type five-nucleotide variable loop (5'-CAUUC-3') through 6 rounds of screening, of which 33 of these were of the sequence 5'-AGGU-3'<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br> <br>    



<p class="header_boxx4" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<font><p>Here we use ribodraw to complete the figure, through the 3D structure information. GlnRs-aptamer was the aptamer sequence mainly studied in SELEX article<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup>.</p></font>
<font><p>5'-UGGGGUAUCGCCAAGCGGUAAGGCACCGGAUUCUGAUUCCGGAGGUCGAGGUUCGAAUCCUCGUACCCCAGCCA-3'</p></font>
<img src="/images/2D/GlnRs_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;">   
</div>
<br>


<p class="blowheader_box">3D visualisation</p>             
<font >Bullock TL, Sherlin LD, Perona JJ. sovled the crystal structure, at 2.7 A resolution, of an RNA aptamer bound to glutaminyl-tRNA synthetase has been determined. The structure does not clearly indicate the relationship between the interacting bases and amino acids. The PDB ID of this structure is 1EXD. Sherlin LD, Bullock TL, Newberry KJ, et al. reported the interaction between G10-C16 of tRNA and 6 amino acids of GlnRs<sup>[<a href="#ref4" style="color:#520049">4</a></sup><sup>,<a href="#ref6" style="color:#520049">6</a></sup><sup>]</sup>. Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</font>
<div ><font ><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p></font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/GlnRs_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A',    
  start_residue_number:8, 
  end_residue_number:9, 
  color:{r:10,g:11,b:255}
},
{
  struct_asym_id: 'A', 
  start_residue_number:10, 
  end_residue_number:10, 
  color:{r:9,g:254,b:254}
},{
  struct_asym_id: 'A', 
  start_residue_number:11, 
  end_residue_number:12, 
  color:{r:10,g:11,b:255}
},
{
  struct_asym_id: 'A', 
  start_residue_number:13, 
  end_residue_number:13, 
  color:{r:9,g:254,b:254}
},
{
  struct_asym_id: 'A', 
  start_residue_number:14, 
  end_residue_number:312, 
  color:{r:10,g:11,b:255}
},
{
  struct_asym_id: 'A', 
  start_residue_number:313, 
  end_residue_number:313, 
  color:{r:9,g:254,b:254}
},
{
  struct_asym_id: 'A', 
  start_residue_number:314, 
  end_residue_number:315, 
  color:{r:10,g:11,b:255}
},
{
struct_asym_id: 'A', 
  start_residue_number:316, 
  end_residue_number:316, 
  color:{r:9,g:254,b:254}
},
{
struct_asym_id: 'A', 
  start_residue_number:317, 
  end_residue_number:320, 
  color:{r:10,g:11,b:255}
},
{
struct_asym_id: 'A', 
  start_residue_number:321, 
  end_residue_number:321, 
  color:{r:9,g:254,b:254}
},
{
struct_asym_id: 'A', 
  start_residue_number:322, 
  end_residue_number:322, 
  color:{r:10,g:11,b:255}
},
{
struct_asym_id: 'A', 
  start_residue_number:323, 
  end_residue_number:323, 
  color:{r:9,g:254,b:254}
},
{
struct_asym_id: 'A', 
  start_residue_number:324, 
  end_residue_number:547, 
  color:{r:10,g:11,b:255}
},
{
struct_asym_id: 'A', 
  start_residue_number:998, 
  end_residue_number:998, 
  color:{r:10,g:11,b:255}
},
{
struct_asym_id: 'B', 
  start_residue_number:902, 
  end_residue_number:909, 
  color:{r:255,g:135,b:10}
},
{
struct_asym_id: 'B', 
  start_residue_number:910, 
  end_residue_number:916, 
  color:{r:255,g:0,b:255}
},
{
struct_asym_id: 'B', 
  start_residue_number:918, 
  end_residue_number:943, 
  color:{r:255,g:135,b:10}
},
{
struct_asym_id: 'B', 
  start_residue_number:944, 
  end_residue_number:948, 
  color:{r:50,g:205,b:50}
},
{
struct_asym_id: 'B', 
  start_residue_number:949, 
  end_residue_number:976, 
  color:{r:255,g:135,b:10}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/GlnRs_aptamer_3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html></td>
</tr></table><br>
</div>


<p class="blowheader_box">Binding pocket</p>             
<font >Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 1EXD at 2.7 Å resolution. glutaminyl-tRNA synthetase (GlnRs) (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with GlnRs.</font>
<br><br>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/GlnRs_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/GlnRs_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br><br>





<font ><p class="header_boxx5" id="ligand-recognition">Ligand information</p></font>  
<p class="blowheader_box">SELEX ligand</p>
<font >Bullock T L et al. determined the binding constant of the aptamer was determined by a gel shift binding assay. Equal volumes of tRNA and GlnRS solutions were mixed and incubated for 15 min at ambient temperature. The mixtures were then loaded onto a 20% polyacrylamide gel at 4 ℃ and run at 200 V for 5 h. The gels were exposed for 72 h. Autoradiography and densitometry of the gels was performed with the Storm 840 phosphorimager (Molecular Dynamics). For AGGU and T1 tRNAs, the intensity of the shifted band was quantified to determine the relative amount of complex formed at each enzyme concentration. Equilibrium binding constants were determined by fitting this data to a standard hyperbolic curve using Kaleidagraph (Synergy Software)<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup>.</font >
<div style="display: flex; justify-content: center;">
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Wild type tRNA<sup>Gln</sup></td>
        <td name="td1">UGGGGUAUCGCCAAGCGGUAAGGCACCGGAUUCUGAUUCCGGCAUUCCGAGGUUCGAAUCCUCGUACCCCAGCCA</td>
        <td name="td3">GlnRs</td>
        <td name="td4">7.1nM</td>
      </tr>
      <tr>
        <td name="td0">44-AGGU-48 (var-AGGU)</td>
        <td name="td1">UGGGGUAUCGCCAAGCGGUAAGGCACCGGAUUCUGAUUCCGGAGGUCGAGGUUCGAAUCCUCGUACCCCAGCCA</td>
        <td name="td3">GlnRs</td>
        <td name="td4">0.27nM</td>
      </tr>
      <tr>
        <td name="td0">T1</td>
        <td name="td1">UGGGGUAUCGCCAAAAGAAAGGCACCGGAUUCUGAUUCCGGAGGUCGAGAUUGUCGUCCUCGUACCCCAGCCA</td>
        <td name="td3">GlnRs</td>
        <td name="td4">0.13nM</td>
      </tr>
	  </tbody>
  </table>
  </div>
<br>

<p class="blowheader_box">Structure ligand</p>
<font><p>Glutaminyl-tRNA synthetase (GlnRS) cataytic core domain. These enzymes attach Gln to the appropriate tRNA. Like other class I tRNA synthetases, they aminoacylate the 2'-OH of the nucleotide at the 3' end of the tRNA. The core domain is based on the Rossman fold and is responsible for the ATP-dependent formation of the enzyme bound aminoacyl-adenylate. GlnRS contains the characteristic class I HIGH and KMSKS motifs, which are involved in ATP binding. These enzymes function as monomers. Archaea and most bacteria lack GlnRS. In these organisms, the "non-discriminating" form of GluRS aminoacylates both tRNA(Glu) and tRNA(Gln) with Glu, which is converted to Gln when appropriate by a transamidation enzyme.-----from Pfam</p></font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">MW</th>
        <th onclick="sortTable(3)">Amino acids sequences</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P00962/publications" target="_blank" style="color:#520049"><b>P00962</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/cdd/CD00807/" target="_blank" style="color:#520049"><b>CD00807</b></a></td>
        <td name="td2">63.48 KDa</td>
        <td name="td3">MSEAEARPTNFIRQIIDEDLASGKHTTVHTRFPPEPNGYLHIGHAKSICLNFGIAQDYKGQCNLRFDDTNPVKEDIEYVESIKNDVEWLGFHWSGNVRYSSDYFDQLHAYAIELINKGLAYVDELTPEQIREYRGTLTQPGKNSPYRDRSVEENLALFEKMRAGGFEEGKACLRAKIDMASPFIVMRDPVLYRIKFAEHHQTGNKWCIYPMYDFTHCISDALEGITHSLCTLEFQDNRRLYDWVLDNITIPVHPRQYEFSRLNLEYTVMSKRKLNLLVTDKHVEGWDDPRMPTISGLRRRGYTAASIREFCKRIGVTKQDNTIEMASLESCIREDLNENAPRAMAVIDPVKLVIENYQGEGEMVTMPNHPNKPEMGSRQVPFSGEIWIDRADFREEANKQYKRLVLGKEVRLRNAYVIKAERVEKDAEGNITTIFCTYDADTLSKDPADGRKVKGVIHWVSAAHALPVEIRLYDRLFSVPNPGAADDFLSVINPESLVIKQGFAEPSLKDAVAGKAFQFEREGYFCLDSRHSTAEKPVFNRTVGLRDTWAKVGE</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1EXD" target="_blank" style="color:#520049"><b>1EXD</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/nuccore/V01575" target="_blank" style="color:#520049"><b>V01575</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/GlnRs_stru_ligand.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;">
  </div>


<p class="blowheader_box">Similar compound</p>
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">7WRS-A</td>
        <td name="td2">36.3</td>
        <td name="td3">2.8</td>
        <td name="td4">Glutamyl-tRNA synthetase</td>
      </tr>
      <tr>
        <td name="td1">4YE6-A</td>
        <td name="td2">34.1</td>
        <td name="td3">2.5</td>
        <td name="td4">Glutamine--tRNA ligase</td>
      </tr>
      <tr>
        <td name="td1">3A10-C</td>
        <td name="td2">21</td>
        <td name="td3">4</td>
        <td name="td4">Glutamyl-tRNA(gln) glutamate--tRNA ligase 1amidotransferase subunit A</td>
      </tr>
      <tr>
        <td name="td1">6B1P-A</td>
        <td name="td2">20.1</td>
        <td name="td3">3.4</td>
        <td name="td4">Glutamate--tRNA ligase 1</td>
      </tr>
      <tr>
        <td name="td1">1UOB-B</td>
        <td name="td2">14</td>
        <td name="td3">5.1</td>
        <td name="td4">Cysteinyl-tRNA synthetase</td>
      </tr>
      <tr>
        <td name="td1">3FNR-A</td>
        <td name="td2">12.7</td>
        <td name="td3">10</td>
        <td name="td4">Arginyl-tRNA synthetase</td>
      </tr>
      <tr>
        <td name="td1">LIRX-B</td>
        <td name="td2">12.6</td>
        <td name="td3">3.6</td>
        <td name="td4">Lysyl-tRNA synthetase</td>
      </tr>
      <tr>
        <td name="td1">LRQG-A</td>
        <td name="td2">10.5</td>
        <td name="td3">4.8</td>
        <td name="td4">Methionyl-tRNA synthetase</td>
      </tr>
      <tr>
        <td name="td1">4R3Z-B</td>
        <td name="td2">10.5</td>
        <td name="td3">6.9</td>
        <td name="td4">Aminoacyl tRNA synthase complex-interacting multi</td>
      </tr>
      <tr>
        <td name="td1">LPG2-A</td>
        <td name="td2">10.2</td>
        <td name="td3">5.2</td>
        <td name="td4">Methionyl-tRNA synthetase</td>
      </tr>
	  </tbody>
  </table>

<br><br>

                 
<p class="header_boxx6" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Overproduction and purification of Escherichia coli tRNA(2Gln) and its use in crystallization of the glutaminyl-tRNA synthetase-tRNA(Gln) complex.</strong></font><br />
Perona, J. J., Swanson, R., Steitz, T. A., & Söll, D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/2459391/" target="_blank" style="color:#520049" >Journal of molecular biology, 202(1), 121–126. (1988)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Structure of E. coli glutaminyl-tRNA synthetase complexed with tRNA(Gln) and ATP at 2.8 A resolution.</strong></font><br />
Rould, M. A., Perona, J. J., Söll, D., & Steitz, T. A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/2479982/" target="_blank" style="color:#520049">Science (New York, N.Y.), 246(4934), 1135–1142. (1989)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] In vitro selection and characterization of tRNA-like substrates of E. coli glutaminyl-tRNA synthetase.</strong></font><br />
Stubenrauch, M.<br />
<a href="https://www.proquest.com/openview/6857e35043f04e11f53714d80cdc4dcf/1?pq-origsite=gscholar&cbl=18750&diss=y" target="_blank" style="color:#520049"> Doctoral thesis, Department of Chemistry and Biochemistry, University of Colorado, Boulder, Colorado (1996)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Tertiary core rearrangements in a tight binding transfer RNA aptamer.</strong></font><br />
Bullock, T. L., Sherlin, L. D., & Perona, J. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10881199/" target="_blank" style="color:#520049">Nature structural biology, 7(6), 497–504. (2000)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Influence of transfer RNA tertiary structure on aminoacylation efficiency by glutaminyl and cysteinyl-tRNA synthetases.</strong></font><br />
Sherlin, L. D., Bullock, T. L., Newberry, K. J., Lipman, R. S., Hou, Y. M., Beijer, B., Sproat, B. S., & Perona, J. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10860750/" target="_blank" style="color:#520049">Journal of molecular biology, 299(2), 431–446. (2000)</a>
<br />

<a id="ref6"></a><font><strong>[6] Aptamer redesigned tRNA is nonfunctional and degraded in cells.</strong></font><br />
Lee, D., & McClain, W. H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/14681579/" target="_blank" style="color:#520049">RNA (New York, N.Y.), 10(1), 7–11. (2004)</a>
<br />       
