---
layout: post
title:  "HCV aptamer"
date:   2024-05-02 00:00:00
author: Jiali Wang, Fu Bo
categories: Aptamer
---
<!--p style="font-size: 24px"><strong>Keywords: </strong> <br/></p>
<font ><strong>small molecule, Synthetic, dyes, PubChem CID 25863</strong></font >
<br /-->

<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
<!--body>
  <p style="font-size: 16px">Click the buttons to navigate to different sections:</p>
  <div class="button-container">
    <a class="button" style="color:#520049" href="#description">Description</a>
    <a class="button" style="color:#520049" href="#SELEX">SELEX</a>
    <a class="button" style="color:#520049" href="#structure">Structure</a>
    <a class="button" style="color:#520049" href="#ligand-recognition">Ligand recognition
    <a class="button" style="color:#520049" href="#references">References</a>
   </a>
  </div>
</body-->
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
    .header_box {
     display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #c9c5c5;
      border-radius: 5px;
      width:225px;
	    height:50px;
  }
  .header_boxx1 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:115px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx2 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:140px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx3 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:85px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx4 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:120px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx5 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:215px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx6 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:135px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .blowheader_box{
    display: block;
      padding: 10px;
      font-size:18px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 5px;
      width:170px;
	    height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<br>
<br>


<font ><p class="header_boxx2" id="description">Description</p></font>
<font >In 2003,  Vo NV, Oh JW, Lai MM used the SELEX method to select the aptamer with high compatibility for the Hepatitis C virus NS5B protein. They found that the aptamers can act as templates for HCV polymerase, but others can interfere with the activity of the viral enzyme. These findings suggest that these RNA aptamers may be useful for further characterization of the binding properties of NS5B, and with further modifications, may have potential therapeutic value<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<br>
<br>


<p class="header_boxx3" id="SELEX">SELEX</p>
<p>In 2003,  Vo NV, Oh JW, Lai MM  utilized an RNA library estimated to contain 10<sub>12</sub> (one hundred trillion) distinct RNA species. Following 20 rounds of screening, the highest affinity RNA enrichment was achieved between rounds 18 and 20. Through the SELEX process, a total of 37 selected ligands were obtained, categorized into eight groups (A-H) based on sequence homology<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br>
<br>


<p class="header_boxx4" id="Structure">Structure</p>
<font>The 2D structure of the figures are based on the article by ribodraw tool to draw. R20-43 apatamer and R20-15 aptamer bind to hepatitis C virus NS5B protein (HCV NS5B protein)<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<font><p>R20-43 aptamer: 5'-GGGAGCUCAGAAUAAACGCUCAAUGUAGAGACUUUUCGAAAAGCGUGUUCCCAAUGAGUCUAUGCAUGCAUGAGGCCCGGAUCCGGC-3'<br>R20-15 aptamer: 5'-GGGAGCUCAGAAUAAACGCUCAAUGUAGGCUAGUACCGACCUGCGUGCCUCGCUCCUUUUCAUGCAUGCAUGAGGCCCGGAUCCGGC-3'</p></font>
<!--b>2D representation</b-->
        
<!--font><p>Szostak et al. obtained 17 different sequences through SELEX, and designed a 40nt sequence after comparing the information of 17 sequences. The aptamer of the 40nt is a sequence of analytic structures, and the following diagram shows its secondary structure.
Through SELEX, 17 different sequences were obtained, and after comparing 17 sequence information, a 40nt sequence was designed
 <sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p></font-->
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/HCV_aptamer_2D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/HCV_aptamer_2D2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
 <br>
                     
<!--b>3D visualisation</b><br>               
<font >ATP aptamer complex was generated from PDB ID:  at 3.05 Å . Additional available structures that have been solved and detailed information are accessible on <i>Structures</i> page <sup>[<a href="#ref3" style="color:#520049" >3</a>]</sup>.</font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/AMP-3D.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/AMP-3D-bp.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table-->
  <br>

<font ><p class="header_boxx5" id="ligand-information">Ligand information</p></font>  
  

<p class="blowheader_box">SELEX ligand</p>
<font>This family includes viral RNA dependent RNA polymerase enzymes from hepatitis C virus and various plant viruses.-----from Pfam</font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Hepatitis C virus NS5B protein (HCV NS5B protein)</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P26663/entry#function" target="_blank" style="color:#520049"><b>P26663</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00998/" target="_blank" style="color:#520049"><b>PF00998</b></a></td>
        <td name="td3">58.978 kDa</td>
        <td name="td4">SMSYTWTGALITPCAAEESKLPINALSNSLLRHHNMVYATTSRSAGLRQKKVTFDRLQVLDDHYRDVLKEMKAKASTVKAKLLSVEEACKLTPPHSAKSKFGYGAKDVRNLSSKAVNHIHSVWKDLLEDTVTPIDTTIMAKNEVFCVQPEKGGRKPARLIVFPDLGVRVCEKMALYDVVSTLPQVVMGSSYGFQYSPGQRVEFLVNTWKSKKNPMGFSYDTRCFDSTVTENDIRVEESIYQCCDLAPEARQAIKSLTERLYIGGPLTNSKGQNCGYRRCRASGVLTTSCGNTLTCYLKASAACRAAKLQDCTMLVNGDDLVVICESAGTQEDAASLRVFTEAMTRYSAPPGDPPQPEYDLELITSCSSNVSVAHDASGKRVYYLTRDPTTPLARAAWETARHTPVNSWLGNIIMYAPTLWARMILMTHFFSILLAQEQLEKALDCQIYGACYSIEPLDLPQIIERLHGLSAFSLHSYSPGEINRVASCLRKLGVPPLRVWRHRARSVRARLLSQGGRAATCGKYLFNWAVK</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1CSJ" target="_blank" style="color:#520049"><b>1CSJ</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/nuccore/M58335" target="_blank" style="color:#520049"><b>M58335</b></a></td>
      </tr>
	  </tbody>
  </table>
<font>Some isolated sequences bind to the affinity of the protein.</font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
      <td name="td0">R20-43</td>
      <td name="td1">5'-GGGAGCUCAGAAUAAACGCUCAAUGUAGAGACUUUUCGAAAAGCGUGUUCCCAAUGAGUCUAUGCAUGCAUGAGGCCCGGAUCCGGC-3'</td>
      <td name="td2">HCV NS5B protein</td>
      <td name="td3">(1.4 ± 0.3)* 10<sup>4</sup> (1/s) （dissociation rate constant）</td>
    </tr>
     <tr>
      <td name="td0">R20-15</td>
      <td name="td1">5'-GGGAGCUCAGAAUAAACGCUCAAUGUAGGCUAGUACCGACCUGCGUGCCUCGCUCCUUUUCAUGCAUGCAUGAGGCCCGGAUCCGGC-3'</td>
      <td name="td2">HCV NS6B protein</td>
      <td name="td3">(1.0 ± 0.1)* 10<sup>4</sup> (1/s)</td>
    </tr>
     <tr>
      <td name="td0">R1-1</td>
      <td name="td1">5'-GGGAGCUCAGAAUAAACGCUCAACUCCUAACCUCUCCGCAUGNGCUGUUACCUACACAGCUCAGCAUGCAUGAGGCCCGGAUCCGGC-3'</td>
      <td name="td2">HCV NS7B protein</td>
      <td name="td3">(100 ± 6)* 10<sup>4</sup> (1/s)</td>
    </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/HCV_SELEX_ligand.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>



<p class="blowheader_box">Similar compound</p>                    
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">3GNW-B</td>
      <td name="td1">51.7</td>
      <td name="td2">1.1</td>
      <td name="td3">RNA-directed RNA polymerase</td>
    </tr>
     <tr>
      <td name="td0">5Y6R-A</td>
      <td name="td1">30.3</td>
      <td name="td2">3.3</td>
      <td name="td3">Genome polyprotein</td>
    </tr>
     <tr>
      <td name="td0">4K6M-A</td>
      <td name="td1">27.2</td>
      <td name="td2">3.1</td>
      <td name="td3">Polyprotein</td>
    </tr>
     <tr>
      <td name="td0">1SH0-B</td>
      <td name="td1">24.5</td>
      <td name="td2">4.0</td>
      <td name="td3">RNA polymerase</td>
    </tr>
     <tr>
      <td name="td0">8WU3-B</td>
      <td name="td1">23.5</td>
      <td name="td2">3.3</td>
      <td name="td3">RNA polymerase</td>
    </tr>
     <tr>
      <td name="td0">2IJD-1</td>
      <td name="td1">23.0</td>
      <td name="td2">3.5</td>
      <td name="td3">Picornain 3c, RNA-directed RNA polymerase</td>
    </tr>
     <tr>
      <td name="td0">5I61-A</td>
      <td name="td1">20.5</td>
      <td name="td2">3.4</td>
      <td name="td3">Potential RNA-dependent RNA polymerase</td>
    </tr>
     <tr>
      <td name="td0">6OGZ-A</td>
      <td name="td1">18.4</td>
      <td name="td2">4.0</td>
      <td name="td3">Rotavirus RNA-dependent RNA polymerase</td>
    </tr>
     <tr>
      <td name="td0">8GWE-A</td>
      <td name="td1">17.7</td>
      <td name="td2">3.8</td>
      <td name="td3">RNA-directed RNA polymerase</td>
    </tr>
     <tr>
      <td name="td0">8H69-B</td>
      <td name="td1">17.6</td>
      <td name="td2">3.9</td>
      <td name="td3">Cryo-EM structure of influenza RNA polymerase</td>
    </tr>
    </tbody>
  </table>
<p><br /></p>

                 
<p class="header_boxx6" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Identification of RNA ligands that bind hepatitis C virus polymerase selectively and inhibit its RNA synthesis from the natural viral RNA templates.</strong></font><br />
Vo, N. V., Oh, J. W., & Lai, M. M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12667800/" target="_blank" style="color:#520049" >Virology, 307(2), 301–316. (2003)</a>
<br/>