---
layout: post
title:  "HIV-1 PR aptamer"
date:   2024-05-06 00:00:00
author: Jiali Wang, Chen Ke
categories: Aptamer
---
<!--p style="font-size: 24px"><strong>Keywords: </strong> <br/></p>
<font ><strong>small molecule, Synthetic, dyes, PubChem CID 25863</strong></font >
<br /-->

<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
<!--body>
  <p style="font-size: 16px">Click the buttons to navigate to different sections:</p>
  <div class="button-container">
    <a class="button" style="color:#520049" href="#description">Description</a>
    <a class="button" style="color:#520049" href="#SELEX">SELEX</a>
    <a class="button" style="color:#520049" href="#structure">Structure</a>
    <a class="button" style="color:#520049" href="#ligand-recognition">Ligand recognition
    <a class="button" style="color:#520049" href="#references">References</a>
   </a>
  </div>
</body-->
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
    .header_box {
     display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #c9c5c5;
      border-radius: 5px;
      width:225px;
	    height:50px;
  }
  .header_boxx1 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:115px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx2 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:140px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx3 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:85px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx4 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:120px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx5 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:215px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx6 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:135px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .blowheader_box{
    display: block;
      padding: 10px;
      font-size:18px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 5px;
      width:170px;
	    height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<br>
<br>


<font ><p class="header_boxx2" id="description">Description</p></font>
<font >In 2015, Duclair S, Gautam A, Ellington A, et al. selexed several RNA aptamers targeting the human immunodeficiency virus type 1 (HIV-1) protease (PR) using a method called Systematic Evolution of Ligands by Exponential Enrichment (SELEX). The function of these aptamers is to bind to HIV-1 PR, inhibit its activity, and thus prevent the replication cycle of the HIV virus. Research has shown that these RNA aptamers have high affinity and specificity, and can non competitively inhibit the protease activity of HIV-1 PR. Through further mutation analysis, researchers identified the secondary structural elements of the ligands crucial for PR binding and demonstrated their ability to inhibit HIV replication in vitro<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<br>
<br>


<p class="header_boxx3" id="SELEX">SELEX</p>
<p>In 2015, Duclair S, Gautam A, Ellington A, et al. used an RNA library containing 10<sub>14</sub> unique species for screening, and improved their binding ability to HIV-1 PR through nine rounds of SELEX process. After the 10th round of selection, 48 clones were sequenced and four different sequence families were discovered. In order to improve the binding affinity of the first generation RNA adapter, researchers screened variants from a newly generated partially randomized (dosed) pool and conducted an additional eight rounds of selection. Several variants were identified from the 8th round pool of PR10.1 and PR10.9 RNA adapters, and these second-generation PR RNA adapters showed significantly improved binding affinity. Through site-specific mutation analysis, researchers investigated the predicted secondary structure of the second-generation PR10.1-8E RNA adapter to identify structural elements crucial for PR binding<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br>
<br>


<p class="header_boxx4" id="Structure">Structure</p>
<font>The 2D structure of the figure is based on the article by ribodraw tool to draw. PR10.1-8E aptamer binds to HIV-1 aspartyl protease (PR)<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<font><p>5'-GGGUUACCUAGGAGGAGAUGCUCUUAAGUGUAACUUCUCAUAACAUCCGGAGGCUUUUACUUCCGGGGACCUAAGUGACGUCUGAACUGCUUCGAA-3'</p></font>
<!--b>2D representation</b-->
        
<!--font><p>Szostak et al. obtained 17 different sequences through SELEX, and designed a 40nt sequence after comparing the information of 17 sequences. The aptamer of the 40nt is a sequence of analytic structures, and the following diagram shows its secondary structure.
Through SELEX, 17 different sequences were obtained, and after comparing 17 sequence information, a 40nt sequence was designed
 <sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p></font-->
<img src="/images/2D/PR10_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;">   
</div>
 <br>
                     
<!--b>3D visualisation</b><br>               
<font >ATP aptamer complex was generated from PDB ID:  at 3.05 Å . Additional available structures that have been solved and detailed information are accessible on <i>Structures</i> page <sup>[<a href="#ref3" style="color:#520049" >3</a>]</sup>.</font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/AMP-3D.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/AMP-3D-bp.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table-->
  <br>

<font ><p class="header_boxx5" id="ligand-information">Ligand information</p></font>  
  

<p class="blowheader_box">SELEX ligand</p>
<font>Aspartyl protease that mediates proteolytic cleavages of Gag and Gag-Pol polyproteins during or shortly after the release of the virion from the plasma membrane. Cleavages take place as an ordered, step-wise cascade to yield mature proteins. This process is called maturation. Displays maximal activity during the budding process just prior to particle release from the cell. Also cleaves Nef and Vif, probably concomitantly with viral structural proteins on maturation of virus particles. Hydrolyzes host EIF4GI and PABP1 in order to shut off the capped cellular mRNA translation. The resulting inhibition of cellular protein synthesis serves to ensure maximal viral gene expression and to evade host immune response. Also mediates cleavage of host YTHDF3. Mediates cleavage of host CARD8, thereby activating the CARD8 inflammasome, leading to the clearance of latent HIV-1 in patient CD4 T-cells after viral reactivation; in contrast, HIV-1 can evade CARD8-sensing when its protease remains inactive in infected cells prior to viral budding (By similarity).-----from Uniprot</font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">HIV-1 protease (PR)</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P03366/entry" target="_blank" style="color:#520049"><b>P03366</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00077/" target="_blank" style="color:#520049"><b>PF00077</b></a></td>
        <td name="td3">10.792 kDa</td>
        <td name="td4">PQITLWQRPLVTIKIGGQLKEALLDTGADDTVLEEMSLPGRWKPKMIGGIGGFIKVRQYDQILIEICGHKAIGTVLVGPTPVNIIGRNLLTQIGCTLNF</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/4ZIP" target="_blank" style="color:#520049"><b>4ZIP</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/nuccore/AH002345" target="_blank" style="color:#520049"><b>AH002345</b></a></td>
      </tr>
	  </tbody>
  </table>
<font>Some isolated sequences bind to the affinity of the protein.</font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
      <td name="td0">PR10.1-8E</td>
      <td name="td1">5'-GGGUUACCUAGGAGGAGAUGCUCUUAAGUGUAACUUCUCAUAACAUCCGGAGGCUUUUACUUCCGGGGACCUAAGUGACGUCUGAACUGCUUCGAA-3'</td>
      <td name="td2">HIV-1 protease (PR)</td>
      <td name="td3">17 ± 3 nM</td>
    </tr>
     <tr>
      <td name="td0">PR10.1-8A</td>
      <td name="td1">5'-GGGUUACCUAGGAGGAGAUGCUCUUAAGUGUAACUUCUCGUAAUUCCCAAGGCUUUUACCUCGGGGUCCUAAGUGACGUCUGAACUGCUUCGAA-3'</td>
      <td name="td2">HIV-1 protease (PR)</td>
      <td name="td3">2.2 ± 0.2 nM</td>
    </tr>
     <tr>
      <td name="td0">PR10.1</td>
      <td name="td1">5'-GGGUUACCUAGGAGGAGAUGCUCUUCAUUGUAACUUCUCAUAAUUUCCCGAGGCUUUUACUUUCGGGGUCCUAAGUGACGUCUGAACUGCUUCGAA-3'</td>
      <td name="td2">HIV-1 protease (PR)</td>
      <td name="td3">115 ± 22 nM</td>
    </tr>
     <tr>
      <td name="td0">PR10.9</td>
      <td name="td1">5'-GGGUUACCUAGGAGGAGAUGCUACAUUACCUAAGUAAGAUUACGGCUUCGAGUUUAGAGACCUCUCCCUGGUAAGUGACGUCUGAACUGCUUCGAA-3'</td>
      <td name="td2">HIV-1 protease (PR)</td>
      <td name="td3">93 ± 19 nM</td>
    </tr>
     <tr>
      <td name="td0">PR10.13</td>
      <td name="td1">5'-GGGUUACCUAGGAGGAGAUGCUCCGGGUCGUCCCCUACGGGGACCUAAAGACUGUGUCCAACCGCCCUCGCCUAAGUGACGUCUGAACUGCUUCGAA-3'</td>
      <td name="td2">HIV-1 protease (PR)</td>
      <td name="td3">154 ± 13 nM</td>
    </tr>
     <tr>
      <td name="td0">PR10.18</td>
      <td name="td1">5'-GGGUUACCUAGGAGGAGAUGCUUCAGACAUUUACCUCACUUCGUCUGUUCAAUCGGGUAACACUCGGGAUGAAAGUGACGUCUGAACUGCUUCGAA-3'</td>
      <td name="td2">HIV-1 protease (PR)</td>
      <td name="td3">140 ± 17 nM</td>
    </tr>
     <tr>
      <td name="td0">PR10.9-8E</td>
      <td name="td1">5'-GGGUUACCUAGGAGGAGAUGCUACAUUACCUAGGUAAGAUAACGGCUUCGCGUUCAGAGACCUCUCCCUGGUAAGUGACGUCUGAACUGCUUCGAA-3'</td>
      <td name="td2">HIV-1 protease (PR)</td>
      <td name="td3">22 ± 9 nM</td>
    </tr>
     <tr>
      <td name="td0">PR10.9-8N</td>
      <td name="td1">5'-GGGUUACCUAGGAGGAGAUGCUUUGACCUAAGGUAAGAUAACGGCUUCGAGUUCAGAGACCUCGCCCUGGUAAGUGACGUCUGAACUGCUUCGAA-3'</td>
      <td name="td2">HIV-1 protease (PR)</td>
      <td name="td3">4.3 ± 0.2 nM</td>
    </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/PR10_SELEX_ligand.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>                    
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">2O40-A</td>
      <td name="td1">17.9</td>
      <td name="td2">0.5</td>
      <td name="td3">Covalent dimer HIV-1 protease</td>
    </tr>
     <tr>
      <td name="td0">1B11-A</td>
      <td name="td1">15.4</td>
      <td name="td2">1.4</td>
      <td name="td3">Protein (feline immunodeficiency virus protease)</td>
    </tr>
     <tr>
      <td name="td0">1FMB-A</td>
      <td name="td1">14.8</td>
      <td name="td2">1.7</td>
      <td name="td3">Eiav protease</td>
    </tr>
     <tr>
      <td name="td0">1BAI-A</td>
      <td name="td1">14.1</td>
      <td name="td2">1.9</td>
      <td name="td3">Protease</td>
    </tr>
     <tr>
      <td name="td0">3LIN-B</td>
      <td name="td1">13.3</td>
      <td name="td2">2.0</td>
      <td name="td3">Protease</td>
    </tr>
     <tr>
      <td name="td0">6S1V-B</td>
      <td name="td1">12.5</td>
      <td name="td2">2.4</td>
      <td name="td3">Gag-pro-pol polyprotein</td>
    </tr>
     <tr>
      <td name="td0">5C9F-A</td>
      <td name="td1">10.1</td>
      <td name="td2">2.8</td>
      <td name="td3">Aprick protease</td>
    </tr>
     <tr>
      <td name="td0">3NR6-A</td>
      <td name="td1">9.6</td>
      <td name="td2">2.5</td>
      <td name="td3">Protease P14</td>
    </tr>
     <tr>
      <td name="td0">4Z2Z-B</td>
      <td name="td1">9.2</td>
      <td name="td2">2.5</td>
      <td name="td3">DNA damage-inducible protein 1</td>
    </tr>
     <tr>
      <td name="td0">7KSE-A</td>
      <td name="td1">8.7</td>
      <td name="td2">2.7</td>
      <td name="td3">Peptidase a9/Reverse transcriptase/Rnase H</td>
    </tr>
    </tbody>
  </table>
<p><br /></p>

                 
<p class="header_boxx6" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] High-affinity RNA Aptamers Against the HIV-1 Protease Inhibit Both In Vitro Protease Activity and Late Events of Viral Replication.</strong></font><br />
Duclair, S., Gautam, A., Ellington, A., & Prasad, V. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25689224/" target="_blank" style="color:#520049" >Molecular therapy. Nucleic acids, 4(2), e228. (2015)</a>
<br/>