---
layout: post
title:  "HIV-1 REV peptide apatmer"
date:   2024-04-8 00:00:00
author: Ziyu Guo, Zhizhong Lu
categories: Aptamer
---
<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
<!--body>
  <p style="font-size: 16px">Click the buttons to navigate to different sections:</p>
  <div class="button-container">
    <a class="button" style="color:#520049" href="#timeline">Timeline</a>
    <a class="button" style="color:#520049" href="#description">Description</a>
    <a class="button" style="color:#520049" href="#SELEX">SELEX</a>
    <a class="button" style="color:#520049" href="#structure">Structure</a>
    <a class="button" style="color:#520049" href="#ligand-recognition">Ligand recognition
    <a class="button" style="color:#520049" href="#references">References</a>
   </a>
  </div>
</body-->
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
  .header_box {
     display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #c9c5c5;
      border-radius: 5px;
      width:225px;
	    height:50px;
  }
  .header_boxx1 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:115px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx2 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:140px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx3 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:85px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx4 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:120px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx5 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:215px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx6 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:135px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .blowheader_box{
    display: block;
      padding: 10px;
      font-size:18px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 5px;
      width:170px;
	    height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<p><br /></p>


<p class="header_boxx1" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8289245/" target="_blank" style="color:#520049">1994</a></h3>
    </div>
    <div class="body">
      <p>Jensen KB and Green L produced small RNA ligands with high affinity for HIV-1 Rev protein by SELEX in vitro evolution method for a series of chemical modification studies<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8946856/" target="_blank" style="color:#520049">1996</a></h3>
    </div>
    <div class="body">
      <p>The solution structure of the 35-mer high affinity RNA aptamer binding site of HIV-1 17-mer rev peptide was determined by NMR molecular dynamics<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10467126/" target="_blank" style="color:#520049">1999</a></h3>
    </div>
    <div class="body">
      <p>The study showed that the RNA structure determines whether the same HIV-1 Rev peptide folds into an extended conformation or an alpha-helical conformation at the time of complex formation<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10647177/" target="_blank" style="color:#520049">1999</a></h3>
    </div>
    <div class="body">
      <p>The solution structure of HTLV-1 arginine-rich Rex peptide bound to its RNA aptamer target as determined by multidimensional heteronuclear NMR was demonstrated<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div> <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/18922466/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>It was demonstrated that HIV Rev, which is limited by the limited viral genome, can also form organized RNP by assembling homologous oligomers on Rev reaction element (RRE) RNA<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/23972852/" target="_blank" style="color:#520049">2013</a></h3>
    </div>
    <div class="body">
      <p>The arginine-rich RNA-binding motif of HIV-1 Rev is intrinsically disordered and folds upon RRE binding<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25486594/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>Rev-RNA recognition relies on well-characterized sequence-specific contact at the IIB site and local RNA structure at the second site. The structure supports a model in which RRE utilizes the inherent plasticity of the Rev subunit interface to guide the formation of functional complexes<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
</div>


         
<font ><p class="header_boxx2" id="description">Description</p></font>
<font >In 1994, Jensen KB and Green L produced small RNA ligands with high affinity for HIV-1 Rev protein by SELEX in vitro evolution method for a series of chemical modification studies. In 1996, Ye X and Gorin A combined NMR molecular dynamics method have been applied to determine the solution structure of the HIV-1 17-mer rev peptide binding to its 35-mer high affinity RNA aptamer binding site<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.<br></font>
<p><br /></p> <br>

<p class="header_boxx3" id="SELEX">SELEX</p>
<p>In 1994, Jensen KB and Green L produced small RNA ligands with high affinity for HIV-1 Rev protein by SELEX in vitro evolution method for a series of chemical modification studies<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br> <br>    



<p class="header_boxx4" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<font><p>Here we use ribodraw to complete the figure, through the 3D structure information<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p></font>
<font><p>5'-GGCUGGACUCGUACUUCGGUACUGGAGAAACAGCC-3'</p></font>
<img src="/images/2D/Rev_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;">   
</div>



<p class="blowheader_box">3D visualisation</p>             
<font >Ye X, and Gorin A et al determined the solution structure of the 35-mer high-affinity RNA aptamer binding site of HIV-1 17-mer rev peptide by NMR molecular dynamics. The PDB ID of this structure is 1ULL<sup>[<a href="#ref2" style="color:#520049" >2</a>]</sup>. Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</font>
<div ><font ><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p></font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/Rev_RNA_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
struct_asym_id: 'A',
start_residue_number:1,
end_residue_number:21,
color:{r:128,g:219,b:158}
},
{
struct_asym_id: 'A',
start_residue_number:22,
end_residue_number:23,
color:{r:255,g:0,b:255}
},
{
struct_asym_id: 'A',
start_residue_number:24,
end_residue_number:35,
color:{r:128,g:219,b:158}
},
{
struct_asym_id: 'B',
start_residue_number:1,
end_residue_number:17,
color:{r:255,g:135,b:10}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1ull.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html></td>
</tr></table><br>
</div>


<p class="blowheader_box">Binding pocket</p>             
<font >Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 1ULL. HIV-1 REV peptide (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with HIV-1 REV peptide.</font>
<br><br>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Rev_RNA_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Rev_RNA_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br><br>






  <br><br>




<font ><p class="header_boxx5" id="ligand-recognition">Ligand information</p></font>  
  
<p class="blowheader_box">SELEX ligand</p>
<font >Jensen KB and Green L Select rna that is chemically modified under denaturation conditions and assign it through a filter for Rev binding. Selected at Rev concentrations of 30, 6, and 1 "2 nM (binding buffer volumes of 1.5 and 25 ml, respectively), 200 mM potassium acetate, 50mM Tris 'hci (pH 7" 7), 10 mM I)TT. About 3pmo of modified RNA was added to each protein solution, mixed and stored in ice or 15 rain, then transferred to 37o(' 10 rain). The binding solution is flushed with 5ml of binding buffer through the pre-wet nitrocellulose filter. The RNA is eluted from the membrane and the remaining modification sites are detected. The modified RNA is also labeled on a filter and elutes to check for uniform recovery of the modified RNA<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</font >
<br><br>
<div style="display: flex; justify-content: center;">
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">6a aptamer</td>
        <td name="td1">5'-GGGUGALUGAGAAACACGUUUGUGGACUCUGAUCU-3'</td>
        <td name="td3">REV peptide</td>
        <td name="td4">100nM</td>
      </tr>
	  </tbody>
  </table>
  </div>
<br>

<p class="blowheader_box">Structure ligand</p>
<font><p>This domain family spans seven repeats of glycoprotein (gp)/ transmembrane subunits of various endogenous retroviruses (ERVs) and infectious retroviruses, including human, monkey, and feline immunodeficiency viruses (HIV, SIV, and FIV), bovine immunodeficiency like viruses (BIV), equine infectious anemia virus (EIAV), and other viruses. Jaagsiekte Sheep retrovirus (JSRV), mouse mammary tumor virus (MMTV) and various ERVs, including sheep enJSRV-26 and human erv (herv). HERV-K_c1q23.3 and HERV-K_c12q14.1. This domain belongs to a larger superfamily that contains the HR1-HR2 domains of erv and infectious retroviruses, including Ebola virus and Rous sarcoma virus.</p></font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">MW</th>
        <th onclick="sortTable(3)">Amino acids sequences</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P69718/entry" target="_blank" style="color:#520049"><b>P69718</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/cdd/CD09909/" target="_blank" style="color:#520049"><b>CD09909</b></a></td>
        <td name="td2">13.05 kDa</td>
        <td name="td3">MAGRSGDSDEDLLKAVRLIKFLYQSNPPPNPEGTRQARRNRRRRWRERQRQIHSISERILSTYLGRSAEPVPLQLPPLERLTLDCNEDCGTSGTQGVGSPQILVESPTILESGAKE</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1RPV" target="_blank" style="color:#520049"><b>1RPV</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/nuccore/M14100" target="_blank" style="color:#520049"><b>M14100</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
<img src="/images/Structure_ligand/Rev_aptamer_stru_ligand.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;">
  </div>


<p class="blowheader_box">Similar compound</p>
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">TM-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">2LD2</td>
        <td name="td2">0.358</td>
        <td name="td3">0.89</td>
        <td name="td4">Solution structure of the N-terminal domain of huntingtin (htt17) in presence of DPC micelles</td>
      </tr>
      <tr>
        <td name="td1">1DNG</td>
        <td name="td2">0.442</td>
        <td name="td3">1.68</td>
        <td name="td4">NMR structure of a model hydrophilic amphipathic helical acidic peptide</td>
      </tr>
      <tr>
        <td name="td1">3N95-E</td>
        <td name="td2">0.034</td>
        <td name="td3">1.7</td>
        <td name="td4">Crystal structure of human CRFR2 alpha extracellular domain in complex with Urocortin 2</td>
      </tr>
      <tr>
        <td name="td1">1OMQ</td>
        <td name="td2">0.366</td>
        <td name="td3">1.1</td>
        <td name="td4">Structure of penetratin in bicellar solution</td>
      </tr>
      <tr>
        <td name="td1">7LSO</td>
        <td name="td2">0.44</td>
        <td name="td3">0.67</td>
        <td name="td4">L-Phenylseptin</td>
      </tr>
      <tr>
        <td name="td1">2RLG</td>
        <td name="td2">0.34</td>
        <td name="td3">0.79</td>
        <td name="td4">NMR structure of the antimicrobial peptide RP-1 bound to SDS micelles</td>
      </tr>
      <tr>
        <td name="td1">3N93-E</td>
        <td name="td2">0.03</td>
        <td name="td3">1.62</td>
        <td name="td4">Crystal structure of human CRFR2 alpha extracellular domain in complex with Urocortin 3</td>
      </tr>
      <tr>
        <td name="td1">7LSP</td>
        <td name="td2">0.465</td>
        <td name="td3">0.56</td>
        <td name="td4">D-Phenylseptin - The second residue of PHE of the peptide is a D-amino acid</td>
      </tr>
      <tr>
        <td name="td1">1O53</td>
        <td name="td2">0.371</td>
        <td name="td3">0.49</td>
        <td name="td4">Solution structure of the N-terminal membrane anchor of E. coli enzyme IIA(Glucose)</td>
      </tr>
      <tr>
        <td name="td1">6GS9</td>
        <td name="td2">0.303</td>
        <td name="td3">1.36</td>
        <td name="td4">NMR structure of aurein 2.5 in SDS micelles</td>
      </tr>
	  </tbody>
  </table>

<br><br>

                 
<p class="header_boxx6" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Characterization of an in vitro-selected RNA ligand to the HIV-1 Rev protein.</strong></font><br />
Jensen KB, Green L, MacDougal-Waugh S, Tuerk C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8289245/" target="_blank" style="color:#520049" >J Mol Biol, 1994, 235(1), 237-247 (1994)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Deep penetration of an alpha-helix into a widened RNA major groove in the HIV-1 rev peptide-RNA aptamer complex.</strong></font><br />
Ye X, Gorin A, Ellington AD, Patel DJ<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8946856/" target="_blank" style="color:#520049">Nat Struct Biol, 1996, 3(12):1026-1033. (1996)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] RNA architecture dictates the conformations of a bound peptide.</strong></font><br />
Ye, X., Gorin, A., Frederick, R., Hu, W., Majumdar, A., Xu, W., McLendon, G., Ellington, A., & Patel, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10467126/" target="_blank" style="color:#520049"> Chem Biol, 1999;6(9), 657-669 (1999)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Anchoring an extended HTLV-1 Rex peptide within an RNA major groove containing junctional base triples.</strong></font><br />
Jiang, F., Gorin, A., Hu, W., Majumdar, A., Baskerville, S., Xu, W., Ellington, A., & Patel, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10647177/" target="_blank" style="color:#520049">Structure, 1999,7(12),1461-1472 (1999)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] A solution to limited genomic capacity: using adaptable binding surfaces to assemble the functional HIV Rev oligomer on RNA.</strong></font><br />
Daugherty MD, D'Orso I, Frankel AD.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18922466/" target="_blank" style="color:#520049">Mol Cell, 2008, 31(6), 824-834. (2008)</a>
<br />
                
<a id="ref6"></a><font><strong>[6] The arginine-rich RNA-binding motif of HIV-1 Rev is intrinsically disordered and folds upon RRE binding.</strong></font><br />
Casu F, Duggan BM, Hennig M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23972852/" target="_blank" style="color:#520049" >Biophys J, 2013, 105(4), 1004-1017 (2013)</a>
<br />

<a id="ref7"></a><font><strong>[7] RNA-directed remodeling of the HIV-1 protein Rev orchestrates assembly of the Rev-Rev response element complex.</strong></font><br />
Jayaraman B, Crosby DC, Homer C, Ribeiro I, Mavor D, Frankel AD.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25486594/" target="_blank" style="color:#520049" >Elife, 2014,3:e04120. (2014)</a>
<br />