---
layout: post
title:  "IgG aptamer"
date:   2024-04-18 00:00:00
author: Zhizhong Lu, Ying Ao
categories: Aptamer
---
<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
<!--body>
  <p style="font-size: 16px">Click the buttons to navigate to different sections:</p>
  <div class="button-container">
    <a class="button" style="color:#520049" href="#timeline">Timeline</a>
    <a class="button" style="color:#520049" href="#description">Description</a>
    <a class="button" style="color:#520049" href="#SELEX">SELEX</a>
    <a class="button" style="color:#520049" href="#structure">Structure</a>
    <a class="button" style="color:#520049" href="#ligand-recognition">Ligand recognition
    <a class="button" style="color:#520049" href="#references">References</a>
   </a>
  </div>
</body-->
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
  .header_box {
     display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #c9c5c5;
      border-radius: 5px;
      width:225px;
	    height:50px;
  }
  .header_boxx1 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:115px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx2 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:140px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx3 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:85px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx4 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:120px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx5 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:215px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx6 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:135px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .blowheader_box{
    display: block;
      padding: 10px;
      font-size:18px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 5px;
      width:170px;
	    height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<p><br /></p>


<p class="header_boxx1" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/18441054/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>The aptamer is initially acquired through the SELEX<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/18931441/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>RNA aptamer complex with the Fc fragment of human IgG1, X-ray diffraction studies revealed 2.2 A resolution<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/20675355/" target="_blank" style="color:#520049">2010</a></h3>
    </div>
    <div class="body">
      <p>Determined the 2.15 Å crystal structure of the Fc fragment of human IgG1 (hFc1) complexed with an anti-Fc RNA aptamer<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/22924890/" target="_blank" style="color:#520049">2012</a></h3>
    </div>
    <div class="body">
      <p>Reported a lightly cross-linked (2%) N-isopropyl acrylamide (NIPAm) synthetic polymer NP (50-65 nm) incorporating hydrophobic and carboxylate groups that binds with high affinity to the Fc fragment of IgG<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/23661463/" target="_blank" style="color:#520049">2013</a></h3>
    </div>
    <div class="body">
      <p>Efficiently select and identify novel DNA aptamers against the Fc region of mouse IgG by EXponential enrichment (SELEX)<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
</div>


         
<font ><p class="header_boxx2" id="description">Description</p></font>
<font >In 2008, Yamazaki, S., & Nakamura, Y. RNA aptamers against the Fc portion of human IgG1 subclass (hIgG1-Fc) were selected from a library of RNA sequences by modified SELEX with 2′-fluoro pyrimidines. In 2008, Nakamura, Y., & Matsumura, H. solved a crystal structure of an RNA aptamer bound to hIgG1-Fc，diffracted X-rays to 2.2 Å resolution. moreover, In 2012, they determined the 2.15 Å crystal structure of the Fc fragment of human IgG1 (hFc1) complexed with an anti-Fc RNA aptamer<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>,<a href="#ref3" style="color:#520049">3</a></sup><sup>]</sup>.<br></font>
<p><br /></p> <br>

<p class="header_boxx3" id="SELEX">SELEX</p>
<p>In 2008, Yamazaki, S., & Nakamura, Y. RNA aptamers against the Fc portion of human IgG1 subclass (hIgG1-Fc) were selected from a library of 5 × 10<sup>14</sup> different RNA molecules randomized over 40 nucleotides (40N RNA pool) by modified SELEX with 2′-fluoro pyrimidines. After 10 rounds of selection, we obtained six sets of RNA sequences of 71–74 nucleotides (nt) long, Apt1 through Apt6<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br> <br>    



<p class="header_boxx4" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<font><p>Here we use ribodraw to complete the figure, through the 3D structure information. Apt8 aptamer was the aptamer sequence studied in SELEX article<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.</p></font>
<font><p>5'-GGGAACAAAGCUGAAGUACUUACCC-3'</p></font>
<img src="/images/2D/IgG_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;">   
</div>
<br>


<p class="blowheader_box">3D visualisation</p>             
<font >Nakamura, Y., & Matsumura, H. determined the crystal structure of Apt8-2 aptamer bound to the Fc fragment of human IgG1 at 2.15 Å resolution. The PDB ID of this structure is 3AGV. The final model, an asymmetric unit including homodimeric hFc1 residues 241–262, 272–296, 302–321 and 334–443 for chain A and 237–444 for chain B, two RNA aptamer molecules (G1-C11 and G16-C23 and G1-A24), two carbohydrate chains , two Ca2+ ions, and 254 water molecules<sup>[<a href="#ref3" style="color:#520049">3</a></sup><sup>]</sup>. Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</font>
<div ><font ><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p></font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/IgG_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A', 
  start_residue_number:241, 
  end_residue_number:443, 
  color:{r:255,g:192,b:203}
},
{
  struct_asym_id: 'B', 
  start_residue_number:237, 
  end_residue_number:444, 
  color:{r:255,g:192,b:203}
},{
  struct_asym_id: 'C', 
  start_residue_number:1, 
  end_residue_number:4, 
  color:{r:255,g:192,b:203}
},
{
  struct_asym_id: 'D', 
  start_residue_number:1, 
  end_residue_number:8, 
  color:{r:255,g:192,b:203}
},
{
  struct_asym_id: 'S', 
  start_residue_number:1, 
  end_residue_number:3, 
  color:{r:255,g:255,b:0}
},
{
  struct_asym_id: 'S', 
  start_residue_number:5, 
  end_residue_number:6, 
  color:{r:255,g:255,b:0}
},{
  struct_asym_id: 'S', 
  start_residue_number:8, 
  end_residue_number:23, 
  color:{r:255,g:255,b:0}
},{
  struct_asym_id: 'U', 
  start_residue_number:1, 
  end_residue_number:3, 
  color:{r:255,g:255,b:0}
},{
  struct_asym_id: 'U', 
  start_residue_number:5, 
  end_residue_number:6, 
  color:{r:255,g:255,b:0}
},
{
  struct_asym_id: 'U', 
  start_residue_number:8, 
  end_residue_number:24, 
  color:{r:255,g:255,b:0}
},
{
  struct_asym_id: 'U', 
  start_residue_number:4, 
  end_residue_number:4, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'U', 
  start_residue_number:7, 
  end_residue_number:7, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'S', 
  start_residue_number:7, 
  end_residue_number:7, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'S', 
  start_residue_number:4, 
  end_residue_number:4, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'U', 
  start_residue_number:25, 
  end_residue_number:25, 
  color:{r:10,g:11,b:255}
},
{
  struct_asym_id: 'S', 
  start_residue_number:25, 
  end_residue_number:25, 
  color:{r:10,g:11,b:255}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/3agv-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
    </body>
    </html></td>
  </tr></table><br>
  </div>


<p class="blowheader_box">Binding pocket</p>             
<font >Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 3AGV at 2.15 Å resolution. the Fc fragment of human IgG1 (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with the Fc fragment of human IgG1.</font>
<br><br>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/IgG_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/IgG_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br><br>





<font ><p class="header_boxx5" id="ligand-recognition">Ligand information</p></font>  
<p class="blowheader_box">SELEX ligand</p>
<font >The affinity of the selected aptamers to hIgG1-Fc was examined by surface plasmon resonance (SPR) using BIAcore 2000. We chose association rate constants 1(Ka1) and dissociation rate constants1(Kd1) to Calculate affinity constant (KD) in the document<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>. This result is for reference only.</font >
<div style="display: flex; justify-content: center;">
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">apt2</td>
        <td name="td1">AGUUACAGGUGCUCCAUCAACAAAAUGUUACAUGGAACUG</td>
        <td name="td3">Human IgG1 Fc portion</td>
        <td name="td4">5.6 nM</td>
      </tr>
      <tr>
        <td name="td0">apt7</td>
        <td name="td1">GGAGGUGCUCUGCGAGCCACGCGGAACUCC</td>
        <td name="td3">Human IgG2 Fc portion</td>
        <td name="td4">7.5 nM</td>
      </tr>
      <tr>
        <td name="td0">apt8</td>
        <td name="td1">GGAGGUGCUCCGAAAGGAACUCC(U and C are modified with fluorine)</td>
        <td name="td3">Human IgG3 Fc portion</td>
        <td name="td4">75 nM</td>
      </tr>
      <tr>
        <td name="td0">apt8-1</td>
        <td name="td1">GGAGGUGCUCCGAAAGGAACUCC(U and C are modified with fluorine, some bases used deoxynucleotide)</td>
        <td name="td3">Human IgG4 Fc portion</td>
        <td name="td4">25.5 nM</td>
      </tr>
      <tr>
        <td name="td0">apt8-2</td>
        <td name="td1">GGAGGUGCUCCGAAAGGAACUCC(some U and C was modified with fluorine, some bases used deoxynucleotide)</td>
        <td name="td3">Human IgG5 Fc portion</td>
        <td name="td4">110 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>
<br>

<p class="blowheader_box">Structure ligand</p>
<font><p>The basic structure of immunoglobulin (Ig) molecules is a tetramer of two light chains and two heavy chains linked by disulphide bonds. There are two types of light chains: kappa and lambda, each composed of a constant domain (CL) and a variable domain (VL). There are five types of heavy chains: alpha, delta, epsilon, gamma and mu, all consisting of a variable domain (VH) and three (in alpha, delta and gamma) or four (in epsilon and mu) constant domains (CH1 to CH4). Ig molecules are highly modular proteins, in which the variable and constant domains have clear, conserved sequence patterns. The domains in Ig and Ig-like molecules are grouped into four types: V-set, C1-set, C2-set and I-set. Structural studies have shown that these domains share a common core Greek-key β-sandwich structure, with the types differing in the number of strands in the β-sheets as well as in their sequence patterns.-----from Pfam(Interpro)</p></font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">MW</th>
        <th onclick="sortTable(3)">Amino acids sequences</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P01857/entry#structure" target="_blank" style="color:#520049"><b>P01857</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR003597/" target="_blank" style="color:#520049"><b>IPR003597</b></a></td>
        <td name="td2">23.89 KDa</td>
        <td name="td3">GPSVFLFPPKPKDTLMISRTPEVTCVVVDVSHEDPEVKFNWYVDGVEVHNAKTKPREEQYNSTYRVVSVLTVLHQDWLNGKEYKCKVSNKALPAPIEKTISKAKGQPREPQVYTLPPSRDELTKNQVSLTCLVKGFYPSDIAVEWESNGQPENNYKTTPPVLDSDGSFFLYSKLTVDKSRWQQGNVFSCSVMHEALHNHYTQKSLSLSPGK</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1FC1" target="_blank" style="color:#520049"><b>1FC1</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&id=9606" target="_blank" style="color:#520049"><b>9606</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/IgG_aptamer_stru_ligand.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;">
  </div>


<p class="blowheader_box">Similar compound</p>
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">3AGV-B</td>
        <td name="td2">31.6</td>
        <td name="td3">0</td>
        <td name="td4">Ig gamma-1 chain c region</td>
      </tr>
      <tr>
        <td name="td1">1OQO-A</td>
        <td name="td2">29.2</td>
        <td name="td3">0.5</td>
        <td name="td4">Immunoglobulin gamma-1 heavy</td>
      </tr>
      <tr>
        <td name="td1">7Q3P-B</td>
        <td name="td2">29.1</td>
        <td name="td3">0.6</td>
        <td name="td4">IgG1-Fc-MST-HH</td>
      </tr>
      <tr>
        <td name="td1">5D4Q-A</td>
        <td name="td2">29</td>
        <td name="td3">0.7</td>
        <td name="td4">Ig gamma-1 chain c region</td>
      </tr>
      <tr>
        <td name="td1">5IW6-A</td>
        <td name="td2">29</td>
        <td name="td3">0.7</td>
        <td name="td4">Ig gamma-1 chain c region</td>
      </tr>
      <tr>
        <td name="td1">3AVE-B</td>
        <td name="td2">29</td>
        <td name="td3">0.7</td>
        <td name="td4">Ig gamma-1 chain c region</td>
      </tr>
      <tr>
        <td name="td1">6FGO-A</td>
        <td name="td2">29</td>
        <td name="td3">0.7</td>
        <td name="td4">Immunoglobulin gamma-1 heavy</td>
      </tr>
      <tr>
        <td name="td1">6GFE-H</td>
        <td name="td2">29</td>
        <td name="td3">0.8</td>
        <td name="td4">Immunoglobulin gamma-4 heavy</td>
      </tr>
      <tr>
        <td name="td1">4KU1-B</td>
        <td name="td2">28.9</td>
        <td name="td3">0.9</td>
        <td name="td4">Ig gamma-1 chain c region</td>
      </tr>
      <tr>
        <td name="td1">5U4Y-A</td>
        <td name="td2">28.9</td>
        <td name="td3">0.6</td>
        <td name="td4">IgG1 Fc</td>
      </tr>
	  </tbody>
  </table>

<br><br>

                 
<p class="header_boxx6" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Structural and molecular basis for hyperspecificity of RNA aptamer to human immunoglobulin G.</strong></font><br />
Miyakawa, S., Nomura, Y., Sakamoto, T., Yamaguchi, Y., Kato, K., Yamazaki, S., & Nakamura, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18441054/" target="_blank" style="color:#520049" >RNA (New York, N.Y.), 14(6), 1154–1163. (2008)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Crystallization and preliminary X-ray diffraction studies of an RNA aptamer in complex with the human IgG Fc fragment.</strong></font><br />
Sugiyama, S., Nomura, Y., Sakamoto, T., Kitatani, T., Kobayashi, A., Miyakawa, S., Takahashi, Y., Adachi, H., Takano, K., Murakami, S., Inoue, T., Mori, Y., Nakamura, Y., & Matsumura, H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18931441/" target="_blank" style="color:#520049">Acta crystallographica. Section F, Structural biology and crystallization communications, 64(Pt 10), 942–944. (2008)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Conformational plasticity of RNA for target recognition as revealed by the 2.15 A crystal structure of a human IgG-aptamer complex.</strong></font><br />
Nomura, Y., Sugiyama, S., Sakamoto, T., Miyakawa, S., Adachi, H., Takano, K., Murakami, S., Inoue, T., Mori, Y., Nakamura, Y., & Matsumura, H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/20675355/" target="_blank" style="color:#520049"> Nucleic acids research, 38(21), 7822–7829. (2010)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Engineered synthetic polymer nanoparticles as IgG affinity ligands.</strong></font><br />
Lee, S. H., Hoshino, Y., Randall, A., Zeng, Z., Baldi, P., Doong, R. A., & Shea, K. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/22924890/" target="_blank" style="color:#520049">Journal of the American Chemical Society, 134(38), 15765–15772. (2012)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Target replacement strategy for selection of DNA aptamers against the Fc region of mouse IgG.</strong></font><br />
Ma, J., Wang, M. G., Mao, A. H., Zeng, J. Y., Liu, Y. Q., Wang, X. Q., Ma, J., Tian, Y. J., Ma, N., Yang, N., Wang, L., & Liao, S. Q.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23661463/" target="_blank" style="color:#520049">Genetics and molecular research : GMR, 12(2), 1399–1410. (2013)</a>
<br />

