---
layout: theophylline_style
title:  "ligand 1.1 aptamer"
date:   2024-05-08 00:00:00
author: Ke Chen, Yangyi Ren
categories: Aptamer
---
<html>




<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/1379730/" target="_blank" style="color:#520049">1992</a></h3>
  </div>
  <div class="body">
    <p>High-affinity ligands of the reverse transcriptase of human immunodeficiency virus type 1 (HIV-1) were isolated by the SELEX procedure from RNA populations randomized at 32 positions<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10751399/" target="_blank" style="color:#520049">2000</a></h3>
  </div>
  <div class="body">
    <p>RNA inhibitor bound to HIV RT in a two-step process, with association rates similar to those described for model DNA/DNA and DNA/RNA substrates<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12050367/" target="_blank" style="color:#520049">2002</a></h3>
  </div>
  <div class="body">
    <p>With the help of RNA aptamers, some of the HIV subtypes were severely inhibited (subtypes A, B, D, E, and F), while others were either moderately inhibited (subtypes C and O) or were naturally resistant to inhibition (chimeric A/D subtype)<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25073457/" target="_blank" style="color:#520049">2014</a></h3>
  </div>
  <div class="body">
    <p>RNA aptamers interacted with HIV-1 RT in a fingers-and-thumb-closed conformation, at the bound at the nucleic acid substrate binding site<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/31943114/" target="_blank" style="color:#520049">2020</a></h3>
  </div>
  <div class="body">
    <p>Features at the RT aptamer interface that govern recognition specificity by a broad-spectrum antiviral aptamer, and they open new possibilities for accelerating RT maturation and interfering with viral replication<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/32732393/" target="_blank" style="color:#520049">2020</a></h3>
  </div>
  <div class="body">
    <p>2'-fluoro-modified pyrimidines enhance affinity of RNA aptamers to HIV-1 RT<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
  </div>
 </div>
</div>





<p><p class="header_box" id="description">Description</p>
<font >In 1992, Craig Tuerk and Larry Gold et al. used the SELEX method to isolate the aptamer with high affinity for the HIV-1 RT. When the aptamer binds to HIV-1 RT, it reduces the activity of the target protein, thereby preventing RT from catalysing the synthesis of the proviral genome and affecting subsequent integration into the host cell genome<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 1992, Craig Tuerk and Larry Gold et al. isolated high affinity sequences for targeting HIV-1 RT from RNA populations randomized at 32 positions through 9 rounds of selection<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>Ligand 1.1 was the aptamer sequence mainly studied in the article, which had a high affinity with HIV-1 RT. The 2D structure of the figure is based on the prediction results of the RNA fold website by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>5'-GGGAGCAUCAGACUUUUAAUCUGACAAUCAAGAAUUCCGUUUUCAGUCGGGAAAAACUGAACAAUCUAUGAAAGAAUUUUAUAUCUCUAUUGAAAC-3'</p>
<img src="/images/2D/ligand1.1_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;">   
</div>


<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>A reverse transcriptase (RT) is an enzyme used to generate complementary DNA (cDNA) from an RNA template, a process termed reverse transcription. Reverse transcriptases are used by viruses such as HIV to replicate their genomes, by retrotransposon mobile genetic elements to proliferate within the host genome, and by eukaryotic cells to extend the telomeres at the ends of their linear chromosomes.-----From Pfam</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">HIV-1 RT</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P03366/entry#names_and_taxonomy" target="_blank" style="color:#520049"><b>P03366</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00078/" target="_blank" style="color:#520049"><b>PF00078</b></a></td>
        <td name="td3">104.53 kDa</td>
        <td name="td4">Subunit P66:PISPIETVPVKLKPGMDGPKVKQWPLTEEKIKALVEICTEMEKEGKISKIGPENPYNTPVFAIVDFRELNKRTQDFWEVQLGIPHPAGLKKKKSVTVLDVGDAYFSVPLDEDFRKYTAFTIPGIRYQYNVLPQGWKGSPAIFQSSMTKILEPFKKQNPDIVIYQYMDDLYVGSDLEIGQHRTKIEELRQHLLRWGLTTPDKKHQKEPPFLWMGYELHPDKWTVQPIVLPEKDSWTVNDIQKLVGKLNWASQIYPGIKVRQLCKLLRGTKALTEVIPLTEEAELELAENREILKEPVHGVYYDPSKDLIAEIQKQGQGQWTYQIYQEPFKNLKTGKYARMRGAHTNDVKQLTEAVQKITTESIVIWGKTPKFKLPIQKETWETWWTEYWQATWIPEWEFVNTPPLVKLWYQLEKEPIVGAETFYVDGAANRETKLGKAGYVTNKGRQKVVPLTNTTNQKTELQAIYLALQDSGLEVNIVTDSQYALGIIQAQPDKSESELVNQIIEQLIKKEKVYLAWVPAHKGIGGNEQVDKLVSA<br>Subunit P51:IETVPVKLKPGMDGPKVKQWPLTEEKIKALVEICTEMEKEGKISKIGPENPYNTPVFAIKKWRKLVDFRELNKRTQDFWEVQLGIPHPAGLKKKKSVTVLDVGDAYFSVPLDEDFRKYTAFTIPSINNETPGIRYQYNVLPQGWKGSPAIFQSSMTKILEPFKKQNPDIVIYQYMDDLYVGSDLEIGQHRTKIEELRQHLLRHPDKWTVQPIVLPEKDSWTVNDIQKLVGKLNWASQIYPGIKVRQLCKLLRGTKALTEVIPLTEEAELELAENREILKEPVHGVYYDPSKDLIAEIQKQGQGQWTYQIYQEPFKNLKTGKYARMRGAHTNDVKQLTEAVQKITTESIVIWGKTPKFKLPIQKETWETWWTEYWQATWIPEWEFVNTPPLVKLWY</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1HMV" target="_blank" style="color:#520049"><b>1HMV</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/protein/AAA44198.1" target="_blank" style="color:#520049"><b>AAA44198.1</b></a></td>
      </tr>
	  </tbody>
  </table>
<p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">ligand 1.1</td>
        <td name="td1">GGGAGCAUCAGACUUUUAAUCUGACAAUCAAGAAUUCCGUUUUCAGUCGGGAAAAACUGAACAAUCUAUGAAAGAAUUUUAUAUCUCUAUUGAAAC</td>
        <td name="td2">HIV-1 RT</td>
        <td name="td3">5.0 nM</td>
      </tr>
	  </tbody>
     <tbody>
      <tr>
        <td name="td0">ligand 1.3a</td>
        <td name="td1">GGGAGCAUCAGACUUUUAAUCUGACAAUCAAGAAUAUCUUCCGAAGCCGAACGGGAAAACCGGCAUCUAUGAAAGAAUUUUAUAUCUCUAUUGAAAC</td>
        <td name="td2">HIV-1 RT</td>
        <td name="td3">NA</td>
      </tr>
	  </tbody>
      <tbody>
      <tr>
        <td name="td0">ligand 1.3b</td>
        <td name="td1">GGGAGCAUCAGACUUUUAAUCUGACAAUCAAGAAUAUCUUCCGAGGCCGAACGGGAAAACCGACAUCUAUGAAAGAAUUUUAUAUCUCUAUUGAAAC</td>
        <td name="td2">HIV-1 RT</td>
        <td name="td3">NA</td>
      </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/ligand1.1_aptamer_SELEX_ligand1.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<br>
<img src="/images/SELEX_ligand/ligand1.1_aptamer_SELEX_ligand2.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD (Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">6WB2-A</td>
      <td name="td1">28.8</td>
      <td name="td2">4.9</td>
      <td name="td3">HIV-1 viral RNA genome fragment</td>
    </tr>
     <tr>
      <td name="td0">7SR6-A</td>
      <td name="td1">22.0</td>
      <td name="td2">3.4</td>
      <td name="td3">Polymerase</td>
    </tr>
     <tr>
      <td name="td0">3V1O-A</td>
      <td name="td1">16.5</td>
      <td name="td2">2.2</td>
      <td name="td3">Reverse transcriptase/Ribonuclease H P80</td>
    </tr>
     <tr>
      <td name="td0">7KSE-A</td>
      <td name="td1">15.1</td>
      <td name="td2">18.7</td>
      <td name="td3">Peptidase A9/Reverse transcriptase/Ribonuclease H</td>
    </tr>
     <tr>
      <td name="td0">2QKK-W</td>
      <td name="td1">15.1</td>
      <td name="td2">2.1</td>
      <td name="td3">Ribonuclease H1</td>
    </tr>
     <tr>
      <td name="td0">4OL8-B</td>
      <td name="td1">14.4</td>
      <td name="td2">3.3</td>
      <td name="td3">Reverse transcriptase/Ribonuclease H</td>
    </tr>
     <tr>
      <td name="td0">2EHG-A</td>
      <td name="td1">13.8</td>
      <td name="td2">2.3</td>
      <td name="td3">Ribonuclease HI</td>
    </tr>
     <tr>
      <td name="td0">4IBN-A</td>
      <td name="td1">12.9</td>
      <td name="td2">2.0</td>
      <td name="td3">Ribonuclease H</td>
    </tr>
     <tr>
      <td name="td0">3DU5-A</td>
      <td name="td1">7.9</td>
      <td name="td2">8.4</td>
      <td name="td3">Telomerase reverse transcriptase</td>
    </tr>
     <tr>
      <td name="td0">7V9X-A</td>
      <td name="td1">7.3</td>
      <td name="td2">5.2</td>
      <td name="td3">RNA-directed DNA polymerase from Retron Ec86</td>
    </tr>
    </tbody>
  </table>

                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] RNA pseudoknots that inhibit human immunodeficiency virus type 1 reverse transcriptase.</strong></font><br />
Tuerk, C., MacDougal, S., & Gold, L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/1379730/" target="_blank" style="color:#520049">Proceedings of the National Academy of Sciences of the United States of America, 89(15), 6988–6992. (1992)</a>
<br/>
            
<a id="ref2"></a><font><strong>[2] HIV-1 reverse transcriptase-pseudoknot RNA aptamer interaction has a binding affinity in the low picomolar range coupled with high specificity.</strong></font><br />
Kensch, O., Connolly, B. A., Steinhoff, H. J., McGregor, A., Goody, R. S., & Restle, T. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10751399/" target="_blank" style="color:#520049">The Journal of biological chemistry, 275(24), 18271–18278. (2000)</a>
<br/>
            
<a id="ref3"></a><font><strong>[3] Potent inhibition of human immunodeficiency virus type 1 replication by template analog reverse transcriptase inhibitors derived by SELEX (systematic evolution of ligands by exponential enrichment).</strong></font><br />
Joshi, P., & Prasad, V. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12050367/" target="_blank" style="color:#520049">Journal of virology, 76(13), 6545–6557. (2002)</a>
<br/>
            
<a id="ref4"></a><font><strong>[4] Insight into HIV-1 reverse transcriptase aptamer interaction from molecular dynamics simulations.</strong></font><br />
Aeksiri, N., Songtawee, N., Gleeson, M. P., Hannongbua, S., & Choowongkomon, K.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25073457/" target="_blank" style="color:#520049">Journal of molecular modeling, 20(8), 2380. (2014)</a>
<br/>
            
<a id="ref5"></a><font><strong>[5] Binding interface and impact on protease cleavage for an RNA aptamer to HIV-1 reverse transcriptase.</strong></font><br />
Nguyen, P. D. M., Zheng, J., Gremminger, T. J., Qiu, L., Zhang, D., Tuske, S., Lange, M. J., Griffin, P. R., Arnold, E., Chen, S. J., Zou, X., Heng, X., & Burke, D. H. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/31943114/" target="_blank" style="color:#520049">Nucleic acids research, 48(5), 2709–2722. (2020)</a>
<br/>
            
<a id="ref6"></a><font><strong>[6] 2'-fluoro-modified pyrimidines enhance affinity of RNA oligonucleotides to HIV-1 reverse transcriptase.</strong></font><br />
Gruenke, P. R., Alam, K. K., Singh, K., & Burke, D. H.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/32732393/" target="_blank" style="color:#520049">RNA (New York, N.Y.), 26(11), 1667–1679. (2020)</a>
<br/>


<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1RAW-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>

    
    </html>
