---
layout: theophylline_style
title: LR06(GA) aptamer
date: 2024-04-05 00:00:00
author: Ke Chen, Yangyi Ren
categories: Aptamer
tags:
- Category:Amino Acids & Peptides
- Category:Bacteria
- Category:Biologics & Signaling Molecules
- Category:Fluorophores
- Category:Mammalian
- Category:Nucleosides & Nucleotides
- Category:Other
- Category:Pharmaceuticals
- Category:Prion
- Category:RNA
- Category:Synthetic
- Category:Toxin
- Category:Viral
- Category:cell
- GC:31.0
- GC:33.629999999999995
- GC:35.0
- GC:35.4
- GC:39.0
- GC:40.0
- GC:40.82
- GC:40.849999999999994
- GC:41.0
- GC:41.25
- GC:42.02
- GC:42.370000000000005
- GC:42.699999999999996
- GC:43.0
- GC:43.71
- GC:44.07
- GC:44.3
- GC:44.440000000000005
- GC:45.07
- GC:45.5
- GC:45.57
- GC:45.68
- GC:45.7
- GC:45.95
- GC:46.0
- GC:46.150000000000006
- GC:46.25
- GC:46.339999999999996
- GC:46.46
- GC:46.48
- GC:46.51
- GC:46.62
- GC:46.72
- GC:46.77
- GC:46.88
- GC:47.0
- GC:47.17
- GC:47.5
- GC:47.54
- GC:47.89
- GC:48.0
- GC:48.06
- GC:48.15
- GC:48.57
- GC:48.75
- GC:48.84
- GC:49.0
- GC:49.09
- GC:49.3
- GC:49.370000000000005
- GC:49.51
- GC:49.62
- GC:49.69
- GC:50.0
- GC:50.480000000000004
- GC:50.68
- GC:50.7
- GC:51.160000000000004
- GC:51.2
- GC:51.28
- GC:51.32
- GC:51.349999999999994
- GC:51.519999999999996
- GC:51.61
- GC:51.849999999999994
- GC:51.9
- GC:51.949999999999996
- GC:52.0
- GC:52.11
- GC:52.21
- GC:52.33
- GC:52.5
- GC:52.7
- GC:52.81
- GC:52.94
- GC:53.0
- GC:53.33
- GC:53.52
- GC:53.620000000000005
- GC:53.949999999999996
- GC:54.0
- GC:54.169999999999995
- GC:54.410000000000004
- GC:54.55
- GC:54.64
- GC:55.26
- GC:55.559999999999995
- GC:55.669999999999995
- GC:55.7
- GC:55.75
- GC:55.910000000000004
- GC:56.06
- GC:56.10000000000001
- GC:56.18
- GC:56.25
- GC:56.34
- GC:56.52
- GC:56.67
- GC:56.96
- GC:56.99999999999999
- GC:57.14
- GC:57.58
- GC:57.89
- GC:57.99999999999999
- GC:58.230000000000004
- GC:58.330000000000005
- GC:58.42
- GC:58.440000000000005
- GC:58.46
- GC:58.620000000000005
- GC:58.75
- GC:58.81999999999999
- GC:59.0
- GC:59.040000000000006
- GC:59.209999999999994
- GC:59.4
- GC:59.68
- GC:59.78
- GC:60.0
- GC:60.24
- GC:60.4
- GC:60.760000000000005
- GC:60.81999999999999
- GC:61.0
- GC:61.040000000000006
- GC:61.11
- GC:61.33
- GC:61.53999999999999
- GC:62.86000000000001
- GC:62.96000000000001
- GC:63.0
- GC:63.04
- GC:63.1
- GC:63.21
- GC:63.89
- GC:64.0
- GC:64.1
- GC:64.29
- GC:64.47
- GC:65.52
- GC:65.79
- GC:66.0
- GC:67.0
- GC:67.86
- GC:68.0
- GC:68.17999999999999
- GC:68.85
- GC:69.0
- GC:69.37
- GC:70.0
- GC:70.59
- GC:71.0
- GC:71.74000000000001
- GC:77.42
- Length:10
- Length:100
- Length:101
- Length:103
- Length:105
- Length:106
- Length:108
- Length:111
- Length:113
- Length:118
- Length:119
- Length:12
- Length:122
- Length:124
- Length:129
- Length:130
- Length:131
- Length:132
- Length:133
- Length:14
- Length:150
- Length:152
- Length:159
- Length:16
- Length:162
- Length:167
- Length:18
- Length:20
- Length:23
- Length:28
- Length:29
- Length:30
- Length:31
- Length:32
- Length:33
- Length:34
- Length:35
- Length:36
- Length:37
- Length:38
- Length:39
- Length:40
- Length:41
- Length:42
- Length:43
- Length:44
- Length:46
- Length:48
- Length:51
- Length:52
- Length:54
- Length:55
- Length:57
- Length:58
- Length:60
- Length:61
- Length:62
- Length:63
- Length:65
- Length:68
- Length:69
- Length:70
- Length:71
- Length:72
- Length:73
- Length:74
- Length:75
- Length:76
- Length:77
- Length:78
- Length:79
- Length:80
- Length:81
- Length:82
- Length:83
- Length:84
- Length:86
- Length:89
- Length:90
- Length:91
- Length:92
- Length:93
- Length:96
- Length:97
- Length:98
- Length:99
- Named:3',5'-cyclic adenosine monophosphate (cAMP)_cAMP - b
- Named:3',5'-cyclic adenosine monophosphate (cAMP)_cAMP - b (parent)
- Named:5-Hydroxytryptophan (5HTP)_5GR-II RNA aptamer
- Named:ACE_Okra aptamer
- Named:AML1 Runt domain_Apt1-S aptamer
- Named:AMPA and kainate receptors_AB9
- Named:AMPA and kainate receptors_AB9s
- Named:Adenosine triphosphate (ATP)_ATP binding aptamer
- Named:Adenosine triphosphate (ATP)_V2 Hammerhead Ribozyme
- Named:Adenosine triphosphate (ATP)_V2 trans Hammerhead Ribozyme
- Named:Adenosine triphosphate (ATP)_aptRNA1
- Named:Adenosine triphosphate (ATP)_bG40min
- Named:Adenosine triphosphate (ATP)_bU118
- Named:Adenosine triphosphate (ATP)_bU320
- Named:Adenosine triphosphate (ATP)_bU321
- Named:Adenosine triphosphate (ATP)_pRNAwt
- Named:Adenosine triphosphate (ATP)_rafl28
- Named:Adenosine triphosphate (ATP)_rafl7
- Named:Adenosine triphosphate (ATP)_rafl7-U61C
- Named:Adenosine triphosphate (ATP)_rafl7s
- Named:Amyloid beta-peptide (Aβ)_E2
- Named:Amyloid beta-peptide (Aβ)_N2
- Named:Bacteriophage MS2 coat protein_F6 aptamer
- Named:Basic fibroblast growth factor (bFGF)_12A
- Named:Basic fibroblast growth factor (bFGF)_13A
- Named:Basic fibroblast growth factor (bFGF)_14A
- Named:Basic fibroblast growth factor (bFGF)_21A
- Named:Basic fibroblast growth factor (bFGF)_22B
- Named:Basic fibroblast growth factor (bFGF)_26A
- Named:Basic fibroblast growth factor (bFGF)_26A aptamer(truncation)
- Named:Basic fibroblast growth factor (bFGF)_26At
- Named:Basic fibroblast growth factor (bFGF)_28B
- Named:Basic fibroblast growth factor (bFGF)_5A
- Named:Basic fibroblast growth factor (bFGF)_7A
- Named:Basic fibroblast growth factor (bFGF)_7A aptamer(truncation)
- Named:Bovine prion protein_R12 aptamer
- Named:C-C motif chemokine 2 (L-CCL2)_NOX-E36 L aptamer
- Named:Carbon storage regulation protein A, Escherichia coli (CsrA)_R9-43
- Named:Cell death abnormality gene 9 (CED-9)_R9-2
- Named:Cell death abnormality gene 9 (CED-9)_R9-7
- Named:Cibacron Blue 3GA_CB-42
- Named:Cyanocobalamin (vitamin B12)_35-mer aptamer
- Named:Cyclophilin B (CypB)_M9-5
- Named:Cytoplasmic tail of BACE (B1-CT)_S10
- Named:Cytoplasmic tail of BACE (B1-CT)_TH14
- Named:DFHO_Corn aptamer
- Named:DMHBO+, DMHBI_Chili aptamer
- Named:DNA polymerase beta (polβ)_Aptamer 25
- Named:DNA polymerase beta (polβ)_Aptamer 31
- Named:DNA polymerase beta (polβ)_Aptamer 32
- Named:DNA polymerase beta (polβ)_Aptamer 45
- Named:DNA polymerase beta (polβ)_Aptamer 53
- Named:DNA polymerase kappa (polκ)_Aptamer 25
- Named:DNA polymerase kappa (polκ)_Aptamer 32
- Named:DNA polymerase kappa (polκ)_Aptamer 45
- Named:Drosophila muscleblind protein (Mbl)_RNA 20
- Named:E2F transcription factor (E2F1)_E1 RNA aptamer
- Named:E2F transcription factor (E2F1)_E2F-E1
- Named:Escherichia coli release factor 1_Class II-1
- Named:Estrogen receptor α (ERα)_AptER-1
- Named:Estrogen receptor α (ERα)_AptER-2
- Named:Estrogen receptor α (ERα)_AptER-3
- Named:Eukaryotic translation initiation factor 4A (eIF4A)_APT58 aptamer
- Named:Extracellular domains of HER3_A30
- Named:Flavin adenine dinucleotide (FAD), Flavin mononucleotide(FMN), Riboflavin(Rb)_X2B2-C14U
  aptamer
- Named:Flavin mononucleotide (FMN)_35 nucleotide RNA aptamer
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.18 aptamer
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.20 aptamer
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.28 aptamer
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.51
- Named:Gemini-561o-Coral
- Named:GluR2 AMPA receptors_AN58
- Named:Glutaminyl-tRNA synthetase_var-AGGU RNA aptamer
- Named:Guanosine triphosphate (GTP)_Aptamer 10-10
- Named:Guanosine triphosphate (GTP)_Aptamer 10-24
- Named:Guanosine triphosphate (GTP)_Aptamer 10-59
- Named:Guanosine triphosphate (GTP)_Aptamer 10-6
- Named:Guanosine triphosphate (GTP)_Aptamer 9-12
- Named:Guanosine triphosphate (GTP)_Class I
- Named:Guanosine triphosphate (GTP)_Class I aptamer(truncation)
- Named:HCV NS3 protease (ΔNS3)_G9-I
- Named:HCV NS3 protease (ΔNS3)_G9-II
- Named:HCV NS3 protease (ΔNS3)_G9-III
- Named:HER2-positive breast cancer cells_TSA12 aptamer
- Named:HER2-positive breast cancer cells_TSA14
- Named:HER2-positive breast cancer cells_TSA14 aptamer(truncation)
- Named:HIV-1 Rev protein_6
- Named:HIV-1 TAR RNA_LR06(GA) aptamer
- Named:HIV-1 TAR RNA_R-06
- Named:HIV-1 Tat protein_clone 31
- Named:Haemagglutinin (HA) region of the target strain A/Panama/2007/1999(H3N2)_P30-10-16
- Named:Hfq (host factor required for phage Qβ RNA replication)_AGr aptamer
- Named:Hfq (host factor required for phage Qβ RNA replication)_m49f aptamer
- Named:Human T-cell lymphomatic virus (HTLV-1) tax protein_YT1
- Named:Human immunoglobulin G_apt2 aptamer
- Named:Human immunoglobulin G_apt7 aptamer
- Named:Human immunoglobulin G_apt8 aptamer
- Named:Insulin receptor (glioma cells)_GL56
- Named:L-arginine_ag.06
- Named:L-arginine_ag.06_1
- Named:L-tyrosine_Tyr 1
- Named:Malachite green (MG),Tetramethylrhodamine (TMR)_MG aptamer
- Named:Malachite green (MG)MG4
- Named:Monomeric beta-2-microglobulin (β2m) fibrils_LS-5
- Named:Monomeric beta-2-microglobulin (β2m) fibrils_M-2
- Named:Monomeric beta-2-microglobulin (β2m) fibrils_WL-2
- Named:NBSI_Clivias aptamer
- Named:NF-kappaB P50_RNA aptamer 3
- Named:NF-kappaB P50_α-p50 RNA aptamer
- Named:NS3 Protein of Hepatitis C Virus (HCV)_G6–16
- Named:NS3 Protein of Hepatitis C Virus (HCV)_G6–19
- Named:NS3 protease of hepatitis C virus (HCV)_10G-1
- Named:Neuropeptide Y (NPY)_DP3
- Named:Nonstructural 5B (NS5B) polymerase, hepatitis C virus (HCV) NS5BΔC55 protein_aptamer
  A.2
- Named:Nonstructural 5B (NS5B) polymerase, hepatitis C virus (HCV) NS5BΔC55 protein_aptamer
  B.2 SLII
- 'Named:Nuclear factor of activated T cells (NFAT) DNA binding domain_RNA aptamer
  #1'
- 'Named:Nuclear factor of activated T cells (NFAT) DNA binding domain_RNA aptamer
  #2'
- Named:OX40_11F11
- Named:OX40_9C7
- Named:OX40_9C7 aptamer (truncation)
- Named:Pepocin_8-09 aptamer
- Named:Proinflammatory cytokine interleukin-17A (IL-17A)_Apt21-2 aptamer(truncation)
- Named:Proinflammatory cytokine interleukin-17A (IL-17A)_Apt21‐2
- Named:Prostate cancer cell_E3 aptamer
- Named:Prostate cancer cell_E3 aptamer (truncation)
- Named:Prostate-specific membrane antigen (PSMA)_A9 RNA aptamer
- Named:Prostate-specific membrane antigen (PSMA)_A9g aptamer
- Named:Protein kinase C beta II_PKC 10
- Named:Protein kinase C beta II_PKC 6
- Named:Protein tyrosine phosphatase 1B (PTP1B)_C13
- Named:Protein tyrosine phosphatase 1B (PTP1B)_C13M21
- Named:Protein tyrosine phosphatase 1B (PTP1B)_C9
- Named:RNA stem-loop_g18_4.seq
- Named:RNA-dependent RNA polymerase (3Dpol)_F38
- Named:RNA-dependent RNA polymerase (3Dpol)_F47
- Named:RNA-dependent RNA polymerase (3Dpol)_F47tr
- Named:RNA-dependent RNA polymerase (3Dpol)_F52
- Named:Ras-binding domain (Raf-1 RBD)_21.01
- Named:Ras-binding domain (Raf-1 RBD)_21.07
- Named:Rat monoclonal antibody (mAb198)_tMG RNA
- Named:Reactive Blue 4_B4-25
- Named:Receptor activator of NF-kappaB (RANK)_apt1
- Named:Receptor activator of NF-kappaB (RANK)_apt2
- Named:Receptor activator of NF-kappaB (RANK)_apt3
- Named:Ribosomal protein S8_RNA-2 aptamer
- Named:Serine protease inhibitor plasminogen activator inhibitor-1 (PAI-1)_SM-20
- Named:Serine protease inhibitor plasminogen activator inhibitor-1 (PAI-1)_WT-15
- Named:Sialyl Lewis X (sLeX)_Clone 5
- Named:Sphingosylphosphorylcholine (SPC)_C1
- Named:Sphingosylphosphorylcholine (SPC)_C2
- Named:Sphingosylphosphorylcholine (SPC)_C8
- Named:Sphingosylphosphorylcholine (SPC)_m009
- Named:Sphingosylphosphorylcholine (SPC)_m010
- Named:Sphingosylphosphorylcholine (SPC)_m011
- Named:Sphingosylphosphorylcholine (SPC)_m012
- Named:Substance P_A11
- Named:Substance P_SUP-A-004
- Named:Sulforhodamine B_SRB-2
- Named:T cytokine 1 (TCF-1)_#10
- Named:T4 DNA polymerase (gp43)_Major variant aptamer(truncation)
- Named:T4 DNA polymerase (gp43)_Wild type aptamer(truncation)
- Named:T4 DNA polymerase (gp43)_major variant
- Named:T4 DNA polymerase (gp43)_wild type
- Named:TMR (Tetramethylrhodamine)TMR1
- Named:TMR (Tetramethylrhodamine)TMR2
- Named:TMR (Tetramethylrhodamine)TMR3
- Named:TMR (Tetramethylrhodamine)TMR4
- Named:TMR (Tetramethylrhodamine)_TMR3 aptamer
- Named:TMR-DN, Pyronin B, ATTO495SRB-3
- Named:Tetracycline_cb28 aptamer
- Named:Tobramycin_J6f1
- Named:Transcription repressor TetR_TetR-RNA aptamer
- Named:Triple-negative breast cancer (TNBC)_TN145
- Named:Triple-negative breast cancer (TNBC)_TN2
- Named:Triple-negative breast cancer (TNBC)_TN20
- Named:Triple-negative breast cancer (TNBC)_TN29
- Named:Triple-negative breast cancer (TNBC)_TN3
- Named:Triple-negative breast cancer (TNBC)_TN58
- Named:Tumor necrosis factor alpha (hTNFα)_1
- Named:Tumor necrosis factor alpha (hTNFα)_10
- Named:Tumor necrosis factor alpha (hTNFα)_11.7
- Named:Tumor necrosis factor alpha (hTNFα)_12
- Named:Tumor necrosis factor alpha (hTNFα)_13.2
- Named:Tumor necrosis factor alpha (hTNFα)_14.8
- Named:Tumor necrosis factor alpha (hTNFα)_15
- Named:Tumor necrosis factor alpha (hTNFα)_16.3
- Named:Tumor necrosis factor alpha (hTNFα)_17.9
- Named:Tumor necrosis factor alpha (hTNFα)_18.1
- Named:Tumor necrosis factor alpha (hTNFα)_2
- Named:Tumor necrosis factor alpha (hTNFα)_20.4
- Named:Tumor necrosis factor alpha (hTNFα)_3.1
- Named:Tumor necrosis factor alpha (hTNFα)_4
- Named:Tumor necrosis factor alpha (hTNFα)_6
- Named:Tumor necrosis factor alpha (hTNFα)_7.5
- Named:Tumor necrosis factor alpha (hTNFα)_8.6
- Named:Tumor necrosis factor alpha (hTNFα)_9
- Named:Tumor necrosis factor alpha (hTNFα)_T1.7.5
- Named:Tumor necrosis factor alpha (hTNFα)_T4.14.8
- Named:Vascular Endothelial Growth Factor (VEGF)_100
- Named:Vascular Endothelial Growth Factor (VEGF)_100t
- Named:Vascular Endothelial Growth Factor (VEGF)_12
- Named:Vascular Endothelial Growth Factor (VEGF)_126
- Named:Vascular Endothelial Growth Factor (VEGF)_126t
- Named:Vascular Endothelial Growth Factor (VEGF)_12t
- Named:Vascular Endothelial Growth Factor (VEGF)_40
- Named:Vascular Endothelial Growth Factor (VEGF)_40t
- Named:Vascular Endothelial Growth Factor (VEGF)_44
- Named:Vascular Endothelial Growth Factor (VEGF)_44t
- Named:Vascular Endothelial Growth Factor (VEGF)_84
- Named:Vascular Endothelial Growth Factor (VEGF)_84t
- Named:Vascular Endothelial Growth Factor (VEGF)_t44-OMe
- Named:Zinc-finger antiviral protein (ZAP)_21-E
- Named:Zinc-finger antiviral protein (ZAP)_21-K
- Named:p-amino phenylalanine (pAF)pAF-4Z1
- Named:p-amino phenylalanine (pAF)pAF-R1-1
- Named:β-amyloid 42 (Aβ42)_E22P-AbD31
- Named:β-amyloid 42 (Aβ42)_E22P-AbD4
- Named:β-amyloid 42 (Aβ42)_E22P–AbD43
- Type:Cells
- Type:Nucleic acids
- Type:Proteins
- Type:Small molecules
- Year:1990
- Year:1992
- Year:1993
- Year:1994
- Year:1995
- Year:1996
- Year:1997
- Year:1998
- Year:1999
- Year:2000
- Year:2001
- Year:2002
- Year:2003
- Year:2004
- Year:2005
- Year:2006
- Year:2007
- Year:2008
- Year:2009
- Year:2010
- Year:2011
- Year:2012
- Year:2013
- Year:2014
- Year:2015
- Year:2016
- Year:2017
- Year:2018
- Year:2019
- Year:2020
- Year:2022
- Year:2024
type: Proteins
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10212256/" target="_blank" style="color:#520049">1999</a></h3>
    </div>
    <div class="body">
      <p>The DNA aptamer was selected for the first time<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/7521014/" target="_blank" style="color:#520049">1999</a></h3>
    </div>
    <div class="body">
      <p>The RNA aptamer was selected for the first time<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10954609/" target="_blank" style="color:#520049">2000</a></h3>
    </div>
    <div class="body">
      <p>Nuclear magnetic resonance (NMR) characterization of a kissing complex formed between the trans-activating responsive (TAR) RNA element and DNA aptamer<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12052182/" target="_blank" style="color:#520049">2002</a></h3>
    </div>
    <div class="body">
      <p>Effect of magnesium concentration and temperature on screening of TAR aptamers in vitro<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12853646/" target="_blank" style="color:#520049">2003</a></h3>
    </div>
    <div class="body">
      <p>Stabilization of ring-closed residues in affinity interactions<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/15181175/" target="_blank" style="color:#520049">2004</a></h3>
    </div>
    <div class="body">
      <p>LNA/DNA chimeric oligomers mimic RNA aptamers targeted to the TAR<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/15723535/" target="_blank" style="color:#520049">2005</a></h3>
    </div>
    <div class="body">
      <p>Discovery and detection of hexitol nucleic acid aptamers<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/17312962/" target="_blank" style="color:#520049">2006</a></h3>
    </div>
    <div class="body">
      <p>Application of SELEX technique in TAR aptamer selection<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup></p>
    </div>
  </div>
<div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/16445294/" target="_blank" style="color:#520049">2006</a></h3>
    </div>
    <div class="body">
      <p>The role of bimodal ring interaction in aptamer binding<sup>[<a href="#ref9" style="color:#520049">9</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/17768146/" target="_blank" style="color:#520049">2007</a></h3>
    </div>
    <div class="body">
      <p>NMR structure of complexes formed between TAR RNA and RNA modified aptamers<sup>[<a href="#ref10" style="color:#520049">10</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9257650/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>Exploring TAR–RNA aptamer loop–loop interaction by X-ray crystallography, ultraviolet (UV) spectroscopy and surface plasmon resonance (SPR)<sup>[<a href="#ref11" style="color:#520049">11</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/19496624/" target="_blank" style="color:#520049">2009</a></h3>
    </div>
    <div class="body">
      <p>In vitro selection of RNA aptamers for HIV-1-resistant TAR RNA elements obtained from human genome libraries<sup>[<a href="#ref12" style="color:#520049">12</a>]</sup></p>
    </div>
  </div>
</div>


<p class="header_box" id="description">Description</p>
<p>In 1999, Ducongé F et al. used in vitro selecting techniques to isolate aptamers with high affinity binding TAR RNA sites. This 59-nt imperfect stem-loop is at the 5' end of HIV-1 transcriptIn. In 2007, Lebars I et al. used NMR systems and molecular dynamics methods to determine the structure of the complex formed by the LNA-modified aptamer and TAR RNA<sup>[<a href="#ref2" style="color:#520049">2</a></sup><sup>,<a href="#ref10" style="color:#520049">10</a>]</sup>.</p>


<p class="header_box" id="SELEX">SELEX</p>
<p>The in vitro selection experiment designed by Ducongé F and colleagues in 1999 mainly consisted of RNA pool ampplification and subsequent selection processes. RNAs were obtained by in vitro transcription of the PCR-amplified library with T7 RNA polymerase. The aptamer with high affinity was selected by eight rounds and the washing times were increased in the sixth to eighth rounds<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>Here we used ribodraw to complete the figure, through the 3D structure information. LR06(GA) was the aptamer sequence mainly studied in the article where structural selecting is performed<sup>[<a href="#ref10" style="color:#520049">10</a>]</sup>.</p>
<p>5'-CACGGUCCCAGACGUG-3'</p>
<img src="/images/2D/LR06_aptamer_2D.svg" alt="drawing" style="width:800px;height:375px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<p class="blowheader_box">3D visualisation</p>             
<font >Lebars I and colleagues present the solution structure of TAR/LR06(GA) complex, as determined by heteronuclear NMR spectroscopy and molecular dynamics calculations. Structures were calculated using CNS (cristallography and NMR system) torsion angle molecular dynamics (TAMD) protocol for nucleic acids using NOE and dihedral angle restraints. In the figure, the TAR RNA sequence is labeled in forest, the LR06(GA) sequence is labeled in cyan, and the interacting bases in the two sequences are labeled in magenta. The PDB ID of this structure is 2OOM<sup>[<a href="#ref10" style="color:#520049" >10</a>]</sup>.<br>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</font>
<div ><font ><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/LR06_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A',    
  start_residue_number:1, 
  end_residue_number:5, 
  color:{r:34,g:139,b:34}
},
{
  struct_asym_id: 'A', 
  start_residue_number:6, 
  end_residue_number:10, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'A', 
  start_residue_number:11, 
  end_residue_number:16, 
  color:{r:34,g:139,b:34}
},
{
  struct_asym_id: 'B', 
  start_residue_number:17, 
  end_residue_number:22, 
  color:{r:9,g:254,b:254}
},
{
  struct_asym_id: 'B', 
  start_residue_number:23, 
  end_residue_number:27, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'B', 
  start_residue_number:28, 
  end_residue_number:32, 
  color:{r:9,g:254,b:254}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/2OOM-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>



<p class="blowheader_box">Binding pocket</p>             
<font >Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 2OOM by NMR. TAR RNA (shown in surface) is labeled in gray. Right: The hydrogen bonds of binding sites of the aptamer bound with TAR RNA.</font>
<br><br>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/LR06(GA)_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/LR06(GA)_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>





<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  
  
<p class="blowheader_box">SELEX ligand</p>
<p>Lebars I and colleagues characterized the loop-loop complexes using thermal denaturation experiments monitored by UV spectroscopy. Subsequently, they determined the affinity between LR06(GA) and TAR RNA by SPR method. At the same time, the affinity between the LNA-modified aptamer and TAR RNA was also determined. These methods were employed to comprehensively assess the stability and affinity of different aptamers interaction under different experimental conditions, allowing for a robust evaluation of the binding affinity and dynamics between RNA aptamer and TAR RNA in varied environments<sup>[<a href="#ref10" style="color:#520049">10</a>]</sup>.</p>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">TAR*(UA)</td>
        <td name="td1">GCUGUUCCCAGACAGC</td>
        <td name="td2">TAR RNA</td>
        <td name="td3">92.5 ± 5.2 nM</td>
      </tr>
      <tr>
        <td name="td0">R06(GA)</td>
        <td name="td1">GGUCGGUCCCAGACGACC</td>
        <td name="td2">TAR RNA</td>
        <td name="td3">355.0 ± 18.0 nM</td>
      </tr>
      <tr>
        <td name="td0">LR06(GA)</td>
        <td name="td1">CACGGUCCCAGACGUG</td>
        <td name="td2">TAR RNA</td>
        <td name="td3">27.8 ± 3.4 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>The HIV trans-activation response (TAR) element is an RNA element which is known to be required for the trans-activation of the viral promoter and for virus replication. The TAR hairpin is a dynamic structure that acts as a binding site for the Tat protein, and this interaction stimulates the activity of the long terminal repeat promoter.-----From Rfam</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Mirbase ID</th>
        <th onclick="sortTable(1)">Rfam</th>
        <th onclick="sortTable(2)">MW</th>
        <th onclick="sortTable(3)">Sequence</th>
        <th onclick="sortTable(4)">PDB ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://mirbase.org/hairpin/MI0007073?acc=MI0007073" target="_blank" style="color:#520049"><b>MI0007073</b></a></td>
        <td name="td1"><a href="https://rfam.org/family/RF00250" target="_blank" style="color:#520049"><b>RF00250</b></a></td>
        <td name="td2">9.31 kDa</td>
        <td name="td3">GGCAGAUCUGAGCCUGGGAGCUCUCUGCC</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1ANR" target="_blank" style="color:#520049"><b>1ANR</b></a></td>
      </tr>
	  </tbody>
  </table>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/LR06_stru_ligand.svg" alt="drawing" style="width:400px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>



<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] DNA aptamers selected against the HIV-1 trans-activation-responsive RNA element form RNA-DNA kissing complexes.</strong></font><br />
Boiziau, C., Dausse, E., Yurchenko, L., & Toulmé, J. J. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10212256/" target="_blank" style="color:#520049" >J Biol Chem, 274(18), 12730-7. (1999)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] In vitro selection identifies key determinants for loop-loop interactions: RNA aptamers selective for the TAR RNA element of HIV-1.</strong></font><br />
Ducongé F., & Toulmé, J. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10606271/" target="_blank" style="color:#520049">RNA, 5(12), 1605-14. (1999)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] NMR characterization of a kissing complex formed between the TAR RNA element of HIV-1 and a DNA aptamer.</strong></font><br />
Collin, D., van Heijenoort, C., Boiziau, C., Toulmé, J. J., & Guittet, E.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10954609/" target="_blank" style="color:#520049"> Nucleic Acids Res, 28(17), 3386–3391. (2000)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Driving in vitro selection of anti-HIV-1 TAR aptamers by magnesium concentration and temperature.</strong></font><br />
Darfeuille, F., Sekkai, D., Dausse, E., Kolb, G., Yurchenko, L., Boiziau, C., & Toulmé, J. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12052182/" target="_blank" style="color:#520049">Comb Chem High Throughput Screen, 5(4), 313-25. (2002)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Molecular dynamics reveals the stabilizing role of loop closing residues in kissing interactions: comparison between TAR-TAR* and TAR aptamer.</strong></font><br />
Beaurain, F., Di Primo, C., Toulmé, J. J., & Laguerre, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12853646/" target="_blank" style="color:#520049">Nucleic Acids Res, 31(14), 4275-84. (2003)</a>
<br />
                
<a id="ref6"></a><font><strong>[6] LNA/DNA chimeric oligomers mimic RNA aptamers targeted to the TAR RNA element of HIV-1.</strong></font><br />
Darfeuille, F., Hansen, J. B., Orum, H., Di Primo, C., & Toulmé, J. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15181175/" target="_blank" style="color:#520049">Nucleic Acids Res, 32(10), 3101–3107. (2004)</a>
<br />
                
<a id="ref7"></a><font><strong>[7] Hexitol nucleic acid-containing aptamers are efficient ligands of HIV-1 TAR RNA.</strong></font><br />
Kolb, G., Reigadas, S., Boiziau, C., van Aerschot, A., Arzumanov, A., Gait, M. J., Herdewijn, P., & Toulmé, J. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15723535/" target="_blank" style="color:#520049">Biochemistry, 44(8), 2926-33. (2005)</a>
<br />
                
<a id="ref8"></a><font><strong>[8] Bimodal loop-loop interactions increase the affinity of RNA aptamers for HIV-1 RNA structures.</strong></font><br />
Boucard, D., Toulmé, J. J., & Di Primo, C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16445294/" target="_blank" style="color:#520049">Biochemistry, 45(5), 1518-24. (2006)</a>
<br />
                
<a id="ref9"></a><font><strong>[9] SELEX and dynamic combinatorial chemistry interplay for the selection of conjugated RNA aptamers.</strong></font><br />
Bugaut, A., Toulmé, J. J., & Rayner, B.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/17312962/" target="_blank" style="color:#520049">Org Biomol Chem, 4(22), 4082-8. (2006)</a>
<br />
                
<a id="ref10"></a><font><strong>[10] NMR structure of a kissing complex formed between the TAR RNA element of HIV-1 and a LNA-modified aptamer.</strong></font><br />
Lebars, I., Richard, T., Di Primo, C., & Toulmé, J. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/17768146/" target="_blank" style="color:#520049">Nucleic Acids Res, 35(18), 6103-14. (2007)</a>
<br />
                
<a id="ref11"></a><font><strong>[11] Exploring TAR–RNA aptamer loop–loop interaction by X-ray crystallography, UV spectroscopy and surface plasmon resonance.</strong></font><br />
Lebars, I., Legrand, P., Aimé, A., Pinaud, N., Fribourg, S., & Di Primo, C.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18996893/" target="_blank" style="color:#520049">Nucleic Acids Res, 36(22), 7146-56. (2008)</a>
<br />

<a id="ref12"></a><font><strong>[12] In vitro selection of RNA aptamers derived from a genomic human library against the TAR RNA element of HIV-1.</strong></font><br />
Watrin, M., Von Pelchrzim, F., Dausse, E., Schroeder, R., & Toulmé, J. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/19496624/" target="_blank" style="color:#520049">Biochemistry, 48(26), 6278-84. (2009)</a>
<br />
                       