---
layout: post
title:  "Lys1.2minF aptamer, Lys1.2minE apatmer"
date:   2024-03-25 00:00:00
author: Yuxuan Sun, Baowei Huang
categories: Aptamer
---
<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
 .header_box {
    display: block;
    font-size: 24px;
    background-color: #ffffff;
    text-decoration: none;
    border-radius: 1px;
    width: 500px;
    border-width: 1px 1px 2px 1px;
    border-color: #ffffff #ffffff #ffffff #ffffff;
}
.blowheader_box{
    display: block;
      padding: 6px;
      font-size:20px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 1px;
      width:170px;
      height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<br>
<br>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


         
<font ><p class="header_box" id="description">Description</p></font>
<font >In 2013, Camille S Padlan and Vladimir N Malashkevich used the SELEX method to isolate the aptamer with high compatibility for the HEWL(lysozyme). At the same time, they examined its structure by X-ray diffrcation. It has a sequence length of 59. Camille S Padlan and Vladimir N Malashkevich systematically minimized an RNA aptamer (Lys1) selected against hen egg white lysozyme. The resultant 59-nucleotide compact aptamer (Lys1. 2minE) retains nanomolar binding affinity and the ability to inhibit lysozyme’s catalytic activity. Lys1. 2minE inhibits catalysis of large cell wall substrates but not catalysis of small model substrates. Then,  Camille S Padlan and Vladimir N Malashkevich shortened the terminal stem of Lys1. 2minE to four Watson-Crick base pairs and got the Lys1. 2minF. At the same time, they examined its structure by X-ray diffrcation. It has a sequence length of 45. Its structure may be an adaptable protein binding platform. It can bind lysozyme with high affinity. It inhibits lysozyme cleavage of large substrates, and it bounds aptamer has no effect on catalysis of small substrates<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<br>
<br>


<p class="header_box" id="SELEX">SELEX</p>
<p>RNA aptamers were internally labeled with [α32P]-GTP and allowed to bind to lysozyme for 1 h at room temperature. The bound complex was passed through a dual filter dot blot system attached to a vacuum manifold. The nitrocellulose membrane captures the aptamer–protein complexes while unbound aptamers are trapped by the nylon filter directly beneath. Both filters are visualized by phosphor storage imaging, and the fraction protein bound was calculated  by determining the volume of radioactivity retained on the nitrocellulose divided by the total radioactivity retained by both filters. Sedimentation velocity experiments were performed on a Beckman XL-I using a standard procedure. Determining the S20, and D20, values of the RNA aptamers under each buffer condition specified by analytical ultracentrifugation. A standard lysozyme activity assay was used to determine the enzymatic activity<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br>
<br>   



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<font><p>Here we use ribodraw to complete the figure, through the 3D structure information<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</p></font>
<font><p>Lys1.2minF: 5'-GGGCGGCUAAAGAGUGCAGAGUUACUUAGUUCACUGCAGACGCCC-3'<br>Lys1.2minE: 5'-GGGUUCAUCAGGGCUAAAGAGUGCAGAGUUACUUAGUUCACUGCAGACUUGACGAACCC-3'</p></font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/lysozyme_aptamer_2D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/2D/lysozyme_aptamer_2D2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>


<p class="blowheader_box">3D visualisation</p>             
<font>Camille S Padlan and Vladimir N Malashkevich sovled the crystal structure, at 2.68 A resolution, of an RNA aptamer bound to Lysozyme C protein has been determined. The PDB ID of this structure is 4M6D<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</font>
<div ><font ><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p></font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/Lys1.2minF_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'E',    
  start_residue_number:1, 
  end_residue_number:127, 
  color:{r:0,g:128,b:0}
},
{
  struct_asym_id: 'F', 
  start_residue_number:10, 
  end_residue_number:29, 
  color:{r:255,g:135,b:10}
},
{
  struct_asym_id: 'F', 
  start_residue_number:30, 
  end_residue_number:30, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'F', 
  start_residue_number:35, 
  end_residue_number:36, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'F', 
  start_residue_number:31, 
  end_residue_number:34, 
  color:{r:255,g:135,b:10}
},
{
  struct_asym_id: 'F', 
  start_residue_number:37, 
  end_residue_number:50, 
  color:{r:255,g:135,b:10}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/4M6D-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>
<br>

<font>Camille S Padlan and Vladimir N Malashkevich sovled the crystal structure, at 2.68 A resolution, of an RNA aptamer bound to Lysozyme C protein has been determined. The PDB ID of this structure is 4M4O<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</font>
<div ><font ><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p></font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/Lys1.2minE_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection2 {
            padding-top: 0px;
          }
          .controlsSection2 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox2 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer2{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection2">
            <button onclick="
              var selectSections2 = [
                {
  struct_asym_id: 'A',    
  start_residue_number:1, 
  end_residue_number:127, 
  color:{r:0,g:128,b:0}
},
{
  struct_asym_id: 'B', 
  start_residue_number:1, 
  end_residue_number:29, 
  color:{r:255,g:135,b:10}
},
{
  struct_asym_id: 'B', 
  start_residue_number:30, 
  end_residue_number:30, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'B', 
  start_residue_number:35, 
  end_residue_number:36, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'B', 
  start_residue_number:31, 
  end_residue_number:34, 
  color:{r:255,g:135,b:10}
},
{
  struct_asym_id: 'B', 
  start_residue_number:37, 
  end_residue_number:59, 
  color:{r:255,g:135,b:10}
}
              ]
            viewerInstance2.visual.select({ data: selectSections2, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance2.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection2">
    <!-- Molstar container -->
      <div id="myViewer2"></div>
    </div>
    <script>
      var viewerInstance2 = new PDBeMolstarPlugin();
      var options2 = {
        customData:{
        url:'/pdbfiles/4M4O-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer2 = document.getElementById('myViewer2');
      viewerInstance2.render(viewerContainer2, options2);
  window.addEventListener('load', function() {
    var colorSelectionButton2 = document.querySelector('.controlsSection2 button');
    colorSelectionButton2.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<font>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 4M6D at 2.68 Å resolution. Lysozymes(shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with Lysozymes.</font>
<br>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Lys1. 2minF_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Lys1. 2minF_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
<br>
<font >Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 4M4O at 2 Å resolution. Lysozymes(shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with Lysozymes.</font>
<br>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Lys1. 2minE_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Lys1. 2minE_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>




<font ><p class="header_box" id="ligand-recognition">Ligand information</p></font>  
  
<p class="blowheader_box">SELEX ligand</p>
<font >Camille S Padlan and Vladimir N Malashkevich distinguished structures Lys1. 2 and Lys1. 3 by trimming nucleotides predicted by each to be largely unstructured. Removing different nucleotides to test whether it can binding with lysozyme, and testing its affinity<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</font >
<br><br>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Lys1. 2minF</td>
        <td name="td1">5'-GGGCGGCUAAAGAGUGCAGAGUUACUUAGUUCACUGCAGACGCCC-3'</td>
        <td name="td3">HEWL(Hen Egg White Lysozyme)</td>
        <td name="td4">57±3nM</td>
      </tr>
      <tr>
        <td name="td0">Lys1. 2minE</td>
        <td name="td1">5'-GGGUUCAUCAGGGCUAAAGAGUGCAGAGUUACUUAGUUCACUGCAGACUUGACGAACCC-3'</td>
        <td name="td3">HEWL(Hen Egg White Lysozyme)</td>
        <td name="td4">19±2nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<font><p>Lysozymes have primarily a bacteriolytic function; those in tissues and body fluids are associated with the monocyte-macrophage system and enhance the activity of immunoagents.  Has bacteriolytic activity against M. luteus. Lysozyme C is capable of both hydrolysis and transglycosylation; it shows also a slight esterase activity.  It acts rapidly on both peptide-substituted and unsubstituted peptidoglycan, and slowly on chitin oligosaccharides.-----from uniprot</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">MW</th>
        <th onclick="sortTable(3)">Amino acids sequences</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P00698/entry" target="_blank" style="color:#520049"><b>P00698</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/panther/PTHR11407/" target="_blank" style="color:#520049"><b>PTHR11407</b></a></td>
        <td name="td2">14.31 kDa</td>
        <td name="td3">KVFGRCELAAAMKRHGLDNYRGYSLGNWVCAAKFESNFNTQATNRNTDGSTDYGILQINSRWWCNDGRTPGSRNLCNIPCSALLSSDITASVNCAKKIVSDGNGMNAWVAWRNRCKGTDVQAWIRGCRL</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/4wm6" target="_blank" style="color:#520049"><b>4wm6</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&id=9031" target="_blank" style="color:#520049"><b>9031</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/lysozyme_stru_ligand.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;">
  </div>


<p class="blowheader_box">Similar compound</p>
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">1LSG-A</td>
        <td name="td2">23</td>
        <td name="td3">1</td>
        <td name="td4">Hen Egg White Lysozyme</td>
      </tr>
      <tr>
        <td name="td1">1NF5-C</td>
        <td name="td2">20.6</td>
        <td name="td3">1.4</td>
        <td name="td4">Alpha-Lactalbumin</td>
      </tr>
      <tr>
        <td name="td1">4OWD-A</td>
        <td name="td2">8.1</td>
        <td name="td3">2.7</td>
        <td name="td4">Membrane-Bound Lytic Murein Transglycosylase f</td>
      </tr>
      <tr>
        <td name="td1">4C5F-A</td>
        <td name="td2">8</td>
        <td name="td3">2.5</td>
        <td name="td4">Membrane-Bound Lytic Murein Transglycosylase c</td>
      </tr>
      <tr>
        <td name="td1">6TA9-A</td>
        <td name="td2">7.9</td>
        <td name="td3">2.5</td>
        <td name="td4">Slt Domain-Containing Protein</td>
      </tr>
      <tr>
        <td name="td1">1QTE-A</td>
        <td name="td2">7.9</td>
        <td name="td3">2.6</td>
        <td name="td4">Soluble Lytic Transglycosylase Slt70</td>
      </tr>
      <tr>
        <td name="td1">3BKH-A</td>
        <td name="td2">7.8</td>
        <td name="td3">2.8</td>
        <td name="td4">Lytic Transglycosylase</td>
      </tr>
      <tr>
        <td name="td1">5E27-A</td>
        <td name="td2">7.8</td>
        <td name="td3">2.5</td>
        <td name="td4">Resuscitation-Promoting Factor Rpfb</td>
      </tr>
      <tr>
        <td name="td1">4YIM-B</td>
        <td name="td2">7.7</td>
        <td name="td3">2.8</td>
        <td name="td4">Putative Soluble Lytic Murein Transglycosylase</td>
      </tr>
      <tr>
        <td name="td1">7LAM-A</td>
        <td name="td2">7.5</td>
        <td name="td3">3.1</td>
        <td name="td4">Lytic Transglycosylase Domain-Containing Protein</td>
      </tr>
	  </tbody>
  </table>
<br>
<br>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] An RNA aptamer possessing a novel monovalent cation-mediated fold inhibits lysozyme catalysis by inhibiting the binding of long natural substrates.</strong></font><br />
Padlan, C. S., Malashkevich, V. N., Almo, S. C., Levy, M., Brenowitz, M., & Girvin, M. E.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/24570482/" target="_blank" style="color:#520049" >RNA, 20(4), 447–461. (2013)</a>
<br />
                
