---
layout: post
title:  "NF-kappaB aptamer"
date:   2024-03-01 00:00:00
author: Zhizhong Lu, Ying Ao
categories: Aptamer
---
<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
 .header_box {
    display: block;
    font-size: 24px;
    background-color: #ffffff;
    text-decoration: none;
    border-radius: 1px;
    width: 500px;
    border-width: 1px 1px 2px 1px;
    border-color: #ffffff #ffffff #ffffff #ffffff;
}
.blowheader_box{
    display: block;
      padding: 6px;
      font-size:20px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 1px;
      width:170px;
      height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<br>
<br>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>


<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10074372/" target="_blank" style="color:#520049">1999</a></h3>
    </div>
    <div class="body">
      <p>L J Maher 3rd isolated a small RNA aptamer that binds with nanomolar affinity to human transcription factor NF-kappa B in vitro selection<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12886018/" target="_blank" style="color:#520049">2003</a></h3>
    </div>
    <div class="body">
      <p>Described here the 2.45-A resolution x-ray crystal structure of the p50 RHR/RNA aptamer complex<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12637683/" target="_blank" style="color:#520049">2003</a></h3>
    </div>
    <div class="body">
      <p>Yeast genetic selections were used to optimize the RNA aptamer for binding to NF-kappaB in the eukaryotic nucleus, RNA aptamers with dramatically improved in vivo activity<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/15263062/" target="_blank" style="color:#520049">2004</a></h3>
    </div>
    <div class="body">
      <p>The first example of a post-modification of an NF-kappaB RNA aptamer by 4'-thioribonucleoside units<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/16517938/" target="_blank" style="color:#520049">2006</a></h3>
    </div>
    <div class="body">
      <p>The use of aptamers and siRNA together can be the most effective way to achieve maximal knock-down of protein activity<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/18160411/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>The high affinity of the NF-kappaB transcription factor for this RNA aptamer may largely be due to the structural pre-organization of the RNA that results in its ability to mimic DNA<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/18426920/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>In vitro selection of anti-NF-kappaB p65 RNA aptamers using parallel in vitro selections with either a fully randomized RNA library<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/19696077/" target="_blank" style="color:#520049">2009</a></h3>
    </div>
    <div class="body">
      <p>Described the further characterization of anti-NF-kappaB p50 and p65 RNA aptamers in vitro and in vivo<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup></p>
    </div>
  </div>
</div>


         
<font><p class="header_box" id="description">Description</p></font>
<font>In 1999, L J Maher 3rd isolated an aptamer that binds NF-kappaB p50, and its crystal structure was cracked in 2003. The results of competition experiments demonstrate that binding of the RNA aptamer blocks the ability of NF-kappa B to bind duplex DNA. Expression of this aptamer structure within heterologous nuclear RNA transcripts may provide a new strategy to inhibit NF-kappa B function in vivo. In 2008, He selected the RNA aptamer of anti-NF-kappaB p65. And reported the characterization of these aptamers with respect to NF-kappaB target specificity, affinity, minimal sequence requirements, secondary structure, and competition with DNA kappaB sites<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref2" style="color:#520049">2</a></sup><sup>,<a href="#ref7" style="color:#520049">7</a></sup><sup>]</sup>.<br></font>
<br>
<br>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 1999, L J Maher 3rd identified RNA aptamers capable of tightly binding p50<sub>2</sub> using cycles of in vitro affinity selection and amplification. The initial RNA pool represented ∼10<sup>14</sup> different sequences consisting of a core of 60 random nucleotides flanked. Fourteen cycles of selection and amplification were performed,a dominant RNA was detected as 21 of 26 cloned and sequenced cDNAs<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br>
<br>    



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<font><p>Here we used ribodraw to complete the figure, through the 3D structure information<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</p></font>
<font><p>5'-CAUACUUGAAACUGUAAGGUUGGCGUAUG-3'</p></font>
<img src="/images/2D/NF-kB_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br> 


<p class="blowheader_box">3D visualisation</p>             
<font >L. J., 3rd, & Ghosh, G. et al. sovled the 2.45-Å resolution x-ray crystal structure of the p50 RHR/RNA aptamer complex. The structural analysis indicates that two RNA molecules bind autonomously to the N-terminal Ig-like domains of p50. The RNA secondary structure comprises a stem and a stem-loop motif, with an internal loop adopting a kinked helical conformation due to the stacking interactions between three guanine residues within the loop. The PDB ID of this structure is 1OOA<sup>[<a href="#ref2" style="color:#520049">2</a></sup><sup>]</sup>.<br>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</font>
<div ><font ><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p></font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/NF-kB_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A', 
  start_residue_number:38, 
  end_residue_number:350, 
  color:{r:0,g:255,b:0}
},
{
  struct_asym_id: 'C', 
  start_residue_number:1, 
  end_residue_number:10, 
  color:{r:233,g:150,b:122}
},
{
  struct_asym_id: 'C', 
  start_residue_number:15, 
  end_residue_number:17, 
  color:{r:233,g:150,b:122}
},
{
  struct_asym_id: 'C', 
  start_residue_number:24, 
  end_residue_number:29, 
  color:{r:233,g:150,b:122}
},
{
  struct_asym_id: 'C', 
  start_residue_number:11, 
  end_residue_number:14, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'C', 
  start_residue_number:18, 
  end_residue_number:23, 
  color:{r:255,g:0,b:255}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1ooa-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<font >Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 1OOA at 2.45 Å resolution. NF-kB(p50)  protein (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with NF-kB(p50) protein.</font>
<br><br>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/NF-kB_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/NF-kB_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>



<font ><p class="header_box" id="ligand-recognition">Ligand information</p></font>  
<p class="blowheader_box">SELEX ligand</p>
<font >Nitrocellulose filter binding experiments were employed to estimate the equilibrium dissociation constants of different RNA aptamer/NF-κB complexes. Binding reactions containing variable concentrations of NF-κB protein were assembled in binding buffer with a known concentration (typically 0.85 nM) of end-labeled RNA aptamer. Binding reactions were incubated for 10 min at room temperature and then filtered over 0.45 µm nitrocellulose filters ( Millipore catalog no. HAWP02500).  Filters were washed with binding buffer, and radioactivity was determined by scintillation counting. p50dd: p50 dimerization domain. p65dd: p65 dimerization domain<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</font >
<br><br>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">α-p50 RNA aptamer</td>
        <td name="td1">CAUACUUGAAACUGUAAGGUUGGCGUAUG</td>
        <td name="td3">NF-κB (p50)<sub>2</sub></td>
        <td name="td4">5.4±2.2 nM</td>
      </tr>
      <tr>
        <td name="td0">α-p50 RNA aptamer</td>
        <td name="td1">CAUACUUGAAACUGUAAGGUUGGCGUAUG</td>
        <td name="td3">p50/p65</td>
        <td name="td4">22±7 nM</td>
      </tr>
      <tr>
        <td name="td0">α-p50 RNA aptamer</td>
        <td name="td1">CAUACUUGAAACUGUAAGGUUGGCGUAUG</td>
        <td name="td3">(p65)<sub>2</sub></td>
        <td name="td4">3500±2300 nM</td>
      </tr>
      <tr>
        <td name="td0">α-p50 RNA aptamer</td>
        <td name="td1">CAUACUUGAAACUGUAAGGUUGGCGUAUG</td>
        <td name="td3">p50dd/p65</td>
        <td name="td4">187±155 nM</td>
      </tr>
      <tr>
        <td name="td0">α-p50 RNA aptamer</td>
        <td name="td1">CAUACUUGAAACUGUAAGGUUGGCGUAUG</td>
        <td name="td3">p50/p65dd</td>
        <td name="td4">10±4 nM</td>
      </tr>
      <tr>
        <td name="td0">α-p50 RNA aptamer</td>
        <td name="td1">CAUACUUGAAACUGUAAGGUUGGCGUAUG</td>
        <td name="td3">p50dd/p65dd</td>
        <td name="td4">No binding</td>
      </tr>
	  </tbody>
  </table>
  </div>

<p class="blowheader_box">Structure ligand</p>
<font><p>This entry represents the N-terminal sub-domain of the Rel homology domain (RHD) of NF-kappaB subunit precursor p105, which can undergo cotranslational processing by the 26S proteasome to produce a 50kDa protein (p50). p50 is a DNA binding subunit of the NF-kappaB protein complex.-----From Pfam</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(6)">Name</th>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">MW</th>
        <th onclick="sortTable(3)">Amino acids sequences</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td6">NF-kappaB p50 protein</td>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P25799/entry" target="_blank" style="color:#520049"><b>P25799</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR030503/" target="_blank" style="color:#520049"><b>IPR030503</b></a></td>
        <td name="td2">36.61 kDa</td>
        <td name="td3">GGPYLQILEQPKQRGFRFRYVCEGPSHGGLPGASSEKNKKSYPQVKICNYVGPAKVIVQLVTNGKNIHLHAHSLVGKHCEDGVCTVTAGPKDMVVGFANLGILHVTKKKVFETLEARMTEACIRGYNPGLLVHSDLAYLQAEGGGDRQLTDREKEIIRQAAVQQTKEMDLSVVRLMFTAFLPDSTGSFTRRLEPVVSDAIYDSKAPNASNLKIVRMDRTAGCVTGGEEIYLLCDKVQKDDIQIRFYEEEENGGVWEGFGDFSPTDVHRQFAIVFKTPKYKDVNITKPASVFVQLRRKSDLETSEPKPFLYYPEIKDKEEVQRKRQK</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/1OOA" target="_blank" style="color:#520049"><b>1OOA</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&id=10090" target="_blank" style="color:#520049"><b>10090</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/NF-kB_stru_ligand.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">1OY3-C</td>
        <td name="td2">20.2</td>
        <td name="td3">1.0 </td>
        <td name="td4">Transcription factor p65</td>
      </tr>
      <tr>
        <td name="td1">2O61-A</td>
        <td name="td2">20</td>
        <td name="td3">6.1 </td>
        <td name="td4">36-mer</td>
      </tr>
      <tr>
        <td name="td1">8OW4-B</td>
        <td name="td2">16</td>
        <td name="td3">2.7 </td>
        <td name="td4">Nuclear factor of activated t-cells,cytoplasmic</td>
      </tr>
      <tr>
        <td name="td1">2CXK-A</td>
        <td name="td2">10.7</td>
        <td name="td3">1.9 </td>
        <td name="td4">Calmodulin binding transcription activator 1</td>
      </tr>
      <tr>
        <td name="td1">5L56-A</td>
        <td name="td2">10.5</td>
        <td name="td3">25.0 </td>
        <td name="td4">Plexin-a1</td>
      </tr>
      <tr>
        <td name="td1">6VXK-B</td>
        <td name="td2">10.4</td>
        <td name="td3">22.2 </td>
        <td name="td4">Semaphorin-like protein 139</td>
      </tr>
      <tr>
        <td name="td1">3HRP-A</td>
        <td name="td2">10.2</td>
        <td name="td3">2.6 </td>
        <td name="td4">Uncharacterized protein</td>
      </tr>
      <tr>
        <td name="td1">3MLP-A</td>
        <td name="td2">10</td>
        <td name="td3">10.3 </td>
        <td name="td4">Transcription factor coe1</td>
      </tr>
      <tr>
        <td name="td1">6STX-C</td>
        <td name="td2">9.9</td>
        <td name="td3">8.3 </td>
        <td name="td4">Kelch domain-containing protein</td>
      </tr>
      <tr>
        <td name="td1">7MO7-B</td>
        <td name="td2">9.6</td>
        <td name="td3">11.3 </td>
        <td name="td4">Hepatocyte growth factor</td>
      </tr>
	  </tbody>
  </table>
<br>
<br>



                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Selection and characterization of an RNA decoy for transcription factor NF-kappa B.</strong></font><br />
Lebruska, L. L., & Maher, L. J., 3rd.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10074372/" target="_blank" style="color:#520049" >Biochemistry, 38(10), 3168–3174. (1999)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Crystal structure of NF-kappaB (p50)2 complexed to a high-affinity RNA aptamer.</strong></font><br />
Huang, D. B., Vu, D., Cassiday, L. A., Zimmerman, J. M., Maher, L. J., 3rd, & Ghosh, G.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12886018/" target="_blank" style="color:#520049">Proceedings of the National Academy of Sciences of the United States of America, 100(16), 9268–9273. (2003)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Yeast genetic selections to optimize RNA decoys for transcription factor NF-kappa B.</strong></font><br />
Cassiday, L. A., & Maher, L. J., 3rd.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12637683/" target="_blank" style="color:#520049"> Proceedings of the National Academy of Sciences of the United States of America, 100(7), 3930–3935. (2003)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Synthesis and physical and physiological properties of 4'-thioRNA: application to post-modification of RNA aptamer toward NF-kappaB.</strong></font><br />
Hoshika, S., Minakawa, N., & Matsuda, A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15263062/" target="_blank" style="color:#520049">Nucleic acids research, 32(13), 3815–3825. (2004)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Co-expression of anti-NFkappaB RNA aptamers and siRNAs leads to maximal suppression of NFkappaB activity in mammalian cells.</strong></font><br />
Chan, R., Gilbert, M., Thompson, K. M., Marsh, H. N., Epstein, D. M., & Pendergrast, P. S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16517938/" target="_blank" style="color:#520049">Nucleic acids research, 34(5), e36. (2006)</a>
<br />

<a id="ref6"></a><font><strong>[6] DNA mimicry by a high-affinity anti-NF-kappaB RNA aptamer.</strong></font><br />
Reiter, N. J., Maher, L. J., 3rd, & Butcher, S. E.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18160411/" target="_blank" style="color:#520049">Nucleic acids research, 36(4), 1227–1236. (2008)</a>
<br />   

<a id="ref7"></a><font><strong>[7] Selection and characterization of anti-NF-kappaB p65 RNA aptamers.</strong></font><br />
Wurster, S. E., & Maher, L. J., 3rd.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18426920/" target="_blank" style="color:#520049">RNA (New York, N.Y.), 14(6), 1037–1047. (2008)</a>
<br /> 

<a id="ref8"></a><font><strong>[8] Characterization of anti-NF-kappaB RNA aptamer-binding specificity in vitro and in the yeast three-hybrid system.</strong></font><br />
Wurster, S. E., Bida, J. P., Her, Y. F., & Maher, L. J., 3rd.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/19696077/" target="_blank" style="color:#520049">Nucleic acids research, 37(18), 6214–6224. (2009)</a>
<br /> 
