---
layout: theophylline_style
title: No.21 aptamer
date: 2024-06-05 00:00:00
author: Yuxuan Sun, Baowei Huang
categories: Aptamer
tags:
- Category:Amino Acids & Peptides
- Category:Bacteria
- Category:Biologics & Signaling Molecules
- Category:Fluorophores
- Category:Mammalian
- Category:Nucleosides & Nucleotides
- Category:Other
- Category:Pharmaceuticals
- Category:Prion
- Category:Synthetic
- Category:Viral
- Category:cell
- GC:39.0
- GC:40.0
- GC:40.18
- GC:40.82
- GC:40.849999999999994
- GC:42.370000000000005
- GC:43.120000000000005
- GC:43.71
- GC:43.75
- GC:44.04
- GC:44.3
- GC:44.440000000000005
- GC:44.64
- GC:45.07
- GC:45.45
- GC:45.5
- GC:45.57
- GC:45.68
- GC:45.87
- GC:46.0
- GC:46.25
- GC:46.339999999999996
- GC:46.46
- GC:46.51
- GC:46.72
- GC:46.77
- GC:46.85
- GC:47.0
- GC:47.17
- GC:47.22
- GC:47.5
- GC:47.54
- GC:47.89
- GC:48.0
- GC:48.06
- GC:48.08
- GC:48.15
- GC:48.57
- GC:48.6
- GC:48.65
- GC:48.72
- GC:48.75
- GC:48.84
- GC:49.09
- GC:49.3
- GC:49.51
- GC:49.62
- GC:49.69
- GC:50.0
- GC:50.470000000000006
- GC:50.480000000000004
- GC:50.7
- GC:51.160000000000004
- GC:51.28
- GC:51.32
- GC:51.49
- GC:51.519999999999996
- GC:51.849999999999994
- GC:51.949999999999996
- GC:52.0
- GC:52.11
- GC:52.21
- GC:52.22
- GC:52.33
- GC:52.5
- GC:52.81
- GC:53.0
- GC:53.33
- GC:53.410000000000004
- GC:53.52
- GC:53.620000000000005
- GC:53.949999999999996
- GC:54.0
- GC:54.410000000000004
- GC:54.55
- GC:54.64
- GC:55.169999999999995
- GC:55.559999999999995
- GC:55.67999999999999
- GC:55.75
- GC:55.910000000000004
- GC:56.00000000000001
- GC:56.06
- GC:56.10000000000001
- GC:56.18
- GC:56.34
- GC:56.52
- GC:56.67
- GC:56.96
- GC:56.99999999999999
- GC:57.14
- GC:57.89
- GC:57.99999999999999
- GC:58.330000000000005
- GC:58.42
- GC:58.43000000000001
- GC:58.440000000000005
- GC:58.46
- GC:58.81999999999999
- GC:59.0
- GC:59.040000000000006
- GC:59.4
- GC:59.68
- GC:59.78
- GC:60.0
- GC:60.24
- GC:60.4
- GC:60.49
- GC:60.760000000000005
- GC:61.0
- GC:61.33
- GC:61.53999999999999
- GC:62.96000000000001
- GC:63.0
- GC:63.04
- GC:63.21
- GC:63.89
- GC:64.0
- GC:64.1
- GC:65.79
- GC:66.0
- GC:67.0
- GC:67.39
- GC:69.0
- GC:69.37
- Length:10
- Length:100
- Length:101
- Length:103
- Length:104
- Length:105
- Length:106
- Length:107
- Length:108
- Length:109
- Length:110
- Length:111
- Length:112
- Length:113
- Length:118
- Length:12
- Length:122
- Length:124
- Length:129
- Length:131
- Length:132
- Length:133
- Length:150
- Length:152
- Length:159
- Length:162
- Length:167
- Length:18
- Length:20
- Length:24
- Length:28
- Length:33
- Length:35
- Length:36
- Length:37
- Length:38
- Length:39
- Length:40
- Length:41
- Length:43
- Length:45
- Length:46
- Length:47
- Length:48
- Length:51
- Length:54
- Length:55
- Length:57
- Length:58
- Length:60
- Length:62
- Length:63
- Length:65
- Length:68
- Length:69
- Length:70
- Length:71
- Length:72
- Length:75
- Length:76
- Length:77
- Length:78
- Length:79
- Length:80
- Length:81
- Length:82
- Length:83
- Length:84
- Length:86
- Length:87
- Length:88
- Length:89
- Length:90
- Length:91
- Length:92
- Length:93
- Length:97
- Length:98
- Length:99
- Named:3',5'-cyclic adenosine monophosphate (cAMP)_cAMP - b (parent)
- Named:ACE_Okra aptamer
- Named:AMPA and kainate receptors_AB9
- Named:AMPA and kainate receptors_AB9s
- Named:Adenosine triphosphate (ATP)_ATP binding aptamer
- Named:Adenosine triphosphate (ATP)_V2 Hammerhead Ribozyme
- Named:Adenosine triphosphate (ATP)_V2 trans Hammerhead Ribozyme
- Named:Adenosine triphosphate (ATP)_aptRNA1
- Named:Adenosine triphosphate (ATP)_pRNAwt
- 'Named:Alpha-Amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) receptors_Aptamer
  #25'
- Named:Amyloid beta-peptide (Aβ)_E2
- Named:Amyloid beta-peptide (Aβ)_N2
- Named:Basic fibroblast growth factor (bFGF)_21A
- Named:Bovine prion protein_R12 aptamer
- Named:Cibacron Blue 3GA_CB-42
- Named:Cytoplasmic tail of BACE (B1-CT)_S10
- Named:Cytoplasmic tail of BACE (B1-CT)_TH14
- Named:DFHO_Corn aptamer
- Named:DNA polymerase beta (polβ)_Aptamer 25
- Named:DNA polymerase beta (polβ)_Aptamer 31
- Named:DNA polymerase beta (polβ)_Aptamer 32
- Named:DNA polymerase beta (polβ)_Aptamer 45
- Named:DNA polymerase beta (polβ)_Aptamer 53
- Named:DNA polymerase kappa (polκ)_Aptamer 25
- Named:DNA polymerase kappa (polκ)_Aptamer 32
- Named:DNA polymerase kappa (polκ)_Aptamer 45
- Named:Drosophila muscleblind protein (Mbl)_RNA 20
- Named:E2F transcription factor (E2F1)_E1 RNA aptamer
- Named:E2F transcription factor (E2F1)_E2F-E1
- Named:Epithelial Growth Factor Receptor (EGFR)_J18
- Named:Escherichia coli release factor 1_Class II-1
- Named:Estrogen receptor α (ERα)_AptER-1
- Named:Estrogen receptor α (ERα)_AptER-2
- Named:Estrogen receptor α (ERα)_AptER-3
- Named:Eukaryotic translation initiation factor 4A (eIF4A)_#1
- Named:Eukaryotic translation initiation factor 4A (eIF4A)_no.21
- Named:Eukaryotic translation initiation factor 4E (eIF4E)_Aptamer 1
- Named:Eukaryotic translation initiation factor 4E (eIF4E)_Aptamer 2
- Named:Extracellular domains of HER3_A30
- Named:Flavin adenine dinucleotide (FAD), Flavin mononucleotide(FMN), Riboflavin(Rb)_X2B2-C14U
  aptamer
- Named:Flavin mononucleotide (FMN)_35 nucleotide RNA aptamer
- Named:Fluorescein_FB-1
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.18 aptamer
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.20 aptamer
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.28 aptamer
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.51
- Named:Gemini-561o-Coral
- Named:GluR2 AMPA receptors_AN58
- Named:Guanosine triphosphate (GTP)_Aptamer 10-10
- Named:Guanosine triphosphate (GTP)_Aptamer 10-24
- Named:Guanosine triphosphate (GTP)_Aptamer 10-59
- Named:Guanosine triphosphate (GTP)_Aptamer 10-6
- Named:Guanosine triphosphate (GTP)_Aptamer 9-12
- Named:Guanosine triphosphate (GTP)_Class I
- Named:Guanosine triphosphate (GTP)_Class I aptamer(truncation)
- Named:HER2-positive breast cancer cells_TSA12 aptamer
- Named:HER2-positive breast cancer cells_TSA14
- Named:HER2-positive breast cancer cells_TSA14 aptamer(truncation)
- Named:HIV-1 Rev protein_1
- Named:HIV-1 Rev protein_2
- Named:HIV-1 Tat protein_clone 31
- Named:Haemagglutinin (HA) region of the target strain A/Panama/2007/1999(H3N2)_P30-10-16
- Named:Hfq (host factor required for phage Qβ RNA replication)_AGr aptamer
- Named:Human T-cell lymphomatic virus (HTLV-1) tax protein_YT1
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C1
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C15
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C17
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C18
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C24
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C33
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C34
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C37
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C5
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C52
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C8
- Named:Human immunodeficiency virus type 1 Rev (HIV-1 Rev) protein_C9
- Named:Human immunoglobulin G_apt2 aptamer
- Named:L-tyrosine_Tyr 1
- Named:Malachite green (MG),Tetramethylrhodamine (TMR)_MG aptamer
- Named:Malachite green (MG)MG4
- Named:Monomeric beta-2-microglobulin (β2m) fibrils_LS-5
- Named:Monomeric beta-2-microglobulin (β2m) fibrils_M-2
- Named:Monomeric beta-2-microglobulin (β2m) fibrils_WL-2
- Named:NF-kappaB P50_RNA aptamer 3
- Named:NS3 Protein of Hepatitis C Virus (HCV)_G6–16
- Named:NS3 Protein of Hepatitis C Virus (HCV)_G6–19
- Named:NS3 protease of hepatitis C virus (HCV)_10G-1
- Named:Neuropeptide Y (NPY)_DP3
- Named:Nonstructural 5B (NS5B) polymerase, hepatitis C virus (HCV) NS5BΔC55 protein_aptamer
  A.2
- Named:Nonstructural 5B (NS5B) polymerase, hepatitis C virus (HCV) NS5BΔC55 protein_aptamer
  B.2
- 'Named:Nuclear factor of activated T cells (NFAT) DNA binding domain_RNA aptamer
  #1'
- 'Named:Nuclear factor of activated T cells (NFAT) DNA binding domain_RNA aptamer
  #2'
- Named:OX40_11F11
- Named:OX40_9C7
- Named:OX40_9C7 aptamer (truncation)
- Named:Prostate cancer cell_E3 aptamer
- Named:Prostate cancer cell_E3 aptamer (truncation)
- Named:Prostate-specific membrane antigen (PSMA)_A9g aptamer
- Named:Protein kinase C beta II_PKC 10
- Named:Protein kinase C beta II_PKC 6
- Named:Ras-binding domain (Raf-1 RBD)_21.01
- Named:Ras-binding domain (Raf-1 RBD)_21.07
- Named:Reactive Blue 4_B4-25
- Named:Receptor activator of NF-kappaB (RANK)_apt1
- Named:Receptor activator of NF-kappaB (RANK)_apt2
- Named:Receptor activator of NF-kappaB (RANK)_apt3
- Named:Ribosomal protein S8_RNA-2 aptamer
- Named:SARS-CoV nucleocapsid (N) protein_aptamer 1
- Named:SARS-CoV nucleocapsid (N) protein_aptamer 2
- 'Named:SR Protein B52 (SRp55)_BBS #11'
- 'Named:SR Protein B52 (SRp55)_BBS #23'
- 'Named:SR Protein B52 (SRp55)_BBS #4,14,15'
- 'Named:SR Protein B52 (SRp55)_BBS #8'
- Named:Serine protease inhibitor plasminogen activator inhibitor-1 (PAI-1)_SM-20
- Named:Serine protease inhibitor plasminogen activator inhibitor-1 (PAI-1)_WT-15
- Named:Sheep prion protein_RM312 aptamer
- Named:Sphingosylphosphorylcholine (SPC)_C1
- Named:Sphingosylphosphorylcholine (SPC)_C2
- Named:Sphingosylphosphorylcholine (SPC)_C8
- Named:Sphingosylphosphorylcholine (SPC)_m009
- Named:Sphingosylphosphorylcholine (SPC)_m010
- Named:Sphingosylphosphorylcholine (SPC)_m011
- Named:Sphingosylphosphorylcholine (SPC)_m012
- Named:Sulforhodamine B_SRB-2
- Named:T cytokine 1 (TCF-1)_#10
- Named:TFKBP*3R-2G (cell surface protein)_T1 s23
- Named:TFKBP*3R-2G (cell surface protein)_T2 s23
- Named:TFKBP*3R-2G (cell surface protein)_s23
- Named:TMR (Tetramethylrhodamine)TMR1
- Named:TMR (Tetramethylrhodamine)TMR2
- Named:TMR (Tetramethylrhodamine)TMR3
- Named:TMR (Tetramethylrhodamine)TMR4
- Named:TMR (Tetramethylrhodamine)_TMR3 aptamer
- Named:TMR-DN, Pyronin B, ATTO495SRB-3
- Named:Tetracycline_cb28 aptamer
- Named:The protein calsenilin_aptamer-12
- Named:The protein calsenilin_aptamer-2
- Named:Tobramycin_J6f1
- Named:Triple-negative breast cancer (TNBC)_TN145
- Named:Triple-negative breast cancer (TNBC)_TN2
- Named:Triple-negative breast cancer (TNBC)_TN20
- Named:Triple-negative breast cancer (TNBC)_TN29
- Named:Triple-negative breast cancer (TNBC)_TN3
- Named:Triple-negative breast cancer (TNBC)_TN58
- Named:Tumor necrosis factor alpha (hTNFα)_1
- Named:Tumor necrosis factor alpha (hTNFα)_10
- Named:Tumor necrosis factor alpha (hTNFα)_11.7
- Named:Tumor necrosis factor alpha (hTNFα)_12
- Named:Tumor necrosis factor alpha (hTNFα)_13.2
- Named:Tumor necrosis factor alpha (hTNFα)_14.8
- Named:Tumor necrosis factor alpha (hTNFα)_15
- Named:Tumor necrosis factor alpha (hTNFα)_16.3
- Named:Tumor necrosis factor alpha (hTNFα)_17.9
- Named:Tumor necrosis factor alpha (hTNFα)_18.1
- Named:Tumor necrosis factor alpha (hTNFα)_2
- Named:Tumor necrosis factor alpha (hTNFα)_20.4
- Named:Tumor necrosis factor alpha (hTNFα)_3.1
- Named:Tumor necrosis factor alpha (hTNFα)_4
- Named:Tumor necrosis factor alpha (hTNFα)_6
- Named:Tumor necrosis factor alpha (hTNFα)_7.5
- Named:Tumor necrosis factor alpha (hTNFα)_8.6
- Named:Tumor necrosis factor alpha (hTNFα)_9
- Named:Tumor necrosis factor alpha (hTNFα)_T1.7.5
- Named:Tumor necrosis factor alpha (hTNFα)_T4.14.8
- Named:Vascular Endothelial Growth Factor (VEGF)_NX-213 aptamer
- Named:Zinc-finger antiviral protein (ZAP)_21-E
- Named:Zinc-finger antiviral protein (ZAP)_21-K
- Named:p-amino phenylalanine (pAF)pAF-4Z1
- Named:p-amino phenylalanine (pAF)pAF-R1-1
- Named:β-amyloid 42 (Aβ42)_E22P-AbD31
- Named:β-amyloid 42 (Aβ42)_E22P-AbD4
- Named:β-amyloid 42 (Aβ42)_E22P–AbD43
- Type:Cells
- Type:Proteins
- Type:Small molecules
- Year:1990
- Year:1993
- Year:1994
- Year:1995
- Year:1996
- Year:1997
- Year:1998
- Year:1999
- Year:2000
- Year:2001
- Year:2002
- Year:2003
- Year:2004
- Year:2005
- Year:2006
- Year:2007
- Year:2008
- Year:2009
- Year:2010
- Year:2012
- Year:2013
- Year:2014
- Year:2015
- Year:2017
- Year:2018
- Year:2020
- Year:2022
- Year:2024
type: Small molecules
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12649492/" target="_blank" style="color:#520049">2003</a></h3>
  </div>
  <div class="body">
    <p>Oguro A et al.generated RNA aptamers with high affinity for eIF4A<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/16940549/" target="_blank" style="color:#520049">2006</a></h3>
  </div>
  <div class="body">
    <p>Miyakawa S et al. isolated eight different RNA aptamers with high affinity to mammalian eIF4G<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
  </div>
 </div>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2003, Akihiro Oguro and Takashi Ohtsu had used the SELEX  methodology to select RNA aptamers with high affinity for eIF4A by in vitro RNA selection-amplification<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2003, Akihiro Oguro and colleagues used SELEX. Four RNA selection experiments were performed using fulllength eIF4A and RNA pools of either 30 or 40 random nucleotide positions. Then, from each selection, 48 RNA sequences were cloned, aptamers that bind specifically to eIF4A were selected with high affinity<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figures is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>5'-GGGAGACAAGAAUAAACGCUCAAGGGGACCGCGCCCCACAUGUGAGUGAGGCCGAAACAUAGAUUCGACAGGAGGCUCACAACAGGC-3'</p>
<img src="/images/2D/no.21_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>The DEAD-box helicase domain found in eIF4A and related proteins are DEAD-box helicases, a diverse family of proteins involved in ATP-dependent RNA unwinding, needed in a variety of cellular processes including splicing, ribosome biogenesis and RNA degradation. Eukaryotic initiation factor 4A-I (DDX2A) is involved in cap recognition and is required for mRNA binding to ribosome. This domain contains the ATP-binding region.-----From Pfam</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">eIF4A</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P60842/entry" target="_blank" style="color:#520049"><b>P60842</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR044728/" target="_blank" style="color:#520049"><b>IPR044728</b></a></td>
        <td name="td3">46.154 kDA</td>
        <td name="td4">MSASQDSRSRDNGPDGMEPEGVIESNWNEIVDSFDDMNLSESLLRGIYAYGFEKPSAIQQRAILPCIKGYDVIAQAQSGTGKTATFAISILQQIELDLKATQALVLAPTRELAQQIQKVVMALGDYMGASCHACIGGTNVRAEVQKLQMEAPHIIVGTPGRVFDMLNRRYLSPKYIKMFVLDEADEMLSRGFKDQIYDIFQKLNSNTQVVLLSATMPSDVLEVTKKFMRDPIRILVKKEELTLEGIRQFYINVEREEWKLDTLCDLYETLTITQAVIFINTRRKVDWLTEKMHARDFTVSAMHGDMDQKERDVIMREFRSGSSRVLITTDLLARGIDVQQVSLVINYDLPTNRENYIHRIGRGGRFGRKGVAINMVTEEDKRTLRDIETFYNTSIEEMPLNVADLI</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/2G9N" target="_blank" style="color:#520049"><b>2G9N</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/1973" target="_blank" style="color:#520049"><b>1973</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
      <td name="td0">no.21 aptamer</td>
      <td name="td1">5'-GGGAGACAAGAAUAAACGCUCAAGGGGACCGCGCCCCACAUGUGAGUGAGGCCGAAACAUAGAUUCGACAGGAGGCUCACAACAGGC-3'</td>
      <td name="td2">eIF4A</td>
      <td name="td3">27nM</td>
    </tr>
            
     <tr>
      <td name="td0">#11 aptamer</td>
      <td name="td1">5'-GGGAGACAAGAAUAAAACGCUCAAACAUUGCAUCGACAGCUGCAAGGCUCCCGCCGUACAAACCUUCGACAGGAGGCUCACAACAGGC-3'</td>
      <td name="td2">eIF4A</td>
      <td name="td3">8 μM</td>
    </tr>
            
     <tr>
      <td name="td0">#1 aptamer</td>
      <td name="td1">5'-GGGAGACAAGAAUAAACGCUCAACAGGCGUUUAGCCUCUAAGUAACAGGGGCCUCCCAUGAGCUUCGACAGGAGGCUCACAACAGGC-3'</td>
      <td name="td2">eIF4A</td>
      <td name="td3">3 μM</td>
    </tr>
            
     <tr>
      <td name="td0">#20 aptamer</td>
      <td name="td1">5'-GGGAGACAAGAAUAAACGCUCAAGGGGACCGCGCCCCACAUGUGAGUGAGGCCGAAACGUAGAUUCGACAGGAGGCUCACAACAGGC-3'</td>
      <td name="td2">eIF4A</td>
      <td name="td3">44 nM</td>
    </tr>
            
     <tr>
      <td name="td0">#30 aptamer</td>
      <td name="td1">5'-GGGAGACAAGAAUAAACGCUCAAUGUGGAUGAUUUGUAUGAUCGCGCAUACAAUUCGACAGGAGGCUCACAACAGGC-3'</td>
      <td name="td2">eIF4A</td>
      <td name="td3">1 μM</td>
    </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/no.21_aptamer_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">8RC0-D</td>
      <td name="td1">27.8</td>
      <td name="td2">1.9</td>
      <td name="td3">Cd2 Antigen Cytoplasmic Tail-Binding Protein 2</td>
    </tr>
            
     <tr>
      <td name="td0">3I5X-A</td>
      <td name="td1">25.0</td>
      <td name="td2">2.3</td>
      <td name="td3">Atp-Dependent Rna Helicase Mss116</td>
    </tr>
            
     <tr>
      <td name="td0">7NAD-X</td>
      <td name="td1">23.3</td>
      <td name="td2">1.9</td>
      <td name="td3">25S Rrna</td>
    </tr>
            
     <tr>
      <td name="td0">6ZNP-A</td>
      <td name="td1">19.1</td>
      <td name="td2">2.7</td>
      <td name="td3">Uncharacterized Atp-Dependent Helicase Ypra</td>
    </tr>
            
     <tr>
      <td name="td0">5V9X-A</td>
      <td name="td1">18.8</td>
      <td name="td2">2.7</td>
      <td name="td3">Atp-Dependent Dna Helicase</td>
    </tr>
            
     <tr>
      <td name="td0">8VX9-B</td>
      <td name="td1">18.5</td>
      <td name="td2">2.9</td>
      <td name="td3">Hama</td>
    </tr>
            
     <tr>
      <td name="td0">4CGZ-A</td>
      <td name="td1">18.0</td>
      <td name="td2">2.8</td>
      <td name="td3">Bloom'S Syndrome Helicase</td>
    </tr>
            
     <tr>
      <td name="td0">1WP9-A</td>
      <td name="td1">17.9</td>
      <td name="td2">2.6</td>
      <td name="td3">Atp-Dependent Rna Helicase, Putative</td>
    </tr>
            
     <tr>
      <td name="td0">5DCA-A</td>
      <td name="td1">17.2</td>
      <td name="td2">2.6</td>
      <td name="td3">Pre-Mrna-Splicing Helicase Brr2</td>
    </tr>
            
     <tr>
      <td name="td0">2OCA-A</td>
      <td name="td1">16.7</td>
      <td name="td2">3.3</td>
      <td name="td3">Atp-Dependent Dna Helicase Uvsw</td>
    </tr>
    </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] RNA aptamers to initiation factor 4A helicase hinder cap-dependent translation by blocking ATP hydrolysis.</strong></font><br />
Oguro A, Ohtsu T, Svitkin YV, Sonenberg N, Nakamura Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12649492/" target="_blank" style="color:#520049">RNA. 9(4):394-407. (2003)</a>
<br/>
            
<a id="ref2"></a><font><strong>[2] RNA aptamers to mammalian initiation factor 4G inhibit cap-dependent translation by blocking the formation of initiation factor complexes.</strong></font><br />
Miyakawa S, Oguro A, Ohtsu T, Imataka H, Sonenberg N, Nakamura Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16940549/" target="_blank" style="color:#520049">RNA. 12(10):1825-34. (2006)</a>
<br/>



<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>




    </html>
