---
layout: theophylline_style
title:  "No.21 aptamer"
date:   2024-06-05 00:00:00
author: Yuxuan Sun, Baowei Huang
categories: Aptamer
---
<html>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/12649492/" target="_blank" style="color:#520049">2003</a></h3>
  </div>
  <div class="body">
    <p>Oguro A et al.generated RNA aptamers with high affinity for eIF4A<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
  </div>
 </div>
            
 <div class="entry">
  <div class="title">
    <h3><a href="https://pubmed.ncbi.nlm.nih.gov/16940549/" target="_blank" style="color:#520049">2006</a></h3>
  </div>
  <div class="body">
    <p>Miyakawa S et al. isolated eight different RNA aptamers with high affinity to mammalian eIF4G<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
  </div>
 </div>
</div>



<p class="header_box" id="description">Description</p>
<p>In 2003, Akihiro Oguro and Takashi Ohtsu had used the SELEX  methodology to select RNA aptamers with high affinity for eIF4A by in vitro RNA selection-amplification<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2003, Akihiro Oguro and colleagues used SELEX. Four RNA selection experiments were performed using fulllength eIF4A and RNA pools of either 30 or 40 random nucleotide positions. Then, from each selection, 48 RNA sequences were cloned, aptamers that bind specifically to eIF4A were selected with high affinity<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>


<p class="header_box" id="Structure">Structure</p>
<p>The 2D structure of the figures is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></p>
<p>5'-GGGAGACAAGAAUAAACGCUCAAGGGGACCGCGCCCCACAUGUGAGUGAGGCCGAAACAUAGAUUCGACAGGAGGCUCACAACAGGC-3'</p>
<img src="/images/2D/no.21_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>



<font ><p class="header_box" id="ligand-recognition">Ligand information</p>  

<p class="blowheader_box">SELEX ligand</p>
<p>The DEAD-box helicase domain found in eIF4A and related proteins are DEAD-box helicases, a diverse family of proteins involved in ATP-dependent RNA unwinding, needed in a variety of cellular processes including splicing, ribosome biogenesis and RNA degradation. Eukaryotic initiation factor 4A-I (DDX2A) is involved in cap recognition and is required for mRNA binding to ribosome. This domain contains the ATP-binding region.-----From Pfam</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">eIF4A</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P60842/entry" target="_blank" style="color:#520049"><b>P60842</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR044728/" target="_blank" style="color:#520049"><b>IPR044728</b></a></td>
        <td name="td3">46.154 kDA</td>
        <td name="td4">MSASQDSRSRDNGPDGMEPEGVIESNWNEIVDSFDDMNLSESLLRGIYAYGFEKPSAIQQRAILPCIKGYDVIAQAQSGTGKTATFAISILQQIELDLKATQALVLAPTRELAQQIQKVVMALGDYMGASCHACIGGTNVRAEVQKLQMEAPHIIVGTPGRVFDMLNRRYLSPKYIKMFVLDEADEMLSRGFKDQIYDIFQKLNSNTQVVLLSATMPSDVLEVTKKFMRDPIRILVKKEELTLEGIRQFYINVEREEWKLDTLCDLYETLTITQAVIFINTRRKVDWLTEKMHARDFTVSAMHGDMDQKERDVIMREFRSGSSRVLITTDLLARGIDVQQVSLVINYDLPTNRENYIHRIGRGGRFGRKGVAINMVTEEDKRTLRDIETFYNTSIEEMPLNVADLI</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/2G9N" target="_blank" style="color:#520049"><b>2G9N</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/1973" target="_blank" style="color:#520049"><b>1973</b></a></td>
      </tr>
	  </tbody>
  </table>

  <p>Some isolated sequences bind to the affinity of the protein.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
      <td name="td0">no.21 aptamer</td>
      <td name="td1">5'-GGGAGACAAGAAUAAACGCUCAAGGGGACCGCGCCCCACAUGUGAGUGAGGCCGAAACAUAGAUUCGACAGGAGGCUCACAACAGGC-3'</td>
      <td name="td2">eIF4A</td>
      <td name="td3">27nM</td>
    </tr>
            
     <tr>
      <td name="td0">#11 aptamer</td>
      <td name="td1">5'-GGGAGACAAGAAUAAAACGCUCAAACAUUGCAUCGACAGCUGCAAGGCUCCCGCCGUACAAACCUUCGACAGGAGGCUCACAACAGGC-3'</td>
      <td name="td2">eIF4A</td>
      <td name="td3">8 μM</td>
    </tr>
            
     <tr>
      <td name="td0">#1 aptamer</td>
      <td name="td1">5'-GGGAGACAAGAAUAAACGCUCAACAGGCGUUUAGCCUCUAAGUAACAGGGGCCUCCCAUGAGCUUCGACAGGAGGCUCACAACAGGC-3'</td>
      <td name="td2">eIF4A</td>
      <td name="td3">3 μM</td>
    </tr>
            
     <tr>
      <td name="td0">#20 aptamer</td>
      <td name="td1">5'-GGGAGACAAGAAUAAACGCUCAAGGGGACCGCGCCCCACAUGUGAGUGAGGCCGAAACGUAGAUUCGACAGGAGGCUCACAACAGGC-3'</td>
      <td name="td2">eIF4A</td>
      <td name="td3">44 nM</td>
    </tr>
            
     <tr>
      <td name="td0">#30 aptamer</td>
      <td name="td1">5'-GGGAGACAAGAAUAAACGCUCAAUGUGGAUGAUUUGUAUGAUCGCGCAUACAAUUCGACAGGAGGCUCACAACAGGC-3'</td>
      <td name="td2">eIF4A</td>
      <td name="td3">1 μM</td>
    </tr>
	  </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/no.21_aptamer_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>


<p class="blowheader_box">Similar compound</p>                    
<p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">8RC0-D</td>
      <td name="td1">27.8</td>
      <td name="td2">1.9</td>
      <td name="td3">Cd2 Antigen Cytoplasmic Tail-Binding Protein 2</td>
    </tr>
            
     <tr>
      <td name="td0">3I5X-A</td>
      <td name="td1">25.0</td>
      <td name="td2">2.3</td>
      <td name="td3">Atp-Dependent Rna Helicase Mss116</td>
    </tr>
            
     <tr>
      <td name="td0">7NAD-X</td>
      <td name="td1">23.3</td>
      <td name="td2">1.9</td>
      <td name="td3">25S Rrna</td>
    </tr>
            
     <tr>
      <td name="td0">6ZNP-A</td>
      <td name="td1">19.1</td>
      <td name="td2">2.7</td>
      <td name="td3">Uncharacterized Atp-Dependent Helicase Ypra</td>
    </tr>
            
     <tr>
      <td name="td0">5V9X-A</td>
      <td name="td1">18.8</td>
      <td name="td2">2.7</td>
      <td name="td3">Atp-Dependent Dna Helicase</td>
    </tr>
            
     <tr>
      <td name="td0">8VX9-B</td>
      <td name="td1">18.5</td>
      <td name="td2">2.9</td>
      <td name="td3">Hama</td>
    </tr>
            
     <tr>
      <td name="td0">4CGZ-A</td>
      <td name="td1">18.0</td>
      <td name="td2">2.8</td>
      <td name="td3">Bloom'S Syndrome Helicase</td>
    </tr>
            
     <tr>
      <td name="td0">1WP9-A</td>
      <td name="td1">17.9</td>
      <td name="td2">2.6</td>
      <td name="td3">Atp-Dependent Rna Helicase, Putative</td>
    </tr>
            
     <tr>
      <td name="td0">5DCA-A</td>
      <td name="td1">17.2</td>
      <td name="td2">2.6</td>
      <td name="td3">Pre-Mrna-Splicing Helicase Brr2</td>
    </tr>
            
     <tr>
      <td name="td0">2OCA-A</td>
      <td name="td1">16.7</td>
      <td name="td2">3.3</td>
      <td name="td3">Atp-Dependent Dna Helicase Uvsw</td>
    </tr>
    </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] RNA aptamers to initiation factor 4A helicase hinder cap-dependent translation by blocking ATP hydrolysis.</strong></font><br />
Oguro A, Ohtsu T, Svitkin YV, Sonenberg N, Nakamura Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/12649492/" target="_blank" style="color:#520049">RNA. 9(4):394-407. (2003)</a>
<br/>
            
<a id="ref2"></a><font><strong>[2] RNA aptamers to mammalian initiation factor 4G inhibit cap-dependent translation by blocking the formation of initiation factor complexes.</strong></font><br />
Miyakawa S, Oguro A, Ohtsu T, Imataka H, Sonenberg N, Nakamura Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16940549/" target="_blank" style="color:#520049">RNA. 12(10):1825-34. (2006)</a>
<br/>



<html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
    </script>




    </html>
