---
layout: theophylline_style
title: RBA-14 aptamer
date: 2024-04-19 00:00:00
author: Ke Chen, Yangyi Ren
categories: Aptamer
tags:
- Category:Amino Acids & Peptides
- Category:Bacteria
- Category:Biologics & Signaling Molecules
- Category:Fluorophores
- Category:Mammalian
- Category:Nucleosides & Nucleotides
- Category:Other
- Category:Pharmaceuticals
- Category:Prion
- Category:Synthetic
- Category:Viral
- Category:cell
- GC:31.0
- GC:32.5
- GC:33.33
- GC:33.629999999999995
- GC:35.0
- GC:35.4
- GC:36.84
- GC:38.95
- GC:39.0
- GC:40.0
- GC:40.82
- GC:41.25
- GC:41.46
- GC:42.02
- GC:42.370000000000005
- GC:44.07
- GC:44.79
- GC:45.07
- GC:45.5
- GC:45.68
- GC:45.87
- GC:46.0
- GC:46.150000000000006
- GC:46.46
- GC:46.48
- GC:46.51
- GC:46.72
- GC:46.88
- GC:47.0
- GC:47.17
- GC:47.5
- GC:47.54
- GC:47.870000000000005
- GC:47.92
- GC:48.0
- GC:48.06
- GC:48.08
- GC:48.15
- GC:48.480000000000004
- GC:48.57
- GC:48.72
- GC:48.84
- GC:49.0
- GC:49.3
- GC:49.370000000000005
- GC:50.0
- GC:50.480000000000004
- GC:50.529999999999994
- GC:51.160000000000004
- GC:51.2
- GC:51.28
- GC:51.32
- GC:51.519999999999996
- GC:51.61
- GC:51.949999999999996
- GC:52.0
- GC:52.11
- GC:52.21
- GC:52.22
- GC:52.33
- GC:52.5
- GC:52.78
- GC:53.0
- GC:53.410000000000004
- GC:53.620000000000005
- GC:53.949999999999996
- GC:54.0
- GC:54.169999999999995
- GC:54.410000000000004
- GC:54.64
- GC:55.169999999999995
- GC:55.559999999999995
- GC:55.67999999999999
- GC:55.78999999999999
- GC:56.00000000000001
- GC:56.06
- GC:56.10000000000001
- GC:56.18
- GC:56.52
- GC:56.79
- GC:56.82000000000001
- GC:56.99999999999999
- GC:57.14
- GC:57.89
- GC:57.99999999999999
- GC:58.330000000000005
- GC:58.42
- GC:58.440000000000005
- GC:58.46
- GC:59.0
- GC:59.38
- GC:59.4
- GC:59.68
- GC:60.0
- GC:60.760000000000005
- GC:61.0
- GC:61.33
- GC:61.53999999999999
- GC:61.82
- GC:62.5
- GC:62.96000000000001
- GC:63.0
- GC:63.04
- GC:63.89
- GC:64.0
- GC:64.1
- GC:65.79
- GC:66.0
- GC:67.0
- GC:70.0
- GC:70.59
- GC:71.0
- GC:73.0
- Length:10
- Length:100
- Length:101
- Length:104
- Length:105
- Length:106
- Length:108
- Length:109
- Length:113
- Length:118
- Length:119
- Length:12
- Length:122
- Length:124
- Length:129
- Length:132
- Length:133
- Length:15
- Length:150
- Length:152
- Length:18
- Length:19
- Length:20
- Length:22
- Length:23
- Length:24
- Length:28
- Length:30
- Length:31
- Length:32
- Length:33
- Length:34
- Length:35
- Length:36
- Length:38
- Length:39
- Length:40
- Length:41
- Length:43
- Length:46
- Length:47
- Length:48
- Length:51
- Length:52
- Length:54
- Length:55
- Length:58
- Length:63
- Length:65
- Length:68
- Length:69
- Length:70
- Length:71
- Length:72
- Length:74
- Length:75
- Length:76
- Length:77
- Length:78
- Length:79
- Length:80
- Length:81
- Length:82
- Length:86
- Length:87
- Length:88
- Length:89
- Length:90
- Length:91
- Length:92
- Length:94
- Length:95
- Length:96
- Length:97
- Length:98
- Length:99
- Named:3',5'-cyclic adenosine monophosphate (cAMP)_cAMP - b
- Named:3',5'-cyclic adenosine monophosphate (cAMP)_cAMP - b (parent)
- Named:5-Hydroxytryptophan (5HTP)_5GR-II RNA aptamer
- Named:ACE_Okra aptamer
- Named:AML1 Runt domain_Apt1-S aptamer
- Named:AML1 Runt domain_Apt1-s2 aptamer
- Named:AMPA and kainate receptors_AB9
- Named:AMPA and kainate receptors_AB9s
- Named:Adenosine triphosphate (ATP)_ATP binding aptamer
- Named:Adenosine triphosphate (ATP)_V2 Hammerhead Ribozyme
- Named:Adenosine triphosphate (ATP)_V2 trans Hammerhead Ribozyme
- Named:Adenosine triphosphate (ATP)_aptRNA1
- Named:Adenosine triphosphate (ATP)_bG40min
- Named:Adenosine triphosphate (ATP)_bU118
- Named:Adenosine triphosphate (ATP)_bU320
- Named:Adenosine triphosphate (ATP)_bU321
- Named:Adenosine triphosphate (ATP)_pRNAwt
- Named:Angiopoietin-2 (Ang2)_11-1.41 aptamer
- Named:Basic fibroblast growth factor (bFGF)_14A
- Named:Biotin_BB8-5 RNA aptamer
- Named:Bovine prion protein_R12 aptamer
- Named:C-C motif chemokine 2 (L-CCL2)_NOX-E36 L aptamer
- Named:C-terminally truncated variant of Rev (Rev93)_RBA14 aptamer
- Named:CD133-positive on cell line HT-29_CD133-A15
- Named:CD133-positive on cell line HT-29_CD133-B19
- Named:CD133-positive on cell line Hep3B_CD133-A15
- Named:CD133-positive on cell line Hep3B_CD133-B19
- Named:Carbon storage regulation protein A, Escherichia coli (CsrA)_R9-43
- Named:Cibacron Blue 3GA_CB-42
- Named:Cyanocobalamin (vitamin B12)_35-mer aptamer
- Named:Cyclophilin B (CypB)_M9-5
- Named:Cytoplasmic tail of BACE (B1-CT)_S10
- Named:Cytoplasmic tail of BACE (B1-CT)_TH14
- Named:DMHBO+, DMHBI_Chili aptamer
- Named:Drosophila muscleblind protein (Mbl)_RNA 20
- Named:E2F transcription factor (E2F1)_E1 RNA aptamer
- Named:E2F transcription factor (E2F1)_E2F-E1
- Named:Escherichia coli release factor 1_Class II-1
- Named:Estrogen receptor α (ERα)_AptER-1
- Named:Estrogen receptor α (ERα)_AptER-2
- Named:Estrogen receptor α (ERα)_AptER-3
- Named:Eukaryotic translation initiation factor 4A (eIF4A)_#1
- Named:Eukaryotic translation initiation factor 4A (eIF4A)_#11
- Named:Eukaryotic translation initiation factor 4A (eIF4A)_#20
- Named:Eukaryotic translation initiation factor 4A (eIF4A)_#30
- Named:Eukaryotic translation initiation factor 4A (eIF4A)_APT58 aptamer
- Named:Eukaryotic translation initiation factor 4A (eIF4A)_no.21
- Named:Eukaryotic translation initiation factor 4E (eIF4E)_Aptamer 1
- Named:Eukaryotic translation initiation factor 4E (eIF4E)_Aptamer 2
- Named:Factor Xa_11F7t aptamer
- Named:Flavin adenine dinucleotide (FAD), Flavin mononucleotide(FMN), Riboflavin(Rb)_X2B2-C14U
  aptamer
- Named:Fluorescein_FB-1
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.18 aptamer
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.20 aptamer
- Named:G-protein-coupled receptor kinase 2 (GRK2)_C13.28 aptamer
- Named:Gemini-561o-Coral
- Named:Glutaminyl-tRNA synthetase_var-AGGU RNA aptamer
- Named:Guanine,Xanthine (2,6-dioxypurine)_XBA RNA
- Named:Guanosine triphosphate (GTP)_Aptamer 10-10
- Named:Guanosine triphosphate (GTP)_Aptamer 10-24
- Named:Guanosine triphosphate (GTP)_Aptamer 10-59
- Named:Guanosine triphosphate (GTP)_Aptamer 10-6
- Named:Guanosine triphosphate (GTP)_Aptamer 9-12
- Named:Guanosine triphosphate (GTP)_Class I
- Named:Guanosine triphosphate (GTP)_Class I aptamer(truncation)
- Named:HER2-positive breast cancer cells_TSA12 aptamer
- Named:HER2-positive breast cancer cells_TSA14
- Named:HER2-positive breast cancer cells_TSA14 aptamer(truncation)
- Named:HIV-1 Rev protein_1
- Named:HIV-1 Rev protein_14
- Named:HIV-1 Rev protein_4
- Named:HIV-1 Tat protein_clone 31
- Named:HIV-1 aspartyl protease (PR)_PR10.1
- Named:HIV-1 capsid (CA) protein_CA15-2
- Named:HIV-1 protease (PR)_PR10.1-8A
- Named:HIV-1 protease (PR)_PR10.1-8E
- Named:HIV-1 protease (PR)_PR10.9
- Named:HIV-1 protease (PR)_PR10.9-8N
- Named:Hfq (host factor required for phage Qβ RNA replication)_AGr aptamer
- Named:Hfq (host factor required for phage Qβ RNA replication)_m49f aptamer
- Named:Hirame rhabdovirus (HIRRV)_H1
- Named:Hirame rhabdovirus (HIRRV)_H1 aptamer(truncation)
- Named:Hirame rhabdovirus (HIRRV)_H2
- Named:Hirame rhabdovirus (HIRRV)_H2 aptamer(truncation)
- Named:Hirame rhabdovirus (HIRRV)_H3
- Named:Hirame rhabdovirus (HIRRV)_H4
- Named:Human T-cell lymphomatic virus (HTLV-1) tax protein_YT1
- Named:Human immunoglobulin G_apt2 aptamer
- Named:Human immunoglobulin G_apt7 aptamer
- Named:Human immunoglobulin G_apt8 aptamer
- Named:Human imunnodeficiency virus type 1 reverse transcriptase (HIV-1 RT)_ligand
  1.1
- Named:Human imunnodeficiency virus type 1 reverse transcriptase (HIV-1 RT)_ligand
  1.3a
- Named:L-tyrosine_Tyr 1
- Named:NF-kappaB P50_RNA aptamer 3
- Named:NS3 protease of hepatitis C virus (HCV)_10G-1
- Named:Nonstructural 5B (NS5B) polymerase, hepatitis C virus (HCV) NS5BΔC55 protein_aptamer
  B.2 SLII
- 'Named:Nuclear factor of activated T cells (NFAT) DNA binding domain_RNA aptamer
  #1'
- 'Named:Nuclear factor of activated T cells (NFAT) DNA binding domain_RNA aptamer
  #2'
- Named:Proinflammatory cytokine interleukin-17A (IL-17A)_Apt21-2 aptamer(truncation)
- Named:Proinflammatory cytokine interleukin-17A (IL-17A)_Apt21‐2
- Named:Prostate-specific membrane antigen (PSMA)_A9 RNA aptamer
- Named:Prostate-specific membrane antigen (PSMA)_A9g aptamer
- Named:Protein tyrosine phosphatase 1B (PTP1B)_C13
- Named:Protein tyrosine phosphatase 1B (PTP1B)_C13M21
- Named:Protein tyrosine phosphatase 1B (PTP1B)_C9
- Named:RNA polymerase II (Pol II), Saccharomyces cerevisiae (S. cerevisiae)_FC
- Named:RNA-dependent RNA polymerase (3Dpol)_F38
- Named:RNA-dependent RNA polymerase (3Dpol)_F47
- Named:RNA-dependent RNA polymerase (3Dpol)_F47tr
- Named:RNA-dependent RNA polymerase (3Dpol)_F52
- Named:Ras-binding domain (Raf-1 RBD)_21.01
- Named:Ras-binding domain (Raf-1 RBD)_21.07
- Named:Reactive Blue 4_B4-25
- Named:Receptor activator of NF-kappaB (RANK)_apt1
- Named:Receptor activator of NF-kappaB (RANK)_apt2
- Named:Receptor activator of NF-kappaB (RANK)_apt3
- Named:RhoBAST aptamer
- Named:Ribosomal protein S8_RNA-2 aptamer
- Named:Serine protease inhibitor plasminogen activator inhibitor-1 (PAI-1)_SM-20
- Named:Serine protease inhibitor plasminogen activator inhibitor-1 (PAI-1)_WT-15
- Named:Sheep prion protein_RM312 aptamer
- Named:Sulforhodamine B_SRB-2
- Named:T cytokine 1 (TCF-1)_#10
- Named:T4 DNA polymerase (gp43)_Major variant aptamer(truncation)
- Named:T4 DNA polymerase (gp43)_Wild type aptamer(truncation)
- Named:T4 DNA polymerase (gp43)_major variant
- Named:T4 DNA polymerase (gp43)_wild type
- Named:TMR (Tetramethylrhodamine)TMR1
- Named:TMR (Tetramethylrhodamine)TMR2
- Named:TMR (Tetramethylrhodamine)TMR3
- Named:TMR (Tetramethylrhodamine)TMR4
- Named:TMR (Tetramethylrhodamine)_TMR3 aptamer
- Named:TMR-DN, Pyronin B, ATTO495SRB-3
- Named:Tobramycin_X1
- Named:Transcription repressor TetR_TetR-RNA aptamer
- Named:Transforming growth factor-β type III receptor expressed on Chinese hamster
  ovary (CHO) cells(TbRIII)_A07
- Named:Triple-negative breast cancer (TNBC)_TN145
- Named:Triple-negative breast cancer (TNBC)_TN2
- Named:Triple-negative breast cancer (TNBC)_TN20
- Named:Triple-negative breast cancer (TNBC)_TN29
- Named:Triple-negative breast cancer (TNBC)_TN3
- Named:Triple-negative breast cancer (TNBC)_TN58
- Named:Tumor necrosis factor alpha (hTNFα)_1
- Named:Tumor necrosis factor alpha (hTNFα)_4
- Named:Vascular Endothelial Growth Factor (VEGF)_NX-213 aptamer
- Named:Zinc-finger antiviral protein (ZAP)_21-E
- Named:Zinc-finger antiviral protein (ZAP)_21-K
- Named:p-amino phenylalanine (pAF)pAF-4Z1
- Named:p-amino phenylalanine (pAF)pAF-R1-1
- Named:β-amyloid 42 (Aβ42)_E22P-AbD31
- Named:β-amyloid 42 (Aβ42)_E22P-AbD4
- Named:β-amyloid 42 (Aβ42)_E22P–AbD43
- Named:β-catenin_pUC19
- Type:Cells
- Type:Proteins
- Type:Small molecules
- Year:1990
- Year:1992
- Year:1993
- Year:1994
- Year:1995
- Year:1996
- Year:1997
- Year:1998
- Year:1999
- Year:2000
- Year:2002
- Year:2003
- Year:2004
- Year:2005
- Year:2006
- Year:2007
- Year:2008
- Year:2009
- Year:2010
- Year:2011
- Year:2012
- Year:2013
- Year:2014
- Year:2015
- Year:2017
- Year:2018
- Year:2019
- Year:2020
- Year:2021
- Year:2022
- Year:2024
type: Proteins
---
<html>




<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/7505429/" target="_blank" style="color:#520049">1993</a></h3>
    </div>
    <div class="body">
      <p>Lori Giver obtained a series of aptamers with good affinity to HIV-1 Rev protein through in vitro selecting<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/7506689/" target="_blank" style="color:#520049">1993</a></h3>
    </div>
    <div class="body">
      <p>Lori Giver found a short helical stem and bulged nucleotides (nt) CUC ... UYGAG that have no counterpart in the wild-type (wt) element contribute to high-affinity binding<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/7664035/" target="_blank" style="color:#520049">1994</a></h3>
    </div>
    <div class="body">
      <p>The 3D model explains why RNA aptamer can bind Rev protein better<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8523524/" target="_blank" style="color:#520049">1996</a></h3>
    </div>
    <div class="body">
      <p>Aptamers were found to be functionally equivalent to the RBE when the assay system was saturated with Rev and better than the wild-type element when Rev was limiting<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9713975/" target="_blank" style="color:#520049">1998</a></h3>
    </div>
    <div class="body">
      <p>Specific inhibition of viral p24 production following co-transfection of the anti-HIV Rev-binding aptamer and HIV proviral DNAs was observed<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/15610009/" target="_blank" style="color:#520049">2004</a></h3>
    </div>
    <div class="body">
      <p>More than half of the residues have increased flexibility in the Rev-RNA aptamer complex that has a higher affinity. This further suggests that the retention of conformational flexibility may be important in high-affinity ARM-RNA recognition<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/30017564/" target="_blank" style="color:#520049">2018</a></h3>
    </div>
    <div class="body">
      <p>Altaira D. Dearborn et al. present the crystal structure of Rev<sup>93</sup>:scFv:RBA-14 Ternary Complex at 3.0 Å through X-ray<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup></p>
    </div>
  </div>
</div>


         
<p><p class="header_box" id="description">Description</p>
<p>In 1993, Lori Giver and colleagues used the SELEX method to isolate the aptamer with high compatibility for the HIV-1 Rev protein. Afterwards, in 2018, Altaira D. Dearborn et al. determined the X-ray crystal structure of an RNA aptamer (RBA-14), bound to a C-terminally truncated variant of Rev (Rev<sup>93</sup>), and a single-chain variable fragment antibody (scFv) crystallization chaperone<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref8" style="color:#520049">8</a></sup><sup>]</sup>.<br></p>



<p class="header_box" id="SELEX">SELEX</p>
<p>In 1993, Lori Giver et al. constructed a pool of random sequences. After in vitro selecting in the pool, selected RNAs were amplified via reverse transcription, polymerase chain reaction (PCR) amplification, in vitro T7 transcription, and allowed to again compete for binding to Rev. Some aptamers with high affinity were selected after three selection cycles<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<p>
<br>   



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<p>Here we use ribodraw to complete the figure, through the 3D structure information. RBA-14 was the aptamer sequence mainly studied in SELEX article<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup>.</p>
<p>5'-GGCUGGACUCGUACUUCGGUACUGGAGAAACAGCC-3'</p>
<img src="/images/2D/RBA-14_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>

<p class="blowheader_box">3D visualisation</p>             
<P>Altaira D. Dearborn et al. present the crystal structure of Rev<sup>93</sup>:scFv:RBA-14 Ternary Complex at 3.0 Å through X-ray. They used Rev:scFv as a crystallization platform for studying nucleic acid binding and made a ternary complex of Rev<sup>93</sup>, scFv and RBA-14, where the single-chain antibody functions as an assembly inhibitor, crystallization chaperone, and initial model for molecular replacement. The PDB ID of this structure is 6CF2<sup>[<a href="#ref8" style="color:#520049">8</a></sup><sup>]</sup>.<br>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</p>
<div ><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/RBA-14_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A',    
  start_residue_number:1, 
  end_residue_number:116, 
  color:{r:128,g:219,b:158}
},
{
  struct_asym_id: 'B', 
  start_residue_number:1, 
  end_residue_number:110, 
  color:{r:128,g:219,b:158}
},
{
  struct_asym_id: 'F',    
  start_residue_number:10, 
  end_residue_number:66, 
  color:{r:10,g:11,b:255}
},
{
  struct_asym_id: 'G', 
  start_residue_number:1, 
  end_residue_number:3, 
  color:{r:255,g:135,b:10}
},
{
  struct_asym_id: 'G',    
  start_residue_number:4, 
  end_residue_number:7, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'G', 
  start_residue_number:8, 
  end_residue_number:21, 
  color:{r:255,g:135,b:10}
},
{
  struct_asym_id: 'G',    
  start_residue_number:22, 
  end_residue_number:23, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'G', 
  start_residue_number:24, 
  end_residue_number:24, 
  color:{r:255,g:135,b:10}
},
{
  struct_asym_id: 'G',    
  start_residue_number:25, 
  end_residue_number:25, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'G', 
  start_residue_number:26, 
  end_residue_number:26, 
  color:{r:255,g:135,b:10}
},
{
  struct_asym_id: 'G',    
  start_residue_number:27, 
  end_residue_number:30, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'G', 
  start_residue_number:31, 
  end_residue_number:35, 
  color:{r:255,g:135,b:10}
}
            ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/6CF2-3D.pdb',
        format: 'pdb'},
        hideCanvasControls: ['expand', 'selection', 'animation', 'controlToggle'],
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>

<p class="blowheader_box">Binding pocket</p>             
<p>Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 6CF2. Rev<sup>93</sup> protein (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with Rev<sup>93</sup> protein.</p>
<br><br>
  <table class="table table-bordered" style="table-layout:fixed;width:1200px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/RBA-14_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/RBA-14_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
</div>


<p class="header_box" id="ligand-recognition">Ligand information</p>
<p class="blowheader_box">SELEX ligand</p>
<p>Altaira D. Dearborn et al. determined the affinity of RBA-14 and Rev by surface plasmon resonance (SPR) method, and also compared the affinity of Rev with hairpins with Stem IIB sequences. SPR was performed on a Biacore ×100 (GE Healthcare), in 30 μL/minute HBS-EP+ buffer (10 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid, pH 7.4, 150 mM sodium chloride, 3 mM ethylenediaminetetraacetic acid, 0.05% polysorbate 20) (GE Healthcare) at 25°C<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup>.</p>
<br><br>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">RBA-14</td>
        <td name="td1">GGCUGGACUCGUACUUCGGUACUGGAGAAACAGCC</td>
        <td name="td3">Rev<sup>93</sup></td>
        <td name="td4">5.9 nM</td>
      </tr>
      <tr>
        <td name="td0">Stem IIB</td>
        <td name="td1">CCUGGGCGCAGUGUCAUUGACGCUGACGGUACAGG</td>
        <td name="td3">Rev<sup>93</sup></td>
        <td name="td4">12.9 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>


<p class="blowheader_box">Structure ligand</p>
<p>Rev is a transactivating protein that is essential to the regulation of HIV-1 protein expression. A nuclear localization signal is encoded in the rev gene, which allows the Rev protein to be localized to the nucleus, where it is involved in the export of unspliced and incompletely spliced mRNAs. In the absence of Rev, mRNAs of the HIV-1 late (structural) genes are retained in the nucleus, preventing their translation.-----From Pfam</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">MW</th>
        <th onclick="sortTable(3)">Amino acids sequences</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P04616/entry#function" target="_blank" style="color:#520049"><b>P04616</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00424/" target="_blank" style="color:#520049"><b>PF00424</b></a></td>
        <td name="td2">3.22 kDa</td>
        <td name="td3">GAMATRQARRNRRRRWRERQRAAAAR (residues T34-R50)</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/2M1A" target="_blank" style="color:#520049"><b>2M1A</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/protein/AAA44200.1" target="_blank" style="color:#520049"><b>AAA44200.1</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/RBA-14_stru_ligand.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<p>We used the RCSB PDB website's similar structure search to find the top 10 structures similar to HIV-1 Rev protein (residues T34-R50), and calculated TM-socre values and RMSD values used the TM-align website.</p>
<table class="table table-bordered" style="table-layout:fixed;width:auto;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">TM-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">2LD2</td>
        <td name="td2">0.358</td>
        <td name="td3">0.89</td>
        <td name="td4">Solution structure of the N-terminal domain of huntingtin (htt17) in presence of DPC micelles</td>
      </tr>
      <tr>
        <td name="td1">1DNG</td>
        <td name="td2">0.442</td>
        <td name="td3">1.68</td>
        <td name="td4">NMR structure of a model hydrophilic amphipathic helical acidic peptide</td>
      </tr>
      <tr>
        <td name="td1">3N95-E</td>
        <td name="td2">0.034</td>
        <td name="td3">1.7</td>
        <td name="td4">Crystal structure of human CRFR2 alpha extracellular domain in complex with Urocortin 2</td>
      </tr>
      <tr>
        <td name="td1">1OMQ</td>
        <td name="td2">0.366</td>
        <td name="td3">1.1</td>
        <td name="td4">Structure of penetratin in bicellar solution</td>
      </tr>
      <tr>
        <td name="td1">7LSO</td>
        <td name="td2">0.44</td>
        <td name="td3">0.67</td>
        <td name="td4">L-Phenylseptin</td>
      </tr>
      <tr>
        <td name="td1">2RLG</td>
        <td name="td2">0.34</td>
        <td name="td3">0.79</td>
        <td name="td4">NMR structure of the antimicrobial peptide RP-1 bound to SDS micelles</td>
      </tr>
      <tr>
        <td name="td1">3N93-E</td>
        <td name="td2">0.03</td>
        <td name="td3">1.62</td>
        <td name="td4">Crystal structure of human CRFR2 alpha extracellular domain in complex with Urocortin 3</td>
      </tr>
      <tr>
        <td name="td1">7LSP</td>
        <td name="td2">0.465</td>
        <td name="td3">0.56</td>
        <td name="td4">D-Phenylseptin - The second residue of PHE of the peptide is a D-amino acid</td>
      </tr>
      <tr>
        <td name="td1">1O53</td>
        <td name="td2">0.371</td>
        <td name="td3">0.49</td>
        <td name="td4">Solution structure of the N-terminal membrane anchor of E. coli enzyme IIA(Glucose)</td>
      </tr>
      <tr>
        <td name="td1">6GS9</td>
        <td name="td2">0.303</td>
        <td name="td3">1.36</td>
        <td name="td4">NMR structure of aurein 2.5 in SDS micelles</td>
      </tr>
	  </tbody>
  </table>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Selective optimization of the Rev-binding element of HIV-1.</strong></font><br />
Giver, L., Bartel, D., Zapp, M., Pawul, A., Green, M., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/7505429/" target="_blank" style="color:#520049" >Nucleic acids research, 21(23), 5509–5516. (1993)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Selection and design of high-affinity RNA ligands for HIV-1 Rev.</strong></font><br />
Giver, L., Bartel, D. P., Zapp, M. L., Green, M. R., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/7506689/" target="_blank" style="color:#520049">Gene, 137(1), 19–24. (1993)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] A three-dimensional model of the Rev-binding element of HIV-1 derived from analyses of aptamers.</strong></font><br />
Leclerc, F., Cedergren, R., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/7664035/" target="_blank" style="color:#520049"> Nature structural biology, 1(5), 293–300. (1994)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] RNA aptamers selected to bind human immunodeficiency virus type 1 Rev in vitro are Rev responsive in vivo.</strong></font><br />
Symensma, T. L., Giver, L., Zapp, M., Takle, G. B., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8523524/" target="_blank" style="color:#520049">Journal of virology, 70(1), 179–187. (1996)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Deep penetration of an alpha-helix into a widened RNA major groove in the HIV-1 rev peptide-RNA aptamer complex.</strong></font><br />
Ye, X., Gorin, A., Ellington, A. D., & Patel, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8946856/" target="_blank" style="color:#520049">Nature structural biology, 3(12), 1026–1033. (1996)</a>
<br />

<a id="ref6"></a><font><strong>[6] Receptor ligand-facilitated cationic liposome delivery of anti-HIV-1 Rev-binding aptamer and ribozyme DNAs.</strong></font><br />
Konopka, K., Düzgüneş, N., Rossi, J., & Lee, N. S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9713975/" target="_blank" style="color:#520049">Journal of drug targeting, 5(4), 247–259. (1998)</a>
<br />       

<a id="ref7"></a><font><strong>[7] Retention of conformational flexibility in HIV-1 Rev-RNA complexes.</strong></font><br />
Wilkinson, T. A., Zhu, L., Hu, W., & Chen, Y. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15610009/" target="_blank" style="color:#520049" >Biochemistry, 43(51), 16153–16160. (2004)</a>
<br />

<a id="ref8"></a><font><strong>[8] Structure of an RNA Aptamer that Can Inhibit HIV-1 by Blocking Rev-Cognate RNA (RRE) Binding and Rev-Rev Association.</strong></font><br />
Dearborn, A. D., Eren, E., Watts, N. R., Palmer, I. W., Kaufman, J. D., Steven, A. C., & Wingfield, P. T.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/30017564/" target="_blank" style="color:#520049" >Structure (London, England : 1993), 26(9), 1187–1195.e4.  (2018)</a>
<br />