---
layout: theophylline_style
title:  "RBA-14 aptamer"
date:   2024-04-19 00:00:00
author: Ke Chen, Yangyi Ren
categories: Aptamer
---
<html>
<head>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
 .header_box {
    display: block;
    font-size: 24px;
    background-color: #ffffff;
    text-decoration: none;
    border-radius: 1px;
    width: 500px;
    border-width: 1px 1px 2px 1px;
    border-color: #ffffff #ffffff #ffffff #ffffff;
}
.blowheader_box{
    display: block;
      padding: 6px;
      font-size:20px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 1px;
      width:170px;
      height:40px;
  }

</style>
</head>
<br>
<br>


<div class="side-nav">
<ul>
    <div class="side-nav-item"><li><a href="#timeline" style="color: #000000;">Timeline</a></li></div>
    <div class="side-nav-item"><li><a href="#description" style="color: #000000;">Description</a></li></div>
    <div class="side-nav-item"><li><a href="#SELEX" style="color: #000000;">SELEX</a></li></div>
    <div class="side-nav-item"><li><a href="#Structure" style="color: #000000;">Structure</a></li></div>
    <div class="side-nav-item"><li><a href="#ligand-recognition" style="color: #000000;">Ligand information</a></li></div>
    <div class="side-nav-item"><li><a href="#references" style="color: #000000;">References</a></li></div>
    </ul>
</div>



<p class="header_box" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/7505429/" target="_blank" style="color:#520049">1993</a></h3>
    </div>
    <div class="body">
      <p>Lori Giver obtained a series of aptamers with good affinity to HIV-1 Rev protein through in vitro selecting<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/7506689/" target="_blank" style="color:#520049">1993</a></h3>
    </div>
    <div class="body">
      <p>Lori Giver found a short helical stem and bulged nucleotides (nt) CUC ... UYGAG that have no counterpart in the wild-type (wt) element contribute to high-affinity binding<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/7664035/" target="_blank" style="color:#520049">1994</a></h3>
    </div>
    <div class="body">
      <p>The 3D model explains why RNA aptamer can bind Rev protein better<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/8523524/" target="_blank" style="color:#520049">1996</a></h3>
    </div>
    <div class="body">
      <p>Aptamers were found to be functionally equivalent to the RBE when the assay system was saturated with Rev and better than the wild-type element when Rev was limiting<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9713975/" target="_blank" style="color:#520049">1998</a></h3>
    </div>
    <div class="body">
      <p>Specific inhibition of viral p24 production following co-transfection of the anti-HIV Rev-binding aptamer and HIV proviral DNAs was observed<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/15610009/" target="_blank" style="color:#520049">2004</a></h3>
    </div>
    <div class="body">
      <p>More than half of the residues have increased flexibility in the Rev-RNA aptamer complex that has a higher affinity. This further suggests that the retention of conformational flexibility may be important in high-affinity ARM-RNA recognition<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/30017564/" target="_blank" style="color:#520049">2018</a></h3>
    </div>
    <div class="body">
      <p>Altaira D. Dearborn et al. present the crystal structure of Rev<sup>93</sup>:scFv:RBA-14 Ternary Complex at 3.0 Å through X-ray<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup></p>
    </div>
  </div>
</div>


         
<font ><p class="header_box" id="description">Description</p></font>
<font >In 1993, Lori Giver and colleagues used the SELEX method to isolate the aptamer with high compatibility for the HIV-1 Rev protein. Afterwards, in 2018, Altaira D. Dearborn et al. determined the X-ray crystal structure of an RNA aptamer (RBA-14), bound to a C-terminally truncated variant of Rev (Rev<sup>93</sup>), and a single-chain variable fragment antibody (scFv) crystallization chaperone<sup>[<a href="#ref1" style="color:#520049">1</a></sup><sup>,<a href="#ref8" style="color:#520049">8</a></sup><sup>]</sup>.<br></font>
<br>
<br>



<p class="header_box" id="SELEX">SELEX</p>
<p>In 1993, Lori Giver et al. constructed a pool of random sequences. After in vitro selecting in the pool, selected RNAs were amplified via reverse transcription, polymerase chain reaction (PCR) amplification, in vitro T7 transcription, and allowed to again compete for binding to Rev. Some aptamers with high affinity were selected after three selection cycles<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br>
<br>   



<p class="header_box" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<font><p>Here we use ribodraw to complete the figure, through the 3D structure information. RBA-14 was the aptamer sequence mainly studied in SELEX article<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup>.</p></font>
<font><p>5'-GGCUGGACUCGUACUUCGGUACUGGAGAAACAGCC-3'</p></font>
<img src="/images/2D/RBA-14_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">3D visualisation</p>             
<font >Altaira D. Dearborn et al. present the crystal structure of Rev<sup>93</sup>:scFv:RBA-14 Ternary Complex at 3.0 Å through X-ray. They used Rev:scFv as a crystallization platform for studying nucleic acid binding and made a ternary complex of Rev<sup>93</sup>, scFv and RBA-14, where the single-chain antibody functions as an assembly inhibitor, crystallization chaperone, and initial model for molecular replacement. The PDB ID of this structure is 6CF2<sup>[<a href="#ref8" style="color:#520049">8</a></sup><sup>]</sup>.<br>Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</font>
<div ><font ><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p></font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/RBA-14_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/pdbe-molstar-plugin.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .msp-plugin .msp-layout-standard {
              border: 1px solid #efefef;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A',    
  start_residue_number:1, 
  end_residue_number:116, 
  color:{r:128,g:219,b:158}
},
{
  struct_asym_id: 'B', 
  start_residue_number:1, 
  end_residue_number:110, 
  color:{r:128,g:219,b:158}
},
{
  struct_asym_id: 'F',    
  start_residue_number:10, 
  end_residue_number:66, 
  color:{r:10,g:11,b:255}
},
{
  struct_asym_id: 'G', 
  start_residue_number:1, 
  end_residue_number:3, 
  color:{r:255,g:135,b:10}
},
{
  struct_asym_id: 'G',    
  start_residue_number:4, 
  end_residue_number:7, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'G', 
  start_residue_number:8, 
  end_residue_number:21, 
  color:{r:255,g:135,b:10}
},
{
  struct_asym_id: 'G',    
  start_residue_number:22, 
  end_residue_number:23, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'G', 
  start_residue_number:24, 
  end_residue_number:24, 
  color:{r:255,g:135,b:10}
},
{
  struct_asym_id: 'G',    
  start_residue_number:25, 
  end_residue_number:25, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'G', 
  start_residue_number:26, 
  end_residue_number:26, 
  color:{r:255,g:135,b:10}
},
{
  struct_asym_id: 'G',    
  start_residue_number:27, 
  end_residue_number:30, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'G', 
  start_residue_number:31, 
  end_residue_number:35, 
  color:{r:255,g:135,b:10}
}
            ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/6CF2-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html>
</td>
</tr>
</table>
</div>


<p class="blowheader_box">Binding pocket</p>             
<font >Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 6CF2. Rev<sup>93</sup> protein (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with Rev<sup>93</sup> protein.</font>
<br><br>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/RBA-14_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/RBA-14_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br>
  <br>




<font ><p class="header_box" id="ligand-recognition">Ligand information</p></font>  
<p class="blowheader_box">SELEX ligand</p>
<font >Altaira D. Dearborn et al. determined the affinity of RBA-14 and Rev by surface plasmon resonance (SPR) method, and also compared the affinity of Rev with hairpins with Stem IIB sequences. SPR was performed on a Biacore ×100 (GE Healthcare), in 30 μL/minute HBS-EP+ buffer (10 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid, pH 7.4, 150 mM sodium chloride, 3 mM ethylenediaminetetraacetic acid, 0.05% polysorbate 20) (GE Healthcare) at 25°C<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup>.</font >
<br><br>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">RBA-14</td>
        <td name="td1">GGCUGGACUCGUACUUCGGUACUGGAGAAACAGCC</td>
        <td name="td3">Rev<sup>93</sup></td>
        <td name="td4">5.9 nM</td>
      </tr>
      <tr>
        <td name="td0">Stem IIB</td>
        <td name="td1">CCUGGGCGCAGUGUCAUUGACGCUGACGGUACAGG</td>
        <td name="td3">Rev<sup>93</sup></td>
        <td name="td4">12.9 nM</td>
      </tr>
	  </tbody>
  </table>
  </div>

<p class="blowheader_box">Structure ligand</p>
<font><p>Rev is a transactivating protein that is essential to the regulation of HIV-1 protein expression. A nuclear localization signal is encoded in the rev gene, which allows the Rev protein to be localized to the nucleus, where it is involved in the export of unspliced and incompletely spliced mRNAs. In the absence of Rev, mRNAs of the HIV-1 late (structural) genes are retained in the nucleus, preventing their translation.-----From Pfam</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">MW</th>
        <th onclick="sortTable(3)">Amino acids sequences</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P04616/entry#function" target="_blank" style="color:#520049"><b>P04616</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF00424/" target="_blank" style="color:#520049"><b>PF00424</b></a></td>
        <td name="td2">3.22 kDa</td>
        <td name="td3">GAMATRQARRNRRRRWRERQRAAAAR (residues T34-R50)</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/2M1A" target="_blank" style="color:#520049"><b>2M1A</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/protein/AAA44200.1" target="_blank" style="color:#520049"><b>AAA44200.1</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/RBA-14_stru_ligand.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;"></div>


<p class="blowheader_box">Similar compound</p>
<font><p>We used the RCSB PDB website's similar structure search to find the top 10 structures similar to HIV-1 Rev protein (residues T34-R50), and calculated TM-socre values and RMSD values used the TM-align website.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">TM-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">2LD2</td>
        <td name="td2">0.358</td>
        <td name="td3">0.89</td>
        <td name="td4">Solution structure of the N-terminal domain of huntingtin (htt17) in presence of DPC micelles</td>
      </tr>
      <tr>
        <td name="td1">1DNG</td>
        <td name="td2">0.442</td>
        <td name="td3">1.68</td>
        <td name="td4">NMR structure of a model hydrophilic amphipathic helical acidic peptide</td>
      </tr>
      <tr>
        <td name="td1">3N95-E</td>
        <td name="td2">0.034</td>
        <td name="td3">1.7</td>
        <td name="td4">Crystal structure of human CRFR2 alpha extracellular domain in complex with Urocortin 2</td>
      </tr>
      <tr>
        <td name="td1">1OMQ</td>
        <td name="td2">0.366</td>
        <td name="td3">1.1</td>
        <td name="td4">Structure of penetratin in bicellar solution</td>
      </tr>
      <tr>
        <td name="td1">7LSO</td>
        <td name="td2">0.44</td>
        <td name="td3">0.67</td>
        <td name="td4">L-Phenylseptin</td>
      </tr>
      <tr>
        <td name="td1">2RLG</td>
        <td name="td2">0.34</td>
        <td name="td3">0.79</td>
        <td name="td4">NMR structure of the antimicrobial peptide RP-1 bound to SDS micelles</td>
      </tr>
      <tr>
        <td name="td1">3N93-E</td>
        <td name="td2">0.03</td>
        <td name="td3">1.62</td>
        <td name="td4">Crystal structure of human CRFR2 alpha extracellular domain in complex with Urocortin 3</td>
      </tr>
      <tr>
        <td name="td1">7LSP</td>
        <td name="td2">0.465</td>
        <td name="td3">0.56</td>
        <td name="td4">D-Phenylseptin - The second residue of PHE of the peptide is a D-amino acid</td>
      </tr>
      <tr>
        <td name="td1">1O53</td>
        <td name="td2">0.371</td>
        <td name="td3">0.49</td>
        <td name="td4">Solution structure of the N-terminal membrane anchor of E. coli enzyme IIA(Glucose)</td>
      </tr>
      <tr>
        <td name="td1">6GS9</td>
        <td name="td2">0.303</td>
        <td name="td3">1.36</td>
        <td name="td4">NMR structure of aurein 2.5 in SDS micelles</td>
      </tr>
	  </tbody>
  </table>
<br>
<br>


                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] Selective optimization of the Rev-binding element of HIV-1.</strong></font><br />
Giver, L., Bartel, D., Zapp, M., Pawul, A., Green, M., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/7505429/" target="_blank" style="color:#520049" >Nucleic acids research, 21(23), 5509–5516. (1993)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Selection and design of high-affinity RNA ligands for HIV-1 Rev.</strong></font><br />
Giver, L., Bartel, D. P., Zapp, M. L., Green, M. R., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/7506689/" target="_blank" style="color:#520049">Gene, 137(1), 19–24. (1993)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] A three-dimensional model of the Rev-binding element of HIV-1 derived from analyses of aptamers.</strong></font><br />
Leclerc, F., Cedergren, R., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/7664035/" target="_blank" style="color:#520049"> Nature structural biology, 1(5), 293–300. (1994)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] RNA aptamers selected to bind human immunodeficiency virus type 1 Rev in vitro are Rev responsive in vivo.</strong></font><br />
Symensma, T. L., Giver, L., Zapp, M., Takle, G. B., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8523524/" target="_blank" style="color:#520049">Journal of virology, 70(1), 179–187. (1996)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Deep penetration of an alpha-helix into a widened RNA major groove in the HIV-1 rev peptide-RNA aptamer complex.</strong></font><br />
Ye, X., Gorin, A., Ellington, A. D., & Patel, D. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/8946856/" target="_blank" style="color:#520049">Nature structural biology, 3(12), 1026–1033. (1996)</a>
<br />

<a id="ref6"></a><font><strong>[6] Receptor ligand-facilitated cationic liposome delivery of anti-HIV-1 Rev-binding aptamer and ribozyme DNAs.</strong></font><br />
Konopka, K., Düzgüneş, N., Rossi, J., & Lee, N. S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9713975/" target="_blank" style="color:#520049">Journal of drug targeting, 5(4), 247–259. (1998)</a>
<br />       

<a id="ref7"></a><font><strong>[7] Retention of conformational flexibility in HIV-1 Rev-RNA complexes.</strong></font><br />
Wilkinson, T. A., Zhu, L., Hu, W., & Chen, Y. <br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15610009/" target="_blank" style="color:#520049" >Biochemistry, 43(51), 16153–16160. (2004)</a>
<br />

<a id="ref8"></a><font><strong>[8] Structure of an RNA Aptamer that Can Inhibit HIV-1 by Blocking Rev-Cognate RNA (RRE) Binding and Rev-Rev Association.</strong></font><br />
Dearborn, A. D., Eren, E., Watts, N. R., Palmer, I. W., Kaufman, J. D., Steven, A. C., & Wingfield, P. T.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/30017564/" target="_blank" style="color:#520049" >Structure (London, England : 1993), 26(9), 1187–1195.e4.  (2018)</a>
<br />