---
layout: post
title:  "Restriction endonucleases (REases) KpnI-aptamer"
date:   2024-05-31 00:00:00
author: Yangyi Ren, Chen Ke
categories: Aptamer
---
<!--p style="font-size: 24px"><strong>Keywords: </strong> <br/></p>
<font ><strong>small molecule, Synthetic, dyes, PubChem CID 25863</strong></font >
<br /-->

<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
    .header_box {
    display: block;
    font-size: 24px;
    background-color: #ffffff;
    text-decoration: none;
    border-radius: 1px;
    width: 500px;
    border-width: 1px 1px 2px 1px;
    border-color: #ffffff #ffffff #ffffff #ffffff;
}
.blowheader_box{
    display: block;
      padding: 6px;
      font-size:20px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 1px;
      width:170px;
	    height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<br>
<br>



<font><p class="header_box" id="description">Description</p></font>
<font>In 2015, Estefanía Mondragón and colleagues used SELEX (Systematic Evolution of Ligands by Exponential enrichment) technique to isolate high affinity RNA aptamers that bind to the Reases KpnI protein<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<br>
<br>


<p class="header_box" id="SELEX">SELEX</p>
<p>In 2015, Estefanía Mondragón and L. James Maher, III isolated 10 specific sequences for targeting Restriction endonucleases (REases) in 20 rounds of selection<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br>
<br>


<p class="header_box" id="Structure">Structure</p>
<font>The 2D structure of the figure is based on the article by ribodraw tool to draw<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br></font>
<font><p>5'-GGGAUAUCCUUCAUGAUUCGGAAAAGCGUAACCGCGGCCAAUCAUGCAUGAAGCGUUCCAUAUUUUU-3'</p></font>
<img src="/images/2D/REases_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>
<br>


<font ><p class="header_box" id="ligand-information">Ligand information</p></font>  

<p class="blowheader_box">SELEX ligand</p>
<font>The restriction endonucleases (REases) KpnI is an orthodox Type IIP enzyme, which binds to DNA in the absence of metal ions and cleaves the DNA sequence 5′-GGTAC^C-3′ in the presence of Mg2+ as shown generating 3′ four base overhangs.</font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Uniprot ID</th>
        <th onclick="sortTable(2)">Pfam</th>
        <th onclick="sortTable(3)">MW</th>
        <th onclick="sortTable(4)">Amino acids sequences</th>
        <th onclick="sortTable(5)">PDB</th>
        <th onclick="sortTable(6)">Gene ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Restriction endonucleases (REases) KpnI</td>
        <td name="td1"><a href="https://www.uniprot.org/uniprotkb/P04062/entry" target="_blank" style="color:#520049"><b>P04062</b></a></td>
        <td name="td2"><a href="https://www.ebi.ac.uk/interpro/entry/pfam/PF02055" target="_blank" style="color:#520049"><b>PF02055</b></a></td>
        <td name="td3">59.716 kDa</td>
        <td name="td4">MEFSSPSREECPKPLSRVSIMAGSLTGLLLLQAVSWASGARPCIPKSFGYSSVVCVCNATYCDSFDPPTFPALGTFSRYESTRSGRRMELSMGPIQANHTGTGLLLTLQPEQKFQKVKGFGGAMTDAAALNILALSPPAQNLLLKSYFSEEGIGYNIIRVPMASCDFSIRTYTYADTPDDFQLHNFSLPEEDTKLKIPLIHRALQLAQRPVSLLASPWTSPTWLKTNGAVNGKGSLKGQPGDIYHQTWARYFVKFLDAYAEHKLQFWAVTAENEPSAGLLSGYPFQCLGFTPEHQRDFIARDLGPTLANSTHHNVRLLMLDDQRLLLPHWAKVVLTDPEAAKYVHGIAVHWYLDFLAPAKATLGETHRLFPNTMLFASEACVGSKFWEQSVRLGSWDRGMQYSHSIITNLLYHVVGWTDWNLALNPEGGPNWVRNFVDSPIIVDITKDTFYKQPMFYHLGHFSKFIPEGSQRVGLVASQKNDLDAVALMHPDGSAVVVVLNRSSKDVPLTIKDPAVGFLETISPGYSIHTYLWRRQ</td>
        <td name="td5"><a href="https://www.rcsb.org/structure/1OGS" target="_blank" style="color:#520049"><b>1OGS</b></a></td>
        <td name="td6"><a href="https://www.ncbi.nlm.nih.gov/gene/2629" target="_blank" style="color:#520049"><b>2629</b></a></td>
      </tr>
	  </tbody>
  </table>

<font>Some isolated sequences bind to the affinity of the protein.</font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(1)">Sequence</th>
        <th onclick="sortTable(2)">Ligand</th>
        <th onclick="sortTable(3)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
      <td name="td0">anti-KpnI RNA Aptamer 20</td>
      <td name="td1">5'-GGGAUAUCCUUCAUGAUUCGGAAAAGCGUAACCGCGGCCAAUCAUGCAUGAAGCGUUCCAUAUUUUU-3'</td>
      <td name="td2">REases KpnI</td>
      <td name="td3">11.6 ± 2.3 nM</td>
    </tr>
     <tr>
      <td name="td0">anti-KpnI RNA Aptamer 24</td>
      <td name="td1">5'-GGGAUAUCCUUCAUGGUGUGAUUAAACGUAAUCUCAAUCGAUCAUGCAUGAAGCGUUCCAUAUUUUU-3'</td>
      <td name="td2">REases KpnI</td>
      <td name="td3">28.7 ± 5.2 nM</td>
    </tr>
     <tr>
      <td name="td0">anti-KpnI RNA Aptamer 29</td>
      <td name="td1">5'-GGGAUAUCCCUACAUGAUAAUCGGGUGAGUAAGGCCUAAACAUCAUGCAUGAAGCGUUCCAUAUUUUU-3'</td>
      <td name="td2">REases KpnI</td>
      <td name="td3">419 ± 46.6 nM</td>
    </tr>
     <tr>
      <td name="td0">anti-KpnI RNA Aptamer 30</td>
      <td name="td1">5'-GGGAUAUCCUUCAUGAUUCGGAAAAGCGUAACCGCGGCCGAUCAUGCAUGAAGCGUUCCAUAUUUUU-3'</td>
      <td name="td2">REases KpnI</td>
      <td name="td3">18.6 ± 4.5 nM</td>
    </tr>
   </tbody>
  </table>
<div style="display: flex; justify-content: center;"></div>
<img src="/images/SELEX_ligand/REases_SELEX_ligand.svg" alt="drawing" style="width:1000px;border:solid 1px #efefef;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;"></div>
<br>



<p class="blowheader_box">Similar compound</p>                    
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and chose the top 10 in terms of similarity for presentation. The Dali server is a network service for comparing protein structures in 3D. Dali compares them against those in the Protein Data Bank (PDB). Z-score is a standard score that is converted from an original score. The list of neighbours is sorted by Z-score. Similarities with a Z-score lower than 2 are spurious. RMSD(Root Mean Square Deviation) value is used to measure the degree to which atoms deviate from the alignment position.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;">
      <thead>
      <tr>
        <th onclick="sortTable(0)">PDB</th>
        <th onclick="sortTable(1)">Z-socre</th>
        <th onclick="sortTable(2)">RMSD</th>
        <th onclick="sortTable(3)">Description</th>
      </tr>
      </thead>
    <tbody>
      <tr>
      <td name="td0">2V3E-A</td>
      <td name="td1">66.2</td>
      <td name="td2">0.7</td>
      <td name="td3">Glucosylceramidase</td>
    </tr>
            
     <tr>
      <td name="td0">6KRN-A</td>
      <td name="td1">37.0</td>
      <td name="td2">2.1</td>
      <td name="td3">Mating factor alpha,gh30 xylanase b</td>
    </tr>
            
     <tr>
      <td name="td0">4QAW-A</td>
      <td name="td1">33.0</td>
      <td name="td2">2.1</td>
      <td name="td3">Xyn30d</td>
    </tr>
            
     <tr>
      <td name="td0">3CLW-A</td>
      <td name="td1">31.4</td>
      <td name="td2">2.6</td>
      <td name="td3">Conserved exported protein</td>
    </tr>
            
     <tr>
      <td name="td0">5N8K-A</td>
      <td name="td1">27.9</td>
      <td name="td2">2.4</td>
      <td name="td3">Galactocerebrosidase</td>
    </tr>
            
     <tr>
      <td name="td0">5JVK-A</td>
      <td name="td1">23.4</td>
      <td name="td2">3.6</td>
      <td name="td3">Uncharacterized protein</td>
    </tr>
            
     <tr>
      <td name="td0">1R8L-B</td>
      <td name="td1">22.0</td>
      <td name="td2">3.2</td>
      <td name="td3">Endo-beta-1,4-galactanase</td>
    </tr>
            
     <tr>
      <td name="td0">5BX9-A</td>
      <td name="td1">21.9</td>
      <td name="td2">3.8</td>
      <td name="td3">Pslg</td>
    </tr>
            
     <tr>
      <td name="td0">2VRK-A</td>
      <td name="td1">21.7</td>
      <td name="td2">2.8</td>
      <td name="td3">Alpha-l-arabinofuranosidase</td>
    </tr>
            
     <tr>
      <td name="td0">8I0A-A</td>
      <td name="td1">21.0</td>
      <td name="td2">2.9</td>
      <td name="td3">Alpha-l-arabinofuranosidase</td>
    </tr>
    </tbody>
  </table>
<br>
<br>
                 
<p class="header_box" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1]  RNA aptamer inhibitors of a restriction endonuclease.</strong></font><br />
Mondragón, E., & Maher III, L. J.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/26184872/" target="_blank" style="color:#520049">Nucleic acids research, 43(15), 7544-7555. (2015)</a>
<br/>