---
layout: post
title:  "Theophylline-aptamer(TCT8-40)"
date:   2024-04-24 00:00:00
author: Linfei Li, Yuhang Luo
categories: Aptamer
---
<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
<!--body>
  <p style="font-size: 16px">Click the buttons to navigate to different sections:</p>
  <div class="button-container">
    <a class="button" style="color:#520049" href="#timeline">Timeline</a>
    <a class="button" style="color:#520049" href="#description">Description</a>
    <a class="button" style="color:#520049" href="#SELEX">SELEX</a>
    <a class="button" style="color:#520049" href="#structure">Structure</a>
    <a class="button" style="color:#520049" href="#ligand-recognition">Ligand recognition
    <a class="button" style="color:#520049" href="#references">References</a>
   </a>
  </div>
</body-->
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
  .header_box {
     display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #c9c5c5;
      border-radius: 5px;
      width:225px;
	    height:50px;
  }
  .header_boxx1 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:115px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx2 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:140px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx3 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:85px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx4 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:120px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx5 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:215px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx6 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:135px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .blowheader_box{
    display: block;
      padding: 10px;
      font-size:18px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 5px;
      width:170px;
	    height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<p><br /></p>


<p class="header_boxx1" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/7510417/" target="_blank" style="color:#520049">1994</a></h3>
    </div>
    <div class="body">
      <p>Aptamers are isolated out for the first time<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9224568/" target="_blank" style="color:#520049">1997</a></h3>
    </div>
    <div class="body">
      <p>First demonstrated that the aptamer could be linked to stem II of a self-cleaving hammerhead ribozyme<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9253414/" target="_blank" style="color:#520049">1997</a></h3>
    </div>
    <div class="body">
      <p>Determination of RNA structural interactions in the theophylline ligand binding site using NMR techniques<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/10097080/" target="_blank" style="color:#520049">1999</a></h3>
    </div>
    <div class="body">
      <p>First discovery of theophylline-reactive riboswitches for controlling gene expression in vivo<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/11266567/" target="_blank" style="color:#520049">2001</a></h3>
    </div>
    <div class="body">
      <p>Created a binary RNA switch by linking the theophylline and FMN aptamers to the hammerhead<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/15826145/" target="_blank" style="color:#520049">2005</a></h3>
    </div>
    <div class="body">
      <p>Demonstrated that the aptamer can be truncated to a 13-mer aptamer and still retain selective binding<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/15723047/" target="_blank" style="color:#520049">2005</a></h3>
    </div>
    <div class="body">
      <p>Theophylline-controlled 'anti-switch' found to control green fluorescent protein expression in Saccharomyces cerevisiae<sup>[<a href="#ref7" style="color:#520049">7</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/16513347/" target="_blank" style="color:#520049">2006</a></h3>
    </div>
    <div class="body">
      <p>Young and Deiters added a second level of control by photocaging the theophylline ligand, resulting in a photochemical activation of the ribozyme by liberation of the caging group<sup>[<a href="#ref8" style="color:#520049">8</a>]</sup></p>
    </div>
  </div>
<div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/16996258/" target="_blank" style="color:#520049">2007</a></h3>
    </div>
    <div class="body">
      <p>Developed a computational model to generate functional theophylline ribozymes<sup>[<a href="#ref9" style="color:#520049">9</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/19033367/" target="_blank" style="color:#520049">2009</a></h3>
    </div>
    <div class="body">
      <p>A flow cytometry-based screen for identifying synthetic riboswitches that induce robust increases in gene expression in the presence of theophylline<sup>[<a href="#ref10" style="color:#520049">10</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/23452219/" target="_blank" style="color:#520049">2013</a></h3>
    </div>
    <div class="body">
      <p>Using a computational model, Penchovsky developed and tested a theophyllinehammerhead ribozyme fusion that was inactivated with a 30-fold dynamic range<sup>[<a href="#ref11" style="color:#520049">11</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/23654267/" target="_blank" style="color:#520049">2013</a></h3>
    </div>
    <div class="body">
      <p>Developed three OFF theophylline switches that are functional in vivo <sup>[<a href="#ref12" style="color:#520049">12</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/25726466/" target="_blank" style="color:#520049">2015</a></h3>
    </div>
    <div class="body">
      <p>The aptamers found to optimise ribose enlightenment<sup>[<a href="#ref13" style="color:#520049">13</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/26594238/" target="_blank" style="color:#520049">2015</a></h3>
    </div>
    <div class="body">
      <p>The better standardization of gene expression or cellular processes<sup>[<a href="#ref14" style="color:#520049">14</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/26642994/" target="_blank" style="color:#520049">2015</a></h3>
    </div>
    <div class="body">
      <p>Six new theophylline aptamers discovered using computerised screening<sup>[<a href="#ref15" style="color:#520049">15</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/29319503/" target="_blank" style="color:#520049">2018</a></h3>
    </div>
    <div class="body">
      <p>The aptamers can build logic gates in cells<sup>[<a href="#ref16" style="color:#520049">16</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/30119599/" target="_blank" style="color:#520049">2018</a></h3>
    </div>
    <div class="body">
      <p>New theophylline-reactive aptamers are screened from a synthetic library in Escherichia coli<sup>[<a href="#ref17" style="color:#520049">17</a>]</sup></p>
    </div>
  </div>
</div>


         
<font ><p class="header_boxx2" id="description">Description</p></font>
<font >In 1994, Zimmermann et al. used SELEX to screen the solution structures of high-affinity RNA aptamers with theophylline. in1997 the interaction of RNA structures in the ligand-binding site was visualised using NMR techniques. in1998 it was found that U27 and G27 RNAs bound theophylline with low affinity (Kd values > 4 μM). the NMR spectra of U27 RNA showed the presence of an A7-U27 base pair in the free RNA that prevented the formation of a key base platform motif. NMR spectroscopy of U27 RNA revealed the presence of the A7-U27 base pair in free RNA, which prevented the formation of structural motifs in the critical base platform, and these interactions inhibited theophylline binding<sup>[<a href="#ref3" style="color:#520049">3</a></sup><sup>,<a href="#ref4" style="color:#520049">4</a></sup><sup>,<a href="#ref7" style="color:#520049">7</a></sup><sup>]</sup>.<br></font>
<p><br /></p> <br>

<p class="header_boxx3" id="SELEX">SELEX</p>
<p>Robert D. Jenison et al. used SELEX to screen for RNA molecules with affinity for theophylline, generated a pool of 1014 RNA molecules that contain a 40-nucleotide region of random sequence. The RNA pool was added to a Sepharose column to which 1-carboxypropyl theophylline was covalently cross-linked. Bound RNA was eluted by the addition of 0.1 M theophylline. The eluted RNA was converted to DNA and amplified by polymerase chain reaction (PCR) as described<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br> <br>    



<p class="header_boxx4" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<font><p>Here we use ribodraw to complete the figure, through the 3D structure information. TCT8-4 was the aptamer sequence mainly studied in SELEX article.<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.</p></font>
<font><p>5'-GGCAUACCAGCCGAAAGGCCCUUGGCAGCGUC-3'</p></font>
<img src="/images/2D/Theophylline_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;">   
</div>


<p class="blowheader_box">3D visualisation</p>             
<font >Zimmermann et al, by using NMR and X-PLOR 3.113 calculations, demonstrated the solution structure of the high-affinity RNA theophylline complex, with the C27 nucleotide being the key residue that recognizes theophylline and distinguishes it from caffeine, which has a PDB ID of 1EHT<sup>[<a href="#ref3" style="color:#520049">3</a></sup><sup>]</sup>. Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</font>
<div ><font ><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p></font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/Theophylline_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A',    
  start_residue_number:1, 
  end_residue_number:1, 
  color:{r:10,g:11,b:255}
},
{
  struct_asym_id: 'A', 
  start_residue_number:2, 
  end_residue_number:4, 
  color:{R:50,G:205,B:50}
},{
  struct_asym_id: 'A', 
  start_residue_number:5, 
  end_residue_number:7, 
  color:{R:50,G:205,B:50}
},
{
  struct_asym_id: 'A', 
  start_residue_number:8, 
  end_residue_number:9, 
  color:{R:9,G:254,B:254}
},
{
  struct_asym_id: 'A', 
  start_residue_number:13, 
  end_residue_number:15, 
  color:{r:9,g:254,b:254}
},
{
  struct_asym_id: 'A', 
  start_residue_number:10, 
  end_residue_number:19, 
  color:{R:194,G:194,B:255}
},{
   struct_asym_id: 'A', 
  start_residue_number:20, 
  end_residue_number:21, 
  color:{R:9,G:254,B:254}
}{
  struct_asym_id: 'A', 
  start_residue_number:22, 
  end_residue_number:22, 
  color:{R:255,G:0,B:255}
},{
   struct_asym_id: 'A', 
  start_residue_number:23, 
  end_residue_number:23, 
  color:{R:9,G:254,B:254}
},{
  struct_asym_id: 'A',    
  start_residue_number:24, 
  end_residue_number:24, 
  color:{R:255,G:0,B:255}
},{
  struct_asym_id: 'A',    
  start_residue_number:25, 
  end_residue_number:26, 
  color:{R:9,G:254,B:254}
},{
  struct_asym_id: 'A',    
  start_residue_number:27, 
  end_residue_number:29, 
  color:{R:50,G:205,B:50}
},{
  struct_asym_id: 'A',    
  start_residue_number:30, 
  end_residue_number:32, 
  color:{R:50,G:205,B:50}
},{
  struct_asym_id: 'A',    
  start_residue_number:33, 
  end_residue_number:33, 
  color:R:10,G:11,B:255}
},{
  struct_asym_id: 'C',    
  start_residue_number:34, 
  end_residue_number:34, 
  color:{r:255,g:255,b:0}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/pdbfiles/1NEM.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
</body>
</html></td>
</tr></table><br>
</div>


<p class="blowheader_box">Binding pocket</p>             
<font >Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 1EHT by NMR. Theophylline (shown in sticks) is labeled in yellow. Right: The hydrogen bonds of binding sites of the aptamer bound with theophylline.</font>
<br><br>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Theophylline_aptamer_binding_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/Theophylline_aptamer_binding_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br><br>

  


<font ><p class="header_boxx5" id="ligand-recognition">Ligand information</p></font>  
  
<p class="blowheader_box">SELEX ligand</p>
<font >Zimmermann et al. The binding properties of specific RNAs obtained from the theophylline SELEX experiments were determined by using SELEX, PCR, counter-SELEX. by equilibrium filtration analyses using Equilibrium filtration analysis was performed on theophylline, a 38-nucleotide truncated version of TCT8-4 RNA and theophylline, TCT8-4 RNA (mTCT8-4) to determine the minimum requirement for high-affinity binding to theophylline. Minimum requirement for high affinity binding to theophylline. This RNA interacts with theophylline with a Kd of 0.1uM, confirming that all structural determinants required for theophylline binding are contained in this truncated RNA<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup>.</font >
<div style="display: flex; justify-content: center;">
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">Theophylline</td>
        <td name="td1">0.1μM</td>
      </tr>
	  </tbody>
  </table>
  </div>
  <img src="/images/SELEX_ligand/Theophylline_SELEX_ligand.svg" alt="drawing" style="width:1000px;display:block;margin:0 auto;border:solid 1px;border-radius:0;" class="img-responsive">
<br>

<p class="blowheader_box">Structure ligand</p>
<font><p>A methylxanthine derivative from tea with diuretic, smooth muscle relaxant, bronchial dilation, cardiac and central nervous system stimulant activities. Mechanistically, theophylline acts as a phosphodiesterase inhibitor, adenosine receptor blocker, and histone deacetylase activator. Theophylline is marketed under several brand names such as Uniphyl and Theochron, and it is indicated mainly for asthma, bronchospasm, and COPD.-----From drugBank</p></font>
<br>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">PubChem CID</th>
        <th onclick="sortTable(1)">Molecular Formula</th>
        <th onclick="sortTable(2)">MW</th>
        <th onclick="sortTable(3)">CAS</th>
        <th onclick="sortTable(4)">Solubility</th>
        <th onclick="sortTable(5)">Drugbank ID</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://pubchem.ncbi.nlm.nih.gov/compound/2153" target="_blank" style="color:#520049"><b>2153</b></a></td>
        <td name="td1">C<sub>7</sub>H<sub>8</sub>N<sub>4</sub>O<sub>2</sub></td>
        <td name="td2">180.16 g/mol</td>
        <td name="td3">58-55-9</td>
        <td name="td4">7360mg/L (at 25 °C)</td>
        <td name="td5"><a href="https://go.drugbank.com/drugs/DB00277" target="_blank" style="color:#520049"><b>DB00277</b> </a></td>
      </tr>
	  </tbody>
  </table>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/Theophylline_stru_ligand1.svg" alt="drawing" style="width:400px"  px="" /></td>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px"><img src="/images/Structure_ligand/Theophylline_stru_ligand2.svg" alt="drawing" style="width:400px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;">
  </div>
                    
<p class="blowheader_box">Similar compound</p>
<font><p>We screened the compounds with great similarity totheophylline by using the ZINC database and showed some of the compounds' structure diagrams. For some CAS numbers not available,we will supplement them with Pubchem CID.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">Zinc_id</th>
        <th onclick="sortTable(2)">Named</th>
        <th onclick="sortTable(3)">CAS</th>
        <th onclick="sortTable(4)">Pubchem CID</th>
        <th onclick="sortTable(5)">Structure</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">ZINC18043251</td>
        <td name="td2">Theophylline</td>
        <td name="td3">58-55-9</td>
        <td name="td4">2153</td>
        <td name="td5"><img src="/images/Similar_compound/Theophylline_Simi_compound1.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1">ZINC13517144</td>
        <td name="td2">1-Methylxanthine</td>
        <td name="td3">6136-37-4</td>
        <td name="td4">80220</td>
        <td name="td5"><img src="/images/Similar_compound/Theophylline_Simi_compound2.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1">ZINC4685854</td>
        <td name="td2">3-Methylxanthine</td>
        <td name="td3">1076-22-8</td>
        <td name="td4">70639</td>
        <td name="td5"><img src="/images/Similar_compound/Theophylline_Simi_compound3.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1">ZINC100018165</td>
        <td name="td2">8-Chlorotheophylline</td>
        <td name="td3">85-18-7</td>
        <td name="td4">10661</td>
        <td name="td5"><img src="/images/Similar_compound/Theophylline_Simi_compound4.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1">ZINC100005670</td>
        <td name="td2">8-Bromotheophylline</td>
        <td name="td3">10381-75-6</td>
        <td name="td4">11808</td>
        <td name="td5"><img src="/images/Similar_compound/Theophylline_Simi_compound5.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1">ZINC1084</td>
        <td name="td2">caffeine</td>
        <td name="td3">21399</td>
        <td name="td4">2519</td>
        <td name="td5"><img src="/images/Similar_compound/Theophylline_Simi_compound6.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1">ZINC8616085</td>
        <td name="td2">3-Methylguanine</td>
        <td name="td3">2958-98-7</td>
        <td name="td4">76292</td>
        <td name="td5"><img src="/images/Similar_compound/Theophylline_Simi_compound7.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1">ZINC100043983</td>
        <td name="td2">1,3-Dimethyluric acid</td>
        <td name="td3">944-73-0</td>
        <td name="td4">70346</td>
        <td name="td5"><img src="/images/Similar_compound/Theophylline_Simi_compound8.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
      <tr>
        <td name="td1">ZINC403604</td>
        <td name="td2">Isocaffeine</td>
        <td name="td3">519-32-4</td>
        <td name="td4">1326</td>
        <td name="td5"><img src="/images/Similar_compound/Theophylline_Simi_compound9.svg" alt="drawing" style="width:500px"  px="" /></td>
      </tr>
	  </tbody>
  </table>

<br><br>

                 
<p class="header_boxx6" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] High-resolution molecular discrimination by RNA.</strong></font><br />
Jenison, R. D., Gill, S. C., Pardi, A., & Polisky, B.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/7510417/" target="_blank" style="color:#520049" >Science (New York, N.Y.), 263(5152), 1425–1429. (1994)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Rational design of allosteric ribozymes.</strong></font><br />
Tang, J., & Breaker, R. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9224568/" target="_blank" style="color:#520049">Chemistry & biology, 4(6), 453–459. (1997)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Interlocking structural motifs mediate molecular discrimination by a theophylline-binding RNA.</strong></font><br />
Zimmermann, G. R., Jenison, R. D., Wick, C. L., Simorre, J. P., & Pardi, A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9253414/" target="_blank" style="color:#520049"> Nature structural biology, 4(8), 644–649. (1997)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] A semiconserved residue inhibits complex formation by stabilizing interactions in the free state of a theophylline-binding RNA.</strong></font><br />
Zimmermann, G. R., Shields, T. P., Jenison, R. D., Wick, C. L., & Pardi, A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/9636066/" target="_blank" style="color:#520049">Biochemistry, 37(25), 9186–919. (1998)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Engineering precision RNA molecular switches.</strong></font><br />
Soukup, G. A., & Breaker, R. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10097080/" target="_blank" style="color:#520049">Proceedings of the National Academy of Sciences of the United States of America, 96(7), 3584–3589. (1999)</a>
<br />
                
<a id="ref6"></a><font><strong>[6] Design of allosteric hammerhead ribozymes activated by ligand-induced structure stabilization.</strong></font><br />
Soukup, G. A., & Breaker, R. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10425680/" target="_blank" style="color:#520049">Structure (London, England : 1993), 7(7), 783–791. (1999)</a>
<br />
                
<a id="ref7"></a><font><strong>[7] Molecular interactions and metal binding in the theophylline-binding core of an RNA aptamer.</strong></font><br />
Zimmermann, G. R., Wick, C. L., Shields, T. P., Jenison, R. D., & Pardi, A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/10836787/" target="_blank" style="color:#520049">RNA (New York, N.Y.), 6(5), 659–667. (2000)</a>
<br />
                
<a id="ref8"></a><font><strong>[8] Cooperative binding of effectors by an allosteric ribozyme.</strong></font><br />
Jose, A. M., Soukup, G. A., & Breaker, R. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/11266567/" target="_blank" style="color:#520049">Nucleic acids research, 29(7), 1631–1637. (2001)</a>
<br />
                
<a id="ref9"></a><font><strong>[9] Unnatural base pairs mediate the site-specific incorporation of an unnatural hydrophobic component into RNA transcripts.</strong></font><br />
Endo, M., Mitsui, T., Okuni, T., Kimoto, M., Hirao, I., & Yokoyama, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15109659/" target="_blank" style="color:#520049">Bioorganic & medicinal chemistry letters, 14(10), 2593–2596. (2004)</a>
<br />
                
<a id="ref10"></a><font><strong>[10]  A theophylline responsive riboswitch based on helix slipping controls gene expression in vivo.</strong></font><br />
Suess, B., Fink, B., Berens, C., Stentz, R., & Hillen, W.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15004248/" target="_blank" style="color:#520049"> Nucleic acids research, 32(4), 1610–1614. (2004)</a>
<br />
                
<a id="ref11"></a><font><strong>[11] Unusually short RNA sequences: design of a 13-mer RNA that selectively binds and recognizes theophylline.</strong></font><br />
Anderson, P. C., & Mecozzi, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15826145/" target="_blank" style="color:#520049" >Journal of the American Chemical Society, 127(15), 5290–5291. (2005)</a>
<br />

<a id="ref12"></a><font><strong>[12] Programmable ligand-controlled riboregulators of eukaryotic gene expression.</strong></font><br />
Bayer, T. S., & Smolke, C. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/15723047/" target="_blank" style="color:#520049" >Nature biotechnology, 23(3), 337–343. (2005)</a>
<br />

<a id="ref13"></a><font><strong>[13] Photochemical hammerhead ribozyme activation.</strong></font><br />
Young, D. D., & Deiters, A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16513347/" target="_blank" style="color:#520049" >Bioorganic & medicinal chemistry letters, 16(10), 2658–2661. (2006)</a>
<br />

<a id="ref14"></a><font><strong>[14] Computational selection of nucleic acid biosensors via a slip structure model.</strong></font><br />
Hall, B., Hesselberth, J. R., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/16996258/" target="_blank" style="color:#520049" >Biosensors & bioelectronics, 22(9-10), 1939–1947. (2007)</a>
<br />

<a id="ref15"></a><font><strong>[15] A high-throughput screen for synthetic riboswitches reveals mechanistic insights into their function.</strong></font><br />
Lynch, S. A., Desai, S. K., Sajja, H. K., & Gallivan, J. P.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/17317571/" target="_blank" style="color:#520049" > Chemistry & biology, 14(2), 173–184. (2007)</a>
<br />

<a id="ref16"></a><font><strong>[16] A flow cytometry-based screen for synthetic riboswitches.</strong></font><br />
Lynch, S. A., & Gallivan, J. P.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/19033367/" target="_blank" style="color:#520049" >Nucleic acids research, 37(1), 184–192.  (2009)</a>
<br />

<a id="ref17"></a><font><strong>[17] Design principles for ligand-sensing, conformation-switching ribozymes.</strong></font><br />
Chen, X., & Ellington, A. D.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/20041206/" target="_blank" style="color:#520049" > PLoS computational biology, 5(12), e1000620. (2009)</a>
<br />

<a id="ref18"></a><font><strong>[18]  Kinetic analysis of aptazyme-regulated gene expression in a cell-free translation system: modeling of ligand-dependent and -independent expression.</strong></font><br />
Kobori, S., Ichihashi, N., Kazuta, Y., Matsuura, T., & Yomo, T.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/22733807/" target="_blank" style="color:#520049" >RNA (New York, N.Y.), 18(8), 1458–1465. (2012)</a>
<br />

<a id="ref19"></a><font><strong>[19] Computational design and biosensor applications of small molecule-sensing allosteric ribozymes.</strong></font><br />
Penchovsky R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23452219/" target="_blank" style="color:#520049" >Biomacromolecules, 14(4), 1240–1249. (2013)</a>
<br />

<a id="ref20"></a><font><strong>[20] Theophylline-dependent riboswitch as a novel genetic tool for strict regulation of protein expression in Cyanobacterium Synechococcus elongatus PCC 7942.</strong></font><br />
Nakahira, Y., Ogawa, A., Asano, H., Oyama, T., & Tozawa, Y.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23969558/" target="_blank" style="color:#520049" > Plant & cell physiology, 54(10), 1724–1735. (2013)</a>
<br />

<a id="ref21"></a><font><strong>[21] Modularity of select riboswitch expression platforms enables facile engineering of novel genetic regulatory devices.</strong></font><br />
Ceres, P., Garst, A. D., Marcano-Velázquez, J. G., & Batey, R. T.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23654267/" target="_blank" style="color:#520049" >ACS synthetic biology, 2(8), 463–472. (2013)</a>
<br />

<a id="ref22"></a><font><strong>[22] Thermodynamic and kinetic folding of riboswitches.</strong></font><br />
Badelt, S., Hammer, S., Flamm, C., & Hofacker, I. L.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/25726466/" target="_blank" style="color:#520049" > Methods in enzymology, 553, 193–213. (2015)</a>
<br />

<a id="ref23"></a><font><strong>[23] Engineering dynamic cell cycle control with synthetic small molecule-responsive RNA devices.</strong></font><br />
Wei, K. Y., & Smolke, C. D<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/26594238/" target="_blank" style="color:#520049" >Journal of biological engineering,  9, 21. (2015)</a>
<br />
                    
<a id="ref24"></a><font><strong>[24] Searching the Sequence Space for Potent Aptamers Using SELEX in Silico.</strong></font><br />
Zhou, Q., Xia, X., Luo, Z., Liang, H., & Shakhnovich, E.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/26642994/" target="_blank" style="color:#520049" >Journal of chemical theory and computation, 11(12), 5939–5946. (2015)</a>
<br />

<a id="ref25"></a><font><strong>[25] Ynthesizing artificial devices that redirect cellular information at will.</strong></font><br />
Liu, Y., Li, J., Chen, Z., Huang, W., & Cai, Z.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/29319503/" target="_blank" style="color:#520049" >eLife, 7, e31936. (2018)</a>
<br />

<a id="ref26"></a><font><strong>[26] Engineering Riboswitches in Vivo Using Dual Genetic Selection and Fluorescence-Activated Cell Sorting.</strong></font><br />
Page, K., Shaffer, J., Lin, S., Zhang, M., & Liu, J. M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/30119599/" target="_blank" style="color:#520049" > ACS synthetic biology, 7(9), 2000–2006. (2018)</a>
<br />

<a id="ref27"></a><font><strong>[27] Detection of Low-Abundance Metabolites in Live Cells Using an RNA Integrator.</strong></font><br />
You, M., Litke, J. L., Wu, R., & Jaffrey, S. R.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/30773480/" target="_blank" style="color:#520049" >Cell chemical biology, 26(4), 471–481.e3. (2019)</a>
<br />

<a id="ref28"></a><font><strong>[28] Controlling Heterogeneity and Increasing Titer from Riboswitch-Regulated Bacillus subtilis Spores for Time-Delayed Protein Expression Applications.</strong></font><br />
Tamiev, D., Lantz, A., Vezeau, G., Salis, H., & Reuel, N. F.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/31490060/" target="_blank" style="color:#520049" >ACS synthetic biology, 8(10), 2336–2346. (2019)</a>
<br />

<a id="ref29"></a><font><strong>[29] The Theophylline Aptamer: 25 Years as an Important Tool in Cellular Engineering Research.</strong></font><br />
Wrist, A., Sun, W., & Summers, R. M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/32142605/" target="_blank" style="color:#520049" >ACS synthetic biology, 9(4), 682–697. (2020)</a>
<br />