---
layout: post
title:  "Bovine prion protein(bPrP) aptamer"
date:   2024-03-29 00:00:00
author: Zhijie Tan, Qiaozhen Liu
permalink: /docs/Bovine-prion-protein(bPrP)-aptamer/
---
<html>
<head>
  <title>横向排列的点击按钮</title>
  <style>
    /* 按钮容器样式 */
    .button-container {
      display: flex;
      justify-content: left;
      align-items: center;
      height: 50px;
    }
    /* 按钮样式 */
    .button {
      display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #ffffff;
      color: #520049;
      text-decoration: none;
      border: 1px solid #520049;
      border-radius: 5px;
    }
    /* 鼠标悬停样式 */
    .button:hover {
      background-color: #c9c5c5;
      cursor: pointer;
    }
  </style>
</head>
<!--body>
  <p style="font-size: 16px">Click the buttons to navigate to different sections:</p>
  <div class="button-container">
    <a class="button" style="color:#520049" href="#timeline">Timeline</a>
    <a class="button" style="color:#520049" href="#description">Description</a>
    <a class="button" style="color:#520049" href="#SELEX">SELEX</a>
    <a class="button" style="color:#520049" href="#structure">Structure</a>
    <a class="button" style="color:#520049" href="#ligand-recognition">Ligand recognition
    <a class="button" style="color:#520049" href="#references">References</a>
   </a>
  </div>
</body-->
</html>

<html lang="zh-cn">
<head>
<meta charset="utf-8"> 
<style>
  .header_box {
     display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #c9c5c5;
      border-radius: 5px;
      width:225px;
	    height:50px;
  }
  .header_boxx1 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:115px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx2 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:140px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx3 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:85px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx4 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:120px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx5 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:215px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .header_boxx6 {
    display: block;
      padding: 10px;
      font-size:24px;
      margin-right: 10px;
      text-align: left;
      background-color: #ffffff;
      text-decoration: none;
      border-radius: 1px;
      width:135px;
	    height:50px;
      border-width: 1px 1px 2px 1px;
      border-style: solid solid solid solid;
      border-color:#ffffff #ffffff #ccc #ffffff;
  }
  .blowheader_box{
    display: block;
      padding: 10px;
      font-size:18px;
      margin-right: 10px;
      text-align: center;
      background-color: #efefef;
      color: #000000;
      text-decoration: none;
      border: 1px solid #ffffff;
      border-radius: 5px;
      width:170px;
	    height:40px;
  }
  .box_style{
    background: #ffffff;
  }
  blockquote {
  margin: 0 0 0px;
  }
</style>
</head>
<p><br /></p>


<p class="header_boxx1" id="timeline">Timeline</p>
<div class="timeline">
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/9343239/" target="_blank" style="color:#520049">1997</a></h3>
    </div>
    <div class="body">
      <p>The first aptamer for PrP was selected in 1997 using the SELEX method directed against recombinant Syrian hamster full-length prion protein (rPrP23–231)<sup>[<a href="#ref1" style="color:#520049">1</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/19098441/" target="_blank" style="color:#520049">2008</a></h3>
    </div>
    <div class="body">
      <p>RNA aptamers against a bovine bPrP were obtained by means of an in vitro selection method from RNA pools containing a 55-nt randomized region<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/23180780/" target="_blank" style="color:#520049">2009</a></h3>
    </div>
    <div class="body">
      <p>High-resolution structure of an RNA aptamer (R12) against isolated domains of the bovine PrP by NMR was first reported and two lysine clusters of bPrP have been identified as binding sites for R12<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup></p>
    </div>
  </div>
    <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/19666720/" target="_blank" style="color:#520049">2013</a></h3>
    </div>
    <div class="body">
      <p>NMR measurements provide the first high-resolution 3D-structure of the complex formed with N-terminal PrP peptides (P1 and P16) and the R12 aptamer. What's more, R12 reduced the PrP<sub>Sc</sub> level in mouse neuronal cells persistently infected with the transmissible spongiform encephalopathy agent<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup></p>
    </div>
  </div>
   <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/24803670/" target="_blank" style="color:#520049">2014</a></h3>
    </div>
    <div class="body">
      <p>The hybrid method and the 3D-RISM theory are employed to demonstrate that the driving force for the binding between R12 and P16 (a PrP peptide) is a robust gain of water entropy, and the energy decrease driven by attractive interactions between R12 and P16 is compensated by the energetic dehydration effect after binding or vice-versa<sup>[<a href="#ref5" style="color:#520049">5</a>]</sup></p>
    </div>
  </div>
  <div class="entry">
    <div class="title">
      <h3><a href="https://pubmed.ncbi.nlm.nih.gov/30916478/" target="_blank" style="color:#520049">2019</a></h3>
    </div>
    <div class="body">
      <p>An anti-prion RNA aptamer, R12, inhibits the interaction of PrP with Aβto prevent Alzheimer's disease<sup>[<a href="#ref6" style="color:#520049">6</a>]</sup></p>
    </div>
  </div>
</div>


         
<font ><p class="header_boxx2" id="description">Description</p></font>
<font >In 2008, Nishikawa er al. used the SELEX method to isolate the aptamer and found four continuous GGA triplet repeats (GGA<sup>4</sup> in obtained major RNA aptamers. Then in 2013, Mashima et al. used NMR measurements to provide the first high-resolution 3D-structure of the complex formed with N-terminal PrP peptides (P1 and P16) and the R12 aptamer<sup>[<a href="#ref2" style="color:#520049">2</a></sup><sup>,<a href="#ref4" style="color:#520049">4</a></sup><sup>]</sup>.<br></font>
<p><br /></p> <br>

<p class="header_boxx3" id="SELEX">SELEX</p>
<p>In 2008, Nishikawa er al. used the SELEX method to isolate the aptamer and found four continuous GGA triplet repeats (GGA<sup>4</sup> in obtained major RNA aptamers<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.<br>
Detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/docs/SELEX" target="_blank" style="color:#520049"><b><i>SELEX</i></b></a> page.</p>
<br> <br>    



<p class="header_boxx4" id="Structure">Structure</p>
<p class="blowheader_box">2D representation</p>
<font><p>R12 forms an intramolecular parallel quadruplex. The quadruplex contains G:G:G:G tetrad and G(:A):G:G(:A):G hexad planes. Two quadruplexes form a dimer through intermolecular hexad–hexad stacking<sup>[<a href="#ref3" style="color:#520049">3</a>]</sup>.</p></font>
<font><p>5'-GGAGGAGGAGGA-3'</p></font>
<img src="/images/2D/prion_protein_aptamer_2D.svg" alt="drawing" style="width:800px;display:block;margin:0 auto;border-radius:0;" class="img-responsive">
<div style="display: flex; justify-content: center;">   
</div>



<p class="blowheader_box">3D visualisation</p>             
<font >Mashima et al. used NMR measurements to provide the first high-resolution 3D-structure of the complex formed with N-terminal PrP peptides (P1 and P16) and the R12 aptamer<sup>[<a href="#ref4" style="color:#520049">4</a>]</sup>. Additional available structures that have been solved and detailed information are accessible on <a href="{{ site.url }}{{ site.baseurl }}/docs/Structures" target="_blank" style="color:#520049"><b><i>Structures</i></b></a> page.</font>
<div ><font ><p style="text-align:right;margin-bottom: 0px;">(Clicking the "Settings/Controls info" to turn Spin off)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p></font>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/3D/prion_protein_aptamer_3D1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
   <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <html lang="en">
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <title>PDBe Molstar</title>
      <!-- Molstar CSS & JS -->
      <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-1.2.1.css">
      <script src="/js/mol/ro_pdbe-molstar-plugin-1.2.1.js"></script>
        <style>
          * {
              margin: 0;
              padding: 0;
              box-sizing: border-box;
          }
          .msp-plugin ::-webkit-scrollbar-thumb {
              background-color: #474748  !important;
          }
          .viewerSection1 {
            padding-top: 0px;
          }
          .controlsSection1 {
            width: 300px;
              display: flex;
              float:left;
              padding: 0px 0 0 0;
              height:25px;
            }
            .controlBox1 {
              border: 0px solid lightgray;
              padding: 0px;
              margin-bottom: 0px;
            }
          #myViewer1{
            float:left;
            width:500px;
            height: 500px;
            position:relative;
          }
        </style>
    </head>
    <body onload="customize()">
        <div class="controlsSection1">
            <button onclick="
              var selectSections1 = [
                {
  struct_asym_id: 'A',
  start_residue_number:1,
  end_residue_number:4,
  color:{r:34,g:139,b:34}
},
{
  struct_asym_id: 'A', 
  start_residue_number:5, 
  end_residue_number:5, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'A',    
  start_residue_number:6, 
  end_residue_number:7, 
  color:{r:34,g:139,b:34}
},
{
  struct_asym_id: 'A', 
  start_residue_number:8, 
  end_residue_number:9, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'A',    
  start_residue_number:10, 
  end_residue_number:10, 
  color:{r:34,g:139,b:34}
},
{
  struct_asym_id: 'A', 
  start_residue_number:11, 
  end_residue_number:11, 
  color:{r:255,g:0,b:255}
},
{
  struct_asym_id: 'A',    
  start_residue_number:12, 
  end_residue_number:12, 
  color:{r:34,g:139,b:34}
},
{
  struct_asym_id: 'C',    
  start_residue_number:25, 
  end_residue_number:36, 
  color:{r:255,g:135,b:10}
}
              ]
            viewerInstance1.visual.select({ data: selectSections1, nonSelectedColor: {r:255,g:255,b:255}})" style="float:right;height:25px;width: 120px;">Color Selection</button><br><br>
          <button button style="float: left;height:25px;width: 120px;" onclick="viewerInstance1.visual.clearSelection()">Clear Selection</button><br><br>
      </div>
    <div class="viewerSection1">
    <!-- Molstar container -->
      <div id="myViewer1"></div>
    </div>
    <script>
      var viewerInstance1 = new PDBeMolstarPlugin();
      var options1 = {
        customData:{
        url:'/docs/pdbfiles/2ru7-3D.pdb',
        format: 'pdb'},
        expanded: false,
        hideControls: true,
        bgColor: {r:255, g:255, b:255},
        }
      var viewerContainer1 = document.getElementById('myViewer1');
      viewerInstance1.render(viewerContainer1, options1);
  window.addEventListener('load', function() {
    var colorSelectionButton1 = document.querySelector('.controlsSection1 button');
    colorSelectionButton1.click();
  });
</script>
    </body>
    </html></td>
  </tr></table><br>
  </div>


<p class="blowheader_box">Binding pocket</p>             
<font >Left: Surface representation of the binding pocket of the aptamer generated from PDB ID: 2RU7 at 2.8 Å resolution. Bovine prion protein (shown in vacuumm electrostatics), blue is positive charge, red is negative charge. Right: The hydrogen bonds of binding sites of the aptamer bound with bovine prion protein.</font>
<br><br>
  <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
  <td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/prion_protein_pocket1.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  <td style="text-align:center;padding-bottom: 0px;padding-right: 0px;padding-top: 0px;padding-right: 0px">
  <img src="/images/Binding_pocket/prion_protein_pocket2.svg" alt="drawing" style="width:500px;margin-top: 0px;margin-bottom: 0px;" >
  </td>
  </tr>
  </table>
  <br><br>




<font ><p class="header_boxx5" id="ligand-recognition">Ligand information</p></font>  
  
<p class="blowheader_box">SELEX ligand</p>
<font >Murakami et al. Radioisotope labeling of RNA by in vitro transcription was carried out using α-<sub>32</sub>P-ATP to determine the binding constant of the aptamer<sup>[<a href="#ref2" style="color:#520049">2</a>]</sup>.</font >
<br><br>
<div style="display: flex; justify-content: center;">
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Name</th>
        <th onclick="sortTable(0)">Sequence</th>
        <th onclick="sortTable(0)">Ligand</th>
        <th onclick="sortTable(1)">Affinity</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0">apt #1</td>
        <td name="td1">5′-CAAUCCAUUCAUCUCUCGAAUGAGGAAGUAGCCCAAGAggaggaggaggaugagc-3′</td>
        <td name="td3">bovine prion protein (bPrP)</td>
        <td name="td4">82nM</td>
      </tr>
      <tr>
        <td name="td0">apt #6</td>
        <td name="td1">5′-ACCUUCUGUUCAUCGUGACCAACCCAAUAGAUUGUGAUAAAGGAGGAGGAGGA-3′</td>
        <td name="td3">bovine prion protein (bPrP)</td>
        <td name="td4">166nM</td>
      </tr>
	  </tbody>
  </table>
  </div>
<br>

<p class="blowheader_box">Structure ligand</p>
<font><p>Prion protein (PrP<sub>C</sub>) is a small glycoprotein found in high quantity in the brain of animals infected with certain degenerative neurological diseases, such as sheep scrapie and bovine.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(0)">Uniprot ID</th>
        <th onclick="sortTable(1)">Pfam</th>
        <th onclick="sortTable(2)">MW</th>
        <th onclick="sortTable(3)">Amino acids sequences</th>
        <th onclick="sortTable(4)">PDB ID</th>
        <th onclick="sortTable(5)">GenBank</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td0"><a href="https://www.uniprot.org/uniprotkb/P10279/entry" target="_blank" style="color:#520049"><b>P10279</b></a></td>
        <td name="td1"><a href="https://www.ebi.ac.uk/interpro/entry/InterPro/IPR000817/" target="_blank" style="color:#520049"><b>IPR000817</b></a></td>
        <td name="td2">1.38 KDa</td>
        <td name="td3">GQWNKPSKPKTN</td>
        <td name="td4"><a href="https://www.rcsb.org/structure/2RU7" target="_blank" style="color:#520049"><b>2RU7</b></a></td>
        <td name="td5"><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?lvl=0&id=9913" target="_blank" style="color:#520049"><b>9913</b> </a></td>
      </tr>
	  </tbody>
  </table>
 <table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;"><tr>
<td style="text-align:center;padding-bottom: 0px;padding-left: 0px;padding-top: 0px;padding-right: 0px">
<img src="/images/Structure_ligand/prion_protein_stru_ligand.svg" alt="drawing" style="width:800px"  px="" /></td>
</tr>
</table>
<div style="display: flex; justify-content: center;">
  </div>


<p class="blowheader_box">Similar compound</p>
<font><p>We used the Dail server website to compare the structural similarities of ligand proteins, and selected the previous information with high similarity for presentation. The Dali server is a network.</p></font>
<table class="table table-bordered" style="table-layout:fixed;width:1000px;margin-left:auto;margin-right:auto;" >
  <thead>
      <tr>
        <th onclick="sortTable(1)">PDB</th>
        <th onclick="sortTable(2)">Z-score</th>
        <th onclick="sortTable(3)">RMSD</th>
        <th onclick="sortTable(4)">Description</th>
      </tr>
  </thead>
    <tbody>
      <tr>
        <td name="td1">2KUN-A</td>
        <td name="td2">12</td>
        <td name="td3">2.7</td>
        <td name="td4">major prion protein</td>
      </tr>
      <tr>
        <td name="td1">1I17-A</td>
        <td name="td2">6.1</td>
        <td name="td3">3.5</td>
        <td name="td4">prion-like protein</td>
      </tr>
      <tr>
        <td name="td1">6THH-C</td>
        <td name="td2">5.2</td>
        <td name="td3">2.8</td>
        <td name="td4">sirv2 acrid1 (gp02) anti-crispr protein</td>
      </tr>
      <tr>
        <td name="td1">7BU0-A</td>
        <td name="td2">5</td>
        <td name="td3">5</td>
        <td name="td4">uncharacterized protein</td>
      </tr>
      <tr>
        <td name="td1">2VWA-A</td>
        <td name="td2">5</td>
        <td name="td3">3.1</td>
        <td name="td4">putative uncharacterized protein pf13_0012</td>
      </tr>
      <tr>
        <td name="td1">6WB9-6</td>
        <td name="td2">4.9</td>
        <td name="td3">3.4</td>
        <td name="td4">endoplasmic reticulum membrane protein complex su</td>
      </tr>
      <tr>
        <td name="td1">2KHM-A</td>
        <td name="td2">4.8</td>
        <td name="td3">3.6</td>
        <td name="td4">fibroin-3</td>
      </tr>
      <tr>
        <td name="td1">3WFW-A</td>
        <td name="td2">4.8</td>
        <td name="td3">2.9</td>
        <td name="td4">hemoglobin-like flavoprotein fused to roadblock/l</td>
      </tr>
      <tr>
        <td name="td1">8CR1-C</td>
        <td name="td2">4.7</td>
        <td name="td3">2.5</td>
        <td name="td4">atpase asna1</td>
      </tr>
      <tr>
        <td name="td1">5ZIQ-A</td>
        <td name="td2">4.6</td>
        <td name="td3">2.3</td>
        <td name="td4">globin protein</td>
      </tr>
	  </tbody>
  </table>

<br><br>

                 
<p class="header_boxx6" id="references">References</p>
                
<a id="ref1"></a><font><strong>[1] An intramolecular quadruplex of (GGA)4 triplet repeat DNA with a G:G:G:G tetrad and a G(:A):G(:A):G(:A):G heptad, and its dimeric interaction.</strong></font><br />
Matsugami, A., Ouhashi, K., Kanagawa, M., Liu, H., Kanagawa, S., Uesugi, S., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/11800555/" target="_blank" style="color:#520049" >Journal of Molecular Biology, 313(2):255-69. (2001)</a>
<br />
                
<a id="ref2"></a><font><strong>[2] Detection of structural changes of RNA aptamer containing GGA repeats under the ionic condition using the microchip electrophoresis.</strong></font><br />
Nishikawa, F., Murakami, K., Noda, K., Yokoyama, T., & Nishikawa, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18029754/" target="_blank" style="color:#520049">Nucleic Acids Symposium Series,(51):397-8. (2007)</a>
<br />
                
<a id="ref3"></a><font><strong>[3] Anti-bovine Prion protein RNA aptamer containing tandem GGA repeat interacts both with recombinant bovine prion protein and its β isoform with high affinity.</strong></font><br />
Murakami, K., Nishikawa, F., Noda, K., Yokoyama, T., & Nishikawa S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/19098441/" target="_blank" style="color:#520049">Prion, 2(2):73-80. (2008)</a>
<br />
                
<a id="ref4"></a><font><strong>[4] Structural analysis of r(GGA)4 found in RNA aptamer for bovine prion protein.</strong></font><br />
Matsugami, A., Mashima, T., Nishikawa, F., Murakami, K., & Nishikawa, S.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/18776312/" target="_blank" style="color:#520049">Nucleic Acids Symposium Series, (52):179-80. (2008)</a>
<br />
                
<a id="ref5"></a><font><strong>[5] Unique quadruplex structure and interaction of an RNA aptamer against bovine prion protein.</strong></font><br />
Mashima, T., Matsugami, A., Nishikawa, F., Nishikawa, S., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/19666719/" target="_blank" style="color:#520049">Nucleic Acids Research, 37(18):6249-58. (2009)</a>
<br />
                
<a id="ref6"></a><font><strong>[6] Anti-prion activity of an RNA aptamer and its structural basis.</strong></font><br />
Mashima, T., Nishikawa, F., Kamatari, YO., Fujiwara, H., Saimura, M., Nagata, T., Kodaki, T., Nishikawa, S., Kuwata, K., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23180780/" target="_blank" style="color:#520049">Nucleic Acids Research, 41(2):1355-62. (2012)</a>
<br />

<a id="ref7"></a><font><strong>[7] Cross-talk between prion protein and quadruplex-forming nucleic acids: a dynamic complex formation.</strong></font><br />
Cavaliere, P., Pagano, B., Granata, V., Prigent, S., Rezaei, H., Giancola, C., & Zagari, A.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/23104426/" target="_blank" style="color:#520049">Nucleic Acids Research, 41(1):327-39. (2013)</a>
<br />

<a id="ref8"></a><font><strong>[8] Binding of an RNA aptamer and a partial peptide of a prion protein: crucial importance of water entropy in molecular recognition.</strong></font><br />
Hayashi, T., Oshima, H., Mashima, T., Nagata, T., Katahira, M., & Kinoshita, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/24803670/" target="_blank" style="color:#520049">Nucleic Acids Research, 42(11):6861-75. (2014)</a>
<br />

<a id="ref9"></a><font><strong>[9] The anti-prion RNA aptamer R12 disrupts the Alzheimer's disease-related complex between prion and amyloid β.</strong></font><br />
Iida, M., Mashima, T., Yamaoki, Y., So, M., Nagata, T., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/30916478/" target="_blank" style="color:#520049">FEBS Journal, 286(12):2355-2365. (2019)</a>
<br />

<a id="ref10"></a><font><strong>[10] Development and structural determination of an anti-PrPC aptamer that blocks pathological conformational conversion of prion protein.</strong></font><br />
Mashima, T., Lee, JH., Kamatari, YO., Hayashi, T., Nagata, T., Nishikawa, F., Nishikawa, S., Kinoshita, M., Kuwata, K., & Katahira, M.<br />
<a href="https://pubmed.ncbi.nlm.nih.gov/32188933/" target="_blank" style="color:#520049">Scientific Reports, 10(1):4934. (2020)</a>
<br />